{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1d63c8a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Imports\n",
    "import xml.etree.ElementTree as ET\n",
    "import requests\n",
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "from sklearn.cluster import KMeans\n",
    "\n",
    "from collections import defaultdict\n",
    "from sklearn.metrics import silhouette_score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "2d8d82d0-beb2-483b-aae2-2e738f896fa2",
   "metadata": {},
   "outputs": [],
   "source": [
    "# generating a list with pmids from the input file\n",
    "with open(\"PMIDs_list.txt\", \"r\") as f:\n",
    "    pmids = [line.strip() for line in f if line.strip()]\n",
    "\n",
    "# Removing duplicats\n",
    "seen = set() # used later\n",
    "unique_pmids = []\n",
    "\n",
    "for pmid in pmids:\n",
    "    if pmid not in seen:\n",
    "        seen.add(pmid)\n",
    "        unique_pmids.append(pmid)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "5a7e81e5-a548-42d2-86bc-d7908b0dee8a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Creating elink request and requesting\n",
    "ids_param = \",\".join(unique_pmids)\n",
    "\n",
    "url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi\"\n",
    "params = {\n",
    "    \"dbfrom\": \"pubmed\",\n",
    "    \"db\": \"gds\",\n",
    "    \"linkname\": \"pubmed_gds\",\n",
    "    \"id\": ids_param,\n",
    "    \"retmode\": \"xml\"\n",
    "}\n",
    "\n",
    "response = requests.get(url, params=params)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b3a20776-b4a2-42ac-bd43-5b281894d89a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n",
      "<!DOCTYPE eLinkResult PUBLIC \"-//NLM//DTD elink 20101123//EN\" \"https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20101123/elink.dtd\">\n",
      "<eLinkResult>\n",
      "\n",
      "  <LinkSet>\n",
      "    <DbFrom>pubmed</DbFrom>\n",
      "    <IdList>\n",
      "      <Id>30530648</Id>\n",
      "      <Id>31820734</Id>\n",
      "      <Id>31018141</Id>\n",
      "      <Id>38539015</Id>\n",
      "      <Id>33763704</Id>\n",
      "      <Id>32572264</Id>\n",
      "      <Id>31002671</Id>\n",
      "      <Id>33309739</Id>\n",
      "      <Id>21057496</Id>\n",
      "      <Id>27716510</Id>\n",
      "      <Id>34059805</Id>\n",
      "      <Id>34941412</Id>\n",
      "      <Id>33879573</Id>\n",
      "      <Id>35440059</Id>\n",
      "      <Id>29462153</Id>\n",
      "      <Id>29794063</Id>\n",
      "      <Id>25939354</Id>\n",
      "      <Id>30322904</Id>\n",
      "      <Id>36879017</Id>\n",
      "      <Id>35419551</Id>\n",
      "      <Id>31501549</Id>\n",
      "      <Id>23042784</Id>\n",
      "      <Id>22219169</Id>\n",
      "      <Id>20670891</Id>\n",
      "      <Id>20385583</Id>\n",
      "      <Id>19723310</Id>\n",
      "      <Id>20602769</Id>\n",
      "      <Id>26566685</Id>\n",
      "      <Id>38030723</Id>\n",
      "      <Id>30498128</Id>\n",
      "      <Id>20485568</Id>\n",
      "      <Id>21613409</Id>\n",
      "      <Id>31076851</Id>\n",
      "      <Id>37169753</Id>\n",
      "      <Id>39800688</Id>\n",
      "      <Id>36510023</Id>\n",
      "      <Id>38177678</Id>\n",
      "      <Id>36539615</Id>\n",
      "      <Id>37871105</Id>\n",
      "      <Id>35235788</Id>\n",
      "      <Id>27799057</Id>\n",
      "      <Id>30820472</Id>\n",
      "      <Id>31666070</Id>\n",
      "      <Id>34686734</Id>\n",
      "      <Id>22384383</Id>\n",
      "      <Id>34033742</Id>\n",
      "      <Id>33589615</Id>\n",
      "      <Id>31792364</Id>\n",
      "      <Id>29576475</Id>\n",
      "      <Id>39762647</Id>\n",
      "      <Id>32084358</Id>\n",
      "      <Id>38977847</Id>\n",
      "      <Id>25493933</Id>\n",
      "      <Id>31125107</Id>\n",
      "      <Id>26749252</Id>\n",
      "      <Id>39587714</Id>\n",
      "      <Id>30333487</Id>\n",
      "      <Id>33743111</Id>\n",
      "      <Id>35172154</Id>\n",
      "      <Id>32025611</Id>\n",
      "      <Id>31136284</Id>\n",
      "      <Id>26740022</Id>\n",
      "      <Id>37989753</Id>\n",
      "      <Id>39838364</Id>\n",
      "      <Id>39367016</Id>\n",
      "      <Id>36650381</Id>\n",
      "      <Id>35511946</Id>\n",
      "      <Id>38641753</Id>\n",
      "      <Id>38287646</Id>\n",
      "      <Id>36840360</Id>\n",
      "      <Id>36544018</Id>\n",
      "      <Id>39637179</Id>\n",
      "      <Id>35767948</Id>\n",
      "      <Id>31801092</Id>\n",
      "      <Id>38909241</Id>\n",
      "      <Id>32384151</Id>\n",
      "      <Id>26280576</Id>\n",
      "      <Id>38379415</Id>\n",
      "      <Id>29550329</Id>\n",
      "      <Id>19211887</Id>\n",
      "      <Id>36803569</Id>\n",
      "      <Id>30320226</Id>\n",
      "      <Id>35920937</Id>\n",
      "      <Id>37958987</Id>\n",
      "      <Id>25340342</Id>\n",
      "      <Id>37277533</Id>\n",
      "      <Id>24223949</Id>\n",
      "    </IdList>\n",
      "    <LinkSetDb>\n",
      "      <DbTo>gds</DbTo>\n",
      "      <LinkName>pubmed_gds</LinkName>\n",
      "      \n",
      "        <Link>\n",
      "\t\t\t\t<Id>200272793</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200268524</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200263013</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200261729</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200254970</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200250598</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200249713</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200249708</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200249493</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200243256</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200242202</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200231588</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200228348</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200227741</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200216999</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200216991</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200216375</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200215954</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200215901</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200214496</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200214458</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200210256</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200210255</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200210254</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200208265</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200199806</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200197916</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200197063</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200197017</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200196613</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200196612</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200196611</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200190939</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200189217</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200186357</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200183766</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200183765</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200183764</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200183556</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200180979</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200172402</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200171017</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200171016</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200169162</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200168661</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200168613</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200168586</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200167559</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200165870</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200164788</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200164174</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200162803</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200162802</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200162728</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200159798</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200159795</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200159794</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200159596</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200157145</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200157027</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200156426</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200156425</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200156424</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200156212</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200149097</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200148990</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200148989</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200148988</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200146264</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200146026</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200145669</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200145668</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200145531</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200142259</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200140977</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200140778</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200140493</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200137444</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200134646</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200133922</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200130711</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200130633</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200130051</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200127893</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200127892</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200127884</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200127711</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200124391</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200122497</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200121750</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200120584</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200118723</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200117064</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200116672</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200115513</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200113740</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200109159</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200109158</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200106817</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200105115</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200103708</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200102248</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200099962</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200097693</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200097179</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200096066</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200088798</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200087162</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200084422</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200080744</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200076381</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200073002</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200072698</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200066017</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200065685</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200063945</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200063944</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200063943</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200063942</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200063203</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200060687</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200060202</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200060167</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200060126</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200054680</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200054362</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200054292</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200047798</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200040539</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200034429</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200034427</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200034077</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200033528</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200028094</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200023694</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200022872</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200022870</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200022859</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200021662</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200018178</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200018126</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200017554</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200015745</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>200009990</Id>\n",
      "\t\t\t</Link>\n",
      "        <Link>\n",
      "\t\t\t\t<Id>3915</Id>\n",
      "\t\t\t</Link>\n",
      "      \n",
      "    </LinkSetDb>\n",
      "  </LinkSet>\n",
      "</eLinkResult>\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(response.status_code)\n",
    "print(response.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "328cbd67-7d50-4a53-8132-5f35072155b4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Converting to XML\n",
    "root = ET.fromstring(response.text)\n",
    "\n",
    "# Creating a list of all the gds ids from the given pmids\n",
    "gds_ids = [link.find(\"Id\").text for link in root.findall(\".//LinkSetDb/Link\")]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "6f3fc615-c66a-4be8-959b-2a0ba1e5de85",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Creating esummary request for all of the gds ids\n",
    "ids_param = \",\".join(gds_ids)\n",
    "\n",
    "url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\"\n",
    "params = {\n",
    "    \"db\": \"gds\",\n",
    "    \"id\": ids_param,\n",
    "    \"retmode\": \"xml\"\n",
    "}\n",
    "\n",
    "response = requests.get(url, params=params)\n",
    "root = ET.fromstring(response.text)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "7daf5ff9-084d-4a9e-9858-5b4cc861b16d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Helper function for geting a single item \n",
    "def get_item(name):\n",
    "    item = docsum.find(f\".//Item[@Name='{name}']\")\n",
    "    return item.text if item is not None else \"\"\n",
    "\n",
    "# Helper function for getting all the values from the item list\n",
    "def get_item_list(name):\n",
    "    return [i.text for i in docsum.findall(f\".//Item[@Name='{name}']/Item\")]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "4dff6717-60aa-4770-b055-bcd251d94c0f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Creating the main dictionary\n",
    "gds_dict = {}\n",
    "for docsum in root:\n",
    "    gds_id = docsum.find(\"Id\").text\n",
    "    title = get_item(\"title\")\n",
    "    summary = get_item(\"summary\")\n",
    "    organism = get_item(\"taxon\")\n",
    "    exp_type = get_item(\"gdsType\")\n",
    "    pmids = get_item_list(\"PubMedIds\")\n",
    "    # ovr_design is not available in gds esummary result, however it is attainable in BioProjekt database\n",
    "\n",
    "    # Filtering the pmids to include only the ones from input txt file\n",
    "    inp_pmids = [pmid for pmid in pmids if pmid in unique_pmids]\n",
    "\n",
    "    # Skipping the dataset if somehow it is not connected to any pmids form the input txt file\n",
    "    if not inp_pmids:\n",
    "        continue\n",
    "\n",
    "    # One entry in the gds_dict is one dataset's description\n",
    "    gds_dict[gds_id] = {\n",
    "        \"inp_pmids\": inp_pmids,\n",
    "        \"title\": title,\n",
    "        \"summary\": summary,\n",
    "        \"organism\": organism,\n",
    "        \"exp_type\": exp_type,\n",
    "    }\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "b25fcc28-980e-48a0-9fe0-e64d84a4f6d8",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'200272793': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development (Mouse embryo RNA-seq)',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200268524': {'inp_pmids': ['39838364'],\n",
       "  'title': 'A robust circulating microRNA signature for diagnosis and early detection in pancreatobiliary cancer',\n",
       "  'summary': 'Purpose: Oncologists for pancreatobiliary cancer (PBca) demand a new circulating biomarker superior to carbohydrate antigen 19-9 (CA19-9). This study aimed to identify the serum microRNA signature composed of reproducible and disease-related microRNAs with clinical bioinformatics. Methods: This multicenter study enrolled patients with treatment-naïve PBca and healthy participants. Optimized serum processing condition was evaluated with t-distributed stochastic neighbor embedding (t-SNE) visualization. The serum microRNA candidates were selected in disease association with Weighted Gene Coexpression Network Analysis (WGCNA). A microRNA signature, combination of multiple serum microRNAs, was examined in exploratory, validation and independent validation set. Biology of the diagnostic miRNAs was evaluated using human pancreatic cancer cells. Results: The 284 (healthy 150, PBca 134) of 827 serum samples were processed within 2hr of time-to serum collection, distributed at the same area of t-SNE map, assigned to exploratory set. The 193 optimized samples were assigned to validation (healthy 50, PBca 47) or independent validation set (healthy 50, PBca 46). Index-1 was a combination of the 5 serum microRNAs having disease-association on WGCNA, showed a sensitivity/specificity >80% and an AUC outperforming CA19-9 in exploratory, validation and independent validation set. The AUC of Index-1 for detecting T1 tumor was superior to CA19-9 (0.856 vs. 0.649, p = 0.038). Human pancreatic cancer cells had intra- and extracellular expression of miR-665, a component of Index-1. Transfection of miR-665 to a human pancreatic cancer cell inhibited cell growth. Conclusions: A serum microRNA signature, Index-1, was useful for detecting PBca, which would improve the early diagnosis of PBca.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200263013': {'inp_pmids': ['38641753'],\n",
       "  'title': 'Repetitive element transcript accumulation is related to inflammaging in humans',\n",
       "  'summary': 'Inflammation is an established feature of many age-related diseases, and growing evidence implicates transcripts from repetitive elements in inflammatory signaling with aging. We generated transcriptomic (RNA-seq) and whole-genome bisulfite sequencing (WGBS, for methylation) data on frozen/biobanked peripheral blood mononuclear cells from middle-aged and older adults, as well as younger and older adults before and after an exercise intervention, to evaluate correlations among repetitive elements and inflammation, along with methylation of repetitive elements.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200261729': {'inp_pmids': ['39762647'],\n",
       "  'title': 'Investigating the transcriptional heterogeneity of the beta cells experiencing Ca2+ excitotoxicity',\n",
       "  'summary': 'Beta cells are remarkably heterogeneous. However, beta cells response to chronic stress at a single cell level remains largely unknown. Here, we used a chemo-genetic model of beta cell excitotoxcity coupled with single cell RNA sequencing to understand beta cells response to chronic stress. Remarkably, we found that a subset of stressed beta cells progressed towards death while the other group evaded the destruction by dedifferentiation. Vulnerable beta cells that progressed towards death showed upregulation of genes implicated in apoptosis, inflammation and immune system activation. The resilient beta cells that evaded the stress showed increased expression aldh1a3, a canonical marker of mammalian beta cell dedifferentiation.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200254970': {'inp_pmids': ['38909241'],\n",
       "  'title': 'Age, Sex and Alzheimer’s disease: a longitudinal study of 3xTg-AD mice reveals sex-specific disease trajectories and inflammatory responses mirrored in postmortem brains from Alzheimer’s patients',\n",
       "  'summary': 'Aging and sex are major risk factors for developing late-onset Alzheimer’s disease. Compared to men, women are not only nearly twice as likely to develop Alzheimer’s, but they also experience worse neuropathological burden and cognitive decline despite living longer with the disease. It remains unclear how and when sex differences in biological aging emerge and contribute to Alzheimer’s disease pathogenesis. We hypothesized that these differences lead to distinct molecular Alzheimer’s disease signatures in males and females, which could be harnessed for therapeutic and biomarker development. We aged male and female, 3xTg-AD and B6129 control mice across their respective lifespans while longitudinally collecting brain samples. We conducted RNA sequencing analysis on bulk brain tissue and examined differentially expressed genes between 3xTg-AD and B6129 samples and across ages in each sex. 3xTg-AD males experienced an accelerated upregulation of immune-related gene expression in the brain relative to females, especially in genes involved in complement system activation, suggesting distinct inflammatory disease trajectories between the sexes. Our data demonstrate that chronic inflammation and complement activation are associated with increased mortality, revealing that age-related changes in immune response act as a primary driver of sex differences in Alzheimer’s disease trajectories.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200250598': {'inp_pmids': ['39637179'],\n",
       "  'title': 'Epigenetic aging of seminal fluid is associated with inflammation',\n",
       "  'summary': 'Male infertility has been a primary cause of global infertility, affecting 8-12% of couples worldwide. Previous studies have shown that semen quality decreases with advanced aging with an increased presence of inflammatory cells. In this study, we examined the DNA methylation of seminal fluid to understand how the methylome changes with age and infertility. We also compare the age associated changes in semen to those observed in buccal swabs in order to characterize differences in epigenetic aging across diverse tissues. We found that infertility associated changes in the DNA methylation of semen are driven by changes in cell composition, while those associated with aging are linked to inflammatory genes. Many age associated sites are demethylated with advanced aging and are associated with the activation of inflammatory pathways. By contrast, we do not observe age associated changes in inflammatory genes in buccal swab methylomes, which instead are characterized by changes to bivalent promoters. Our study suggests that DNA methylation could serve as a biomarker for male infertility assessment in addition to the standard semen analysis.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200249713': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Non-coding RNA profiling by high throughput sequencing'},\n",
       " '200249708': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development [Small-Seq]',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing'},\n",
       " '200249493': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development [RNA-seq]',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200243256': {'inp_pmids': ['38977847'],\n",
       "  'title': 'Mapping the chromatin accessibility landscape of zebrafish embryogenesis at single-cell resolution by SPATAC-seq',\n",
       "  'summary': 'Zebrafish is a popular model for studies of embryonic development. Recent surveys of zebrafish embryogenesis with single-cell transcriptomics have uncovered hundreds of cell types, but the dynamic accessible regions of the genome that determine regulatory programs responsible for the unique identity and function of each cell type during embryogenesis lack detailed study. Here, we present SPATAC-seq: a split-pool ligation-based high-throughput single-cell assay for transposase-accessible chromatin using sequencing. Using SPATAC-seq, we profiled chromatin accessibility in more than 808,046 individual nuclei across 20 developmental stages spanning the sphere stage to the early larval protruding mouth stages in a single experiment. Using this chromatin accessibility map, we identified 604 cell types and inferred their developmental relationships. We also identified \\ufeff959,040 candidate cis-regulatory elements (cCREs) and delineate development-specific cCREs, as well as transcription factors defining diverse cell identities. Importantly, transient reporter assays confirmed that a majority of tested cCREs exhibited robust \\ufeffenhanced EGFP expression in restricted cell types or tissues. Finally, we explored gene regulatory programs that drive pigment and notochord cell differentiation. Our work provides a comprehensive and valuable open resource for exploring transcriptional regulators that determine cell fate specification in zebrafish embryogenesis.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200242202': {'inp_pmids': ['38379415'],\n",
       "  'title': 'Age-related changes in human skeletal muscle transcriptome and proteome are more affected by chronic inflammation and physical inactivity than primary aging',\n",
       "  'summary': 'Evaluation of the influence of primary and secondary aging on the manifestation of molecular and cellular hallmarks of aging is a challenging and currently unresolved issue. Our study represents the first demonstration of the distinct role of primary aging and both chronic inflammation and physical inactivity – the most important drivers of secondary aging, in the regulation of transcriptomic and proteomic profiles in human skeletal muscle. To achieve this purpose, young healthy people (n=15), young (n=8) and older (n=37) patients with knee/hip osteoarthritis, a model to study the effect of long-term inactivity and chronic inflammation on the vastus lateralis muscle, were included in the study. It was revealed that widespread and substantial age-related changes in gene expression (~4,000 genes regulating mitochondrial function, proteostasis, cell membrane, secretory and immune response) were related to the long-term physical inactivity and chronic inflammation rather than primary aging. Primary aging contributed mainly to the regulation of genes (~200) encoding nuclear proteins (regulators of DNA repair, RNA processing, and transcription), mitochondrial proteins (genes encoding respiratory enzymes, mitochondrial complex assembly factors, regulators of cristae formation and mitochondrial reactive oxygen species production), as well as regulators of proteostasis. It was found that proteins associated with aging were regulated mainly at the post-transcriptional level. The set of putative primary aging genes and their potential transcriptional regulators can be used as a resource for further targeted studies investigating the role of individual genes and related transcription factors in the emergence of a senescent cell phenotype.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200231588': {'inp_pmids': ['38030723'],\n",
       "  'title': 'BL6Mousei: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Allele-specific gene expression have been observed in genomic imprinting, X chromosome inactivation, allelic exclusion of immunoglobulin loci and non-canonical random monoallelic expression. Here we show that MSL2 can regulate allele specific gene expression and we elucidate its mechanism of action. MSL2 is known to regulate gene dosage up to two fold in drosophila and mammals. In addition to its canonical role as a milder modulator, we found that the loss of MSL2 switches off gene expression completely in an allele specific way and results in biallelic to monoallelic gene expression, which will be ignored as classical down regulated genes without applying the mouse hybrid F1 system. It indicates that MSL2 can activate the expression of the allele which is otherwise inactive intrinsically. We further found this is achieved by the interplay between MSL and KANSL complex. MSL2 acts as a methylation-insensitive factor, which can prevent DNA methylation of the inactive allele, thus facilitating occupancy of methylation-sensitive factor KANSL3 and activating the gene expression. The novel mechanism of MSL2 provides a safeguarding system to warrant biallelic gene expression of intrinsically monoallelic expressed genes including X-inactivated and imprinting genes. These genes are further discovered as sex bias genes such as X-inactivation escaped genes (escapees) are expressed higher in females. Furthermore, the biallelic expression of MSL2 regulated escapees may protect females from single loss of function mutation and contribute to male severer or predominance neurodevelopmental diseases.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200228348': {'inp_pmids': ['37989753'],\n",
       "  'title': 'Transcriptome analysis of human iPSC-neurons treated with cardiac glycosides',\n",
       "  'summary': 'To investigate the molecular mechanisms of cardiac glycosides’ neuroprotection, we profiled transcriptomes of human iPSC-neurons after 72h of treatment with increasing doses of digoxin, oleandrin, proscillaridin A or vehicle alone (0.1% DMSO) using RNA sequencing.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200227741': {'inp_pmids': ['38030723'],\n",
       "  'title': 'CaBlCaXi: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Allele-specific gene expression have been observed in genomic imprinting, X chromosome inactivation, allelic exclusion of immunoglobulin loci and non-canonical random monoallelic expression. Here we show that MSL2 can regulate allele specific gene expression and we elucidate its mechanism of action. MSL2 is known to regulate gene dosage up to two fold in drosophila and mammals. In addition to its canonical role as a milder modulator, we found that the loss of MSL2 switches off gene expression completely in an allele specific way and results in biallelic to monoallelic gene expression, which will be ignored as classical down regulated genes without applying the mouse hybrid F1 system. It indicates that MSL2 can activate the expression of the allele which is otherwise inactive intrinsically. We further found this is achieved by the interplay between MSL and KANSL complex. MSL2 acts as a methylation-insensitive factor, which can prevent DNA methylation of the inactive allele, thus facilitating occupancy of methylation-sensitive factor KANSL3 and activating the gene expression. The novel mechanism of MSL2 provides a safeguarding system to warrant biallelic gene expression of intrinsically monoallelic expressed genes including X-inactivated and imprinting genes. These genes are further discovered as sex bias genes such as X-inactivation escaped genes (escapees) are expressed higher in females. Furthermore, the biallelic expression of MSL2 regulated escapees may protect females from single loss of function mutation and contribute to male severer or predominance neurodevelopmental diseases.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200216999': {'inp_pmids': ['38539015'],\n",
       "  'title': 'Xenografted human microglia display diverse transcriptomic states in response to Alzheimer’s disease-related Aβ pathology',\n",
       "  'summary': 'Microglia are central players in Alzheimer’s Disease (AD) pathology, but analyzing microglia states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the AppNL-G-F model of amyloid pathology and wild type controls. Xenografted human microglia adopt a disease-associated (DAM) profile similar to that seen in mouse microglia, but display a more pronounced HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine CRM response to oligomeric Aβo. Genetic deletion of TREM2 or APOE, as well as APOE polymorphisms and TREM2R47H expression in the transplanted microglia modulate these responses differentially. The expression of other AD risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in response to AD-related pathology, which should be taken into account in translational studies.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200216991': {'inp_pmids': ['37989753'],\n",
       "  'title': 'Small Molecule Inducers of Neuroprotective miR-132 Identified by HTS-HTS in Human iPSC-derived Neurons',\n",
       "  'summary': 'MicroRNAs (miRNAs) are short RNAs that regulate fundamental biological processes. miR-132, a key miRNA with established functions in Tau homeostasis and neuroprotection, is consistently downregulated in Alzheimer’s disease (AD) and other tauopathies. miR-132 overexpression rescues neurodegenerative phenotypes in several AD models. To complement research on miRNA-mimicking oligonucleotides targeting the central nervous system, we developed a high-throughput-screen coupled high-throughput-sequencing (HTS-HTS) in human induced pluripotent stem cell (iPSC)-derived neurons to identify small molecule inducers of miR-132. We discovered that cardiac glycosides, which are canonical sodium-potassium ATPase inhibitors, selectively upregulated miR-132 in the sub-μM range. Coordinately, cardiac glycoside treatment downregulated total and phosphorylated Tau in rodent and human neurons and protected against toxicity by glutamate, N-methyl-D-aspartate, rotenone, and Aβ oligomers. In conclusion, we identified small-molecule drugs that upregulated the neuroprotective miR-132 and ameliorated neurodegenerative phenotypes. Our dataset also represents a comprehensive resource for discovering small molecules that modulate specific miRNAs for therapeutic purposes.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing'},\n",
       " '200216375': {'inp_pmids': ['36803569'],\n",
       "  'title': 'The essential roles of FXR in diet and aging affected metabolic changes and liver disease development: a multi-omics study',\n",
       "  'summary': 'Aging and unhealthy diets are risks for metabolic diseases including liver cancer. Bile acid receptor farnesoid X receptor (FXR) knockout (KO) mice develop metabolic liver diseases and progress into liver cancer as they age, and Western diet (WD) intake facilitates liver carcinogenesis in those mice. This study aimed to uncover molecular signatures within the gut-liver axis for diet and age-linked liver diseases in FXR-dependent or independent manners. Many more transcripts were changed due to WD intake and aging in WT mice than those in FXR KO mice. In other words, WD intake and aging impact the hepatic transcriptomes and metabolomes in an FXR-dependent manner. In WT mice, WD/aging upregulated inflammation-related genes and downregulated genes involved in oxidative phosphorylation (OXPHOS). Urine metabolomes provided clear distinguishing for differential dietary intake. By contrast, the metabolomes of the liver, serum, or urine could reflect age differences. Further, irrespective of differential diets intake or ages, transcriptomes, metabolomes, and cecal microbiota distinguished WT and FXR KO. Western dietary patterns and aging share molecular commonality with FXR deactivation. Notably, WD, aging, and FXR deactivation commonly altered hepatic cell division-related transcripts (Cenpe, Ect2, Top2a, Kif20a, Tpx2, Nuf2, Kif18b, Aspm, E2f8, and Hmmr), which are associated with overall survival rate in HCC patients. In conclusion, FXR is essential for maintaining metabolic homeostasis in response to WD intake and aging. FXR activation helps to alleviate diet and/or aging-induced metabolic health issues.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200215954': {'inp_pmids': ['37871105'],\n",
       "  'title': 'Precision pharmacological reversal of strain-specific diet-induced metabolic syndrome in mice informed by epigenetic and transcriptional regulation',\n",
       "  'summary': 'We exposed three mouse strains to a high-fat high-carbohydrate diet, leading to varying degrees of metabolic syndrome. Comprehensive methylation and transcriptomic analysis found overlapping but also highly divergent changes in DNA methylation and gene expression, with pathway analysis suggesting a strategy for strain-specific targeted pharmacologic intervention of upstream regulators. Administeration of the drug GW4064 to target one of these genotype-dependent networks, the farnesoid X receptor pathway resulted in genotype-specific protection against dietary effects in BL6 as predicted by our epigenetic analysis.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200215901': {'inp_pmids': ['38977847'],\n",
       "  'title': 'SPATAC-seq: split-pool ligation-based single cell ATAC-seq method',\n",
       "  'summary': \"To establish an ultra-high-throughput single cell chromatin accessibility profiling method that is cost-effective and widely accessible, we built on sci-ATAC-seq (Cusanovich, D. A. et al. Science. 2015; Amini, S. et al. Nature Genetics. 2014) and SPLIT-seq (Rosenberg, A. B. et al. Science. 2018) to design SPATAC-seq, which in situ label chromatin fragment in the same single cell through combinatorial barcoding. Briefly, in SPATAC-seq, (1) fixed nuclei are transposed in 48 different wells by 48 unique Tn5 transposase, which containing barcoded adaptors and 5'-phosphorylation; (2) the nuclei from all wells are collected and redistributed into second and third 48-well plate in turn, where the next two rounds of indexing are achieved through into either end of the custom transposome, which result in the generation of more than 110,000 (48^3) unique barcode combinations. (3) the nuclei are pooled, split into sublibraries and lysed, and the DNA was amplified by polymerase chain reaction (PCR), which introduce illumina sequencing barcodes and complete libraries construction. (4) After sequencing, fastq files were demultiplexed according to the same four-barcode combinations. For profiling more cells in one sublibrary, we can increase the number of barcode combinations by increase the number of indexing of each round to 96, which can produce about 1 million combinations. To assess the fidelity of SPATAC-seq, we performed a species-mixing experiment with cultured human (K562) and mouse (Hepa) cells. Here, we tagged mixed permeabilized nuclei with only 8 barcoded transposome. In round 4, we generated eight sublibraries with different cell-recovery targets, which can be used to evaluating the stability of this method and the correlation between real doublet rates and theoretical value.\",\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200214496': {'inp_pmids': ['38177678'],\n",
       "  'title': 'DNA methylation restricts default deterministic fates towards neural and germline lineages',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200214458': {'inp_pmids': ['38177678'],\n",
       "  'title': 'DNA methylation restricts default deterministic fates towards neural and germline lineages during early development',\n",
       "  'summary': 'We investigate the impact of DNA methylation on chromatin accessibility during epiblast differentiation. To this end, we used in vitro Epiblast-like cell (EpiLC) differentiation on WT and DNA-methylation free TKO murine ESCs. We next performed ATAC seq using TDE1 Tn5 transposase (Nextera).',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200210256': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200210255': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study with Infinium HumanMethylationEPIC BeadChip',\n",
       "  'summary': 'African-American individuals of the GENOA study',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200210254': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study with Infinium HumanMethylation450 BeadChip',\n",
       "  'summary': 'African-American individuals of the GENOA study',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200208265': {'inp_pmids': ['35920937'],\n",
       "  'title': 'Characterization and analysis of the promoter region of Monodehydroascorbate reductase 4 (CpMDAR4) in papaya',\n",
       "  'summary': 'The aim of this research was to study the effects of cis-element differences between the X, Y and Yh alleles on the expression of CpMDAR4, a potential candidate gene for sex differentiation in papaya, using a transcriptional reporter system in a model species Arabidopsis thaliana. Possible effects of a retrotransposon insertion in the Y and Yh alleles on the transcription and expression of CpMDAR4 alleles in papaya flowers were also examined. When comparing promoters and cis-regulatory elements among genes in the non-recombining region of the sex chromosomes, paired genes exhibited differences. Our results showed that differences in the promoter sequences of the CpMDAR4 alleles drove the expression of a reporter gene to different flower tissues in Arabidopsis. β-glucuronidase staining analysis of T2 and T3 lines for constructs containing 5’ deletions of native Y and Yh allele promoters showed the loss of specific expression of the reporter gene in the anthers, confirming the existence and location of cis-regulatory element POLLEN1LELAT52. The expression analysis of CpMDAR4 alleles in papaya flowers also showed that all alleles are actively expressed in different flower tissues, with the existence of a shorter truncated isoform, with unknown function, for the Y and Yh alleles due to an LTR-RT insertion in the Y and Yh chromosomes. The observed expression patterns in Arabidopsis thaliana flowers and the expression patterns of CpMDAR4 alleles in papaya flowers, suggest that MDAR4 might have a role on development of reproductive organs in papaya, and that it constitutes an important candidate for sex differentiation.',\n",
       "  'organism': 'Carica papaya',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200199806': {'inp_pmids': ['36510023'],\n",
       "  'title': 'DNA methyltransferases 3A and 3B target specific sequences during mouse gastrulation',\n",
       "  'summary': 'We performed single-cell transcriptome analysis (using MARS-seq) of mouse embryonic stem cells, embryiod bodies and mouse embryos. Additionally, we performed methylation analysis of bulk and single cells of this source (using whole-genome PBAT and PBAT capture).',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200197916': {'inp_pmids': ['37277533'],\n",
       "  'title': 'Metabolic competition between lipid metabolism and histone methylation regulates sexual differentiation in human malaria parasites',\n",
       "  'summary': 'Like most pathogens, malaria parasites balance persistence in the current host with transmission to the next. For Plasmodium falciparum, the most widespread and virulent human malaria parasite, persistence depends on continuous asexual replication in red blood cells while transmission requires differentiation into non-replicating gametocytes, the male and female cells able to infect the mosquito vector. This developmental decision is controlled via stochastic derepression of a heterochromatin-silenced locus encoding the transcription factor PfAP2-G, which acts as the master switch for sexual differentiation. Recent work showed that the frequency of pfap2-g derepression is responsive to extracellular levels of phospholipid precursors. However, the regulatory mechanisms linking these metabolites to epigenetic regulation of pfap2-g was hitherto unknown. Using chemical, metabolomic, and genetic approaches, we show that this response is mediated by metabolic competition for the methyl donor S-adenosylmethionine between histone methyltransferases and phospho-ethanolamine methyltransferase, a critical enzyme in the parasite’s pathway for de novo phosphatidyl-choline synthesis. When phosphatidyl-choline precursors are scarce, increased consumption of S-adenosylmethionine for de novo synthesis of phosphatidyl-choline results in a decrease in histone methylation marks that mediate silencing of pfap2-g, thereby up-regulating its transcription and increasing the frequency of sexual differentiation. Our findings reveal a key connection between metabolite utilization and gene expression in malaria parasites that forms the mechanistic link between phosphatidyl-choline metabolism and the frequency of sexual commitment.',\n",
       "  'organism': 'Plasmodium falciparum',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200197063': {'inp_pmids': ['36879017'],\n",
       "  'title': 'A gene silencing screen uncovers diverse tools for targeted gene repression in Arabidopsis',\n",
       "  'summary': 'DNA methylation has been utilized for target gene silencing in plants, however it’s not well-understood whether other silencing pathways can be also used to manipulate gene expression. Here we performed a gain-of-function screen for proteins that could silence a target gene when fused to an artificial zinc finger. We uncovered many proteins that suppressed gene expression through DNA methylation, histone H3K27me3 deposition, H3K4me3 demethylation, histone deacetylation, inhibition of RNA Polymerase II transcription elongation or Ser-5 dephosphorylation. These proteins also silenced many other genes with different efficacy, and a machine learning model could accurately predict the efficacy of each silencer based on various chromatin features of the target loci. Furthermore, some proteins were also able to target gene silencing when used in a dCas9-SunTag system. These results provide a more comprehensive understanding of epigenetic regulatory pathways in plants and provide an armament of tools for targeted gene manipulation.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200197017': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [scRNA-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200196613': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200196612': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [ATAC-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200196611': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [RNA-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200190939': {'inp_pmids': ['35511946'],\n",
       "  'title': 'Circadian alignment of early onset caloric restriction promotes longevity in male C57BL/6J mice',\n",
       "  'summary': 'Caloric restriction (CR) prolongs lifespan, yet the mechanisms by which it does so remain poorly understood. Under CR, mice self-impose chronic cycles of 2-hour-feeding and 22-hour-fasting, raising the question whether calories, fasting, or time of day are causal. We show that 30%-CR is sufficient to extend lifespan 10%; however, a daily fasting interval and circadian-alignment of feeding act together to extend lifespan 35% in male C57BL/6J mice. These circadian effects of CR are independent of fasting duration and body weight. Aging induces widespread increases in gene expression associated with inflammation and decreases in expression of genes encoding components of metabolic pathways in liver from ad lib fed mice. CR at night ameliorates these aging-related changes. Thus, circadian interventions promote longevity and provide a perspective to further explore mechanisms of aging.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200189217': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of blood aging [droplet-based scRNAseq - Batch 2]',\n",
       "  'summary': 'To investigate the role of IL1 signaling in hematopoietic aging, we performed droplet-based scRNAseq (10X Genomics) of hematopoietic stem and progenitor cells (LK/LSK) and unfractionated endosteal and central marrow stromal cells (Ter119-/CD45-) from young wild type and young and old wild type and IL1R1-/- C57BL/6 mice.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200186357': {'inp_pmids': ['36539615'],\n",
       "  'title': 'Dynamics of DNA hydroxymethylation and methylation during mouse embryonic and germline development',\n",
       "  'summary': 'In mammals, DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development. Yet, to what extent 5hmC participates in genome-wide methylation reprogramming remains largely unknown. Here, we characterize the 5hmC landscapes in mouse early embryos and germ cells with parental allele specificity. DNA hydroxymethylation was most strongly correlated with DNA demethylation as compared with de novo or maintenance methylation in zygotes, while 5hmC was targeted to particular de novo methylated sites in postimplantation epiblasts. Surprisingly, DNA replication was also required for 5hmC generation, especially in the female pronucleus. More strikingly, aberrant nuclear localization of Dnmt1/Uhrf1 in mouse zygotes due to maternal deficiency of Nlrp14 led to defects in DNA-replication-coupled passive demethylation and impaired 5hmC deposition, revealing the divergency between genome-wide 5-methylcytosine (5mC) maintenance and Tet-mediated oxidation. In summary, our work provides insights and a valuable resource for the study of epigenetic regulation in early embryo development.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200183766': {'inp_pmids': ['38030723'],\n",
       "  'title': 'CaBl: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200183765': {'inp_pmids': ['38030723'],\n",
       "  'title': 'BlCa: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200183764': {'inp_pmids': ['38030723'],\n",
       "  'title': '9sCa: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Other; Methylation profiling by high throughput sequencing'},\n",
       " '200183556': {'inp_pmids': ['38030723'],\n",
       "  'title': 'MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Other'},\n",
       " '200180979': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200172402': {'inp_pmids': ['34033742'],\n",
       "  'title': 'microRNA-132 regulates adult neural stem cells and their progeny to restore adult hippocampal neurogenesis and memory deficits in Alzheimer’s disease',\n",
       "  'summary': 'Neural stem cells residing in the hippocampal neurogenic niche sustain life-long neurogenesis in the adult brain. Adult hippocampal neurogenesis (AHN) is functionally linked to mnemonic and cognitive plasticity in humans and rodents. In Alzheimer’s disease (AD), the process of generating new neurons at the hippocampal neurogenic niche is impeded, yet the mechanisms involved are unknown. Here we identify miR-132, one of the most consistently downregulated microRNAs in AD, as a potent regulator of AHN, exerting cell-autonomous pro-neurogenic effects in the adult neural stem cells and their progeny. Using distinct AD mouse models, cultured human primary and established neural stem cells, and human patient material, we demonstrate that AHN is directly impacted by AD pathology. miR-132 replacement in adult mouse AD hippocampus restores AHN and relevant memory deficits. Our findings corroborate the significance of AHN in AD and reveal the possible therapeutic significance of targeting miR-132 in neurodegeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200171017': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure [RNA]',\n",
       "  'summary': 'Polychlorinated biphenyls (PCBs) are developmental neurotoxicants implicated as environmental risk factors for neurodevelopmental disorders (NDD). We examined the effects of prenatal exposure to a human-relevant mixture of PCBs on the DNA methylation profiles of fetal mouse brain and placenta. We found thousands of differentially methylated regions (DMRs) distinguishing placentas and brains from PCB-exposed embryos from sex-matched vehicle controls. In both placenta and brain, PCB-associated DMRs were enriched for functions related to neurodevelopment and cellular signaling and enriched within regions of bivalent chromatin. The placenta and brain PCB DMRs overlapped significantly and mapped to a shared subset of genes enriched for Wnt signaling, Slit/Robo signaling, and genes differentially expressed in NDD models. PBC DMRs also significantly overlapped with DMRs from the brains of humans with Rett syndrome and Dup15q syndrome. These results demonstrate that placenta can be used as a surrogate for embryonic brain DNA methylation changes relevant to an NDD.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200171016': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure [WGBS]',\n",
       "  'summary': 'Polychlorinated biphenyls (PCBs) are developmental neurotoxicants implicated as environmental risk factors for neurodevelopmental disorders (NDD). We examined the effects of prenatal exposure to a human-relevant mixture of PCBs on the DNA methylation profiles of fetal mouse brain and placenta. We found thousands of differentially methylated regions (DMRs) distinguishing placentas and brains from PCB-exposed embryos from sex-matched vehicle controls. In both placenta and brain, PCB-associated DMRs were enriched for functions related to neurodevelopment and cellular signaling and enriched within regions of bivalent chromatin. The placenta and brain PCB DMRs overlapped significantly and mapped to a shared subset of genes enriched for Wnt signaling, Slit/Robo signaling, and genes differentially expressed in NDD models. PBC DMRs also significantly overlapped with DMRs from the brains of humans with Rett syndrome and Dup15q syndrome. These results demonstrate that placenta can be used as a surrogate for embryonic brain DNA methylation changes relevant to an NDD.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200169162': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal niche inflammation mediated by IL-1 signaling is a targetable driver of hematopoietic aging',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array; Expression profiling by high throughput sequencing'},\n",
       " '200168661': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of hematopoietic aging [Microarray analysis]',\n",
       "  'summary': 'We performed integrated analyses of hematopoietic stem and progenitor cells as well as bone marrow stromal cells to better understand the underlying mechanisms of blood aging. To determine how the molecular state of multipotent progenitor (MPP) cells change with age we performed transcriptomic analyses of myeloid-biased MPP3 and lymphoid-biased MPP4 populations from young and old mice.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200168613': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Inflammation of the Aged Bone Marrow Niche Drives Blood Aging',\n",
       "  'summary': 'Hematopoietic aging is defined by a loss of regenerative capacity and skewed differentiation from hematopoietic stem cells (HSC) leading to dysfunctional blood production. Signals from the bone marrow (BM) microenvironment dynamically tailor hematopoiesis, but the effect of aging on the niche and the contribution of the aging niche to blood aging still remains unclear. Here, we show the development of an inflammatory milieu in the aged marrow cavity, which drives both niche and hematopoietic system remodeling. We find decreased numbers and functionality of osteogenic endosteal mesenchymal stromal cells (MSC), expansion of pro-inflammatory perisinusoidal MSCs, and deterioration of the central marrow sinusoidal endothelium, which together create a self-reinforcing inflamed BM milieu. Single cell molecular mapping of old niche cells further confirms disruption of cell identities and enrichment of inflammatory response genes. Inflammation, in turn, drives chronic activation of emergency myelopoiesis pathways in old HSCs and multipotent progenitors, which promotes myeloid differentiation at the expense of lymphoid and erythroid commitment, and hinders hematopoietic regeneration. Remarkably, both defective hematopoietic regeneration, niche deterioration and HSC aging can be improved by blocking inflammatory IL-1 signaling. Our results indicate that targeting the pro-inflammatory niche milieu can be instrumental in restoring blood production during aging.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200168586': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of hematopoietic aging [droplet-based scRNAseq]',\n",
       "  'summary': 'To investigate global changes in the structure of the bone marrow niche with age, we performed droplet-based scRNAseq (10X Genomics) of unfractionated endosteal and central marrow stromal cells (Ter119-/CD45-). 8,735 cells passed QC, and hematopoietic clusters of cells were excluded based on marker gene expression, leaving 2359 cells distributed across 9 distinct clusters for final presentation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200167559': {'inp_pmids': ['34686734'],\n",
       "  'title': 'Dementia subtype prediction models constructed by penalized regression methods for multiclass classification using serum microRNA expression data',\n",
       "  'summary': 'There are many subtypes of dementia, and identification of diagnostic biomarkers that are minimally-invasive, low-cost, and efficient is desired. Circulating microRNAs (miRNAs) have recently gained attention as easily accessible and non-invasive biomarkers. We conducted a comprehensive miRNA expression analysis of serum samples from 1348 Japanese dementia patients, composed of four subtypes—Alzheimer’s disease (AD), vascular dementia, dementia with Lewy bodies (DLB), and normal pressure hydrocephalus—and 246 control subjects. We used this data to construct dementia subtype prediction models based on penalized regression models with the multiclass classification. We constructed a final prediction model using 46 miRNAs, which classified dementia patients from an independent validation set into four subtypes of dementia. Network analysis of miRNA target genes revealed important hub genes, SRC and CHD3, associated with the AD pathogenesis. Moreover, MCU and CASP3, which are known to be associated with DLB pathogenesis, were identified from our DLB-specific target genes. Our study demonstrates the potential of blood-based biomarkers for use in dementia-subtype prediction models. We believe that further investigation using larger sample sizes will contribute to the accurate classification of subtypes of dementia.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200165870': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Bulk RNA-seq of aplastic anemia (AA) and healthy donors',\n",
       "  'summary': 'Bulk RNA-seq data of Lin-CD34+ hematopoietic stem and progenitor cells derived from bone marrow of healthy donors and untreated aplastic anemia patients',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200164788': {'inp_pmids': ['33589615'],\n",
       "  'title': \"Machine Learning Identifies Candidates for Drug Repurposing in Alzheimer's Disease\",\n",
       "  'summary': \"Clinical trials of novel therapeutics for Alzheimer's Disease (AD) have consumed a large amount of time and resources with largely negative results. Repurposing drugs already approved by the Food and Drug Administration (FDA) for another indication is a more rapid and less expensive option. In this study, we profile 80 FDA-approved and clinically tested drugs in neural cell cultures, with the goal of producing a ranked list of possible repurposing candidates.\",\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200164174': {'inp_pmids': ['33743111'],\n",
       "  'title': 'A novel combination of serum microRNAs for the detection of early gastric cancer',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of patients with early stage gastric cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200162803': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200162802': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention [RNA-seq]',\n",
       "  'summary': 'Aging is the result of an accumulation of molecular damages, driven by multiple factors like metabolic and oxidative stress, (epi)genetic alterations, and microbiome dysbiosis. In addition, aging is characterized by a chronic, low-grade inflammation known as inflammaging that represents a high significant risk factor for most, if not all, age-related diseases. Dietary restriction (DR) is the best-known non-invasive method to ameliorate the aging phenotype in many organisms. While it has been shown that DR reduces the inflammation in some tissues, the molecular mechanisms in old individuals are still largely unknown. Here we identify a multi-tissue gene network which regulates the inflammaging and that is ameliorated by DR. Using transcriptional profiling (RNA-seq), we found that aging upregulates the innate immune system receptor together with the activation of a systemic interferon (IFN) signaling through the Interferon Regulatory Factors (Irf), inflammatory cytokines and finally Stat1 transcription factor. Our results suggest that DR is able to ameliorate inflammaging by downregulating the expression and the activity of almost all the components of this gene network. In addition, chromatin accessibility genome-wide analysis showed that the binding site of Irf and Stat1 in the promoter of their target genes are more open in aging and become closer upon DR treatment indicating a transcriptional control of the inflammaging phenotype involving epigenetic modifications. Our results provide a comprehensive understanding of the molecular network regulating the inflammation in aging and DR and present novel anti-inflammaging therapeutic targets.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200162728': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention [ATAC-seq]',\n",
       "  'summary': 'Aging is the result of an accumulation of molecular damages, driven by multiple factors like metabolic and oxidative stress, (epi)genetic alterations, and microbiome dysbiosis. In addition, aging is characterized by a chronic, low-grade inflammation known as inflammaging that represents a high significant risk factor for most, if not all, age-related diseases. Dietary restriction (DR) is the best-known non-invasive method to ameliorate the aging phenotype in many organisms. While it has been shown that DR reduces the inflammation in some tissues, the molecular mechanisms in old individuals are still largely unknown. Here we identify a multi-tissue gene network which regulates the inflammaging and that is ameliorated by DR. Using transcriptional profiling (RNA-seq), we found that aging upregulates the innate immune system receptor together with the activation of a systemic interferon (IFN) signaling through the Interferon Regulatory Factors (Irf), inflammatory cytokines and finally Stat1 transcription factor. Our results suggest that DR is able to ameliorate inflammaging by downregulating the expression and the activity of almost all the components of this gene network. In addition, chromatin accessibility genome-wide analysis showed that the binding site of Irf and Stat1 in the promoter of their target genes are more open in aging and become closer upon DR treatment indicating a transcriptional control of the inflammaging phenotype involving epigenetic modifications. Our results provide a comprehensive understanding of the molecular network regulating the inflammation in aging and DR and present novel anti-inflammaging therapeutic targets.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200159798': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200159795': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains [RNA-seq]',\n",
       "  'summary': \"Immune system aging contributes significantly to declining health with age. It is unknown to what extent human and mice immune systems go through similar age-related changes and whether there are conserved biomarkers of aging. We characterized age-related changes in the immune system of long (C57BL/6J) and short-lived (NZO/HILtJ) strains and compared them with blood-driven human aging signatures. Peripheral blood lymphocytes (PBL), spleen cells and spleen-driven naive and memory CD8+ T cells were profiled using flow cytometry, RNA-seq and ATAC-seq from young (3 months) and old (18 months) mice. Pro-inflammatory myeloid genes were activated with age across strains and tissues, echoing human 'inflammaging' signatures. ATAC-seq footprinting analyses uncovered increased binding of pro-inflammatory transcription factors with age (e.g., AP1 complex, NFKB signaling pathway). Machine learning models identified inflammation, cytotoxic, and naive T cell derived genes to be strong signatures of immunosenescence. These data are publicly shared as a resource for immune aging (https://immune-aging.jax.org/mice).\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200159794': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains [ATAC-seq]',\n",
       "  'summary': \"Immune system aging contributes significantly to declining health with age. It is unknown to what extent human and mice immune systems go through similar age-related changes and whether there are conserved biomarkers of aging. We characterized age-related changes in the immune system of long (C57BL/6J) and short-lived (NZO/HILtJ) strains and compared them with blood-driven human aging signatures. Peripheral blood lymphocytes (PBL), spleen cells and spleen-driven naive and memory CD8+ T cells were profiled using flow cytometry, RNA-seq and ATAC-seq from young (3 months) and old (18 months) mice. Pro-inflammatory myeloid genes were activated with age across strains and tissues, echoing human 'inflammaging' signatures. ATAC-seq footprinting analyses uncovered increased binding of pro-inflammatory transcription factors with age (e.g., AP1 complex, NFKB signaling pathway). Machine learning models identified inflammation, cytotoxic, and naive T cell derived genes to be strong signatures of immunosenescence. These data are publicly shared as a resource for immune aging (https://immune-aging.jax.org/mice).\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200159596': {'inp_pmids': ['35172154'],\n",
       "  'title': 'Non-coding RNAs shape cortical neurons developmental trajectories',\n",
       "  'summary': 'A hallmark of cortical evolution is the high dynamic subventricular zone (SVZ) expansion, where basal progenitors (BPs) amplify and neuronal transcriptional programs unfold. How non-coding molecular factors such as microRNAs influence these developmental trajectories and regulate the acquisition of cortical type identities is largely unknown. Here we demonstrate that miR-137 and miR-122 regulate the positioning and identity features of superficial layer cortical neurons by acting at distinct steps of their developmental trajectories. MiR-137 sustains basal progenitor amplification by reverting their neurogenic commitment and inducing high proliferative state upregulating Cd63 and inhibiting Myt1l. Cd63 is an extra-cellular matrix (ECM) receptor which interacts with \\uf061\\uf06eb3- and \\uf0621-integrin pathways to promote proliferation, while Myt1l is a transcription factor that promotes and sustains neuronal fate. The BPs amplification by miR-137 is converted in the promotion of intracortical projecting neuron (ICPN) identity and L2/3 expansion. As opposed to miR-137, miR-122 acts postmitotically, affecting the bioelectrical properties, the calcium and cytoskeleton dynamics of newborn neurons as well as their transcriptional program, leading to a persistent molecular immaturity across time. Overall, these findings reveal that miR-137 and miR-122 are key regulators of the developmental trajectory of cortical neurons across evolution.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200157145': {'inp_pmids': ['34059805'],\n",
       "  'title': 'Transcriptional and imprinting complexity in Arabidopsis seeds at single-nucleus resolution',\n",
       "  'summary': 'Seeds are the basis of agriculture, yet their full transcriptional complexity has remained unknown. Here, we employ single-nucleus RNA-sequencing to characterize developing Arabidopsis thaliana seeds, with a focus on endosperm. Endosperm, the site of gene imprinting in plants, mediates the relationship between the maternal parent and embryo. We identify new cell types in the chalazal endosperm region, which interfaces with maternal tissue for nutrient unloading. We further demonstrate that the extent of parental bias of maternally expressed imprinted genes varies with cell cycle phase, and that imprinting of paternally expressed imprinted genes is strongest in chalazal endosperm. These data indicate imprinting in endosperm is heterogeneous and suggest that parental conflict, which is proposed to drive the evolution of imprinting, is fiercest at the boundary between filial and maternal tissues.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200157027': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings exposed to different communities of commensal bacteria [SSC]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings exposed to communities of suppressors (SynCom5-S1) and non-suppressors (SynCom5-NS1) of the plant immune system',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200156426': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of commensal bacteria',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200156425': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of a community of commensal bacteria [RST]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of a 35-member bacterial SynCom',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200156424': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings exposed to the MAMP flg22 in the presence of individual commensal bacteria [MAE]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings exposed to the MAMP flg22 in the presence of 35 different bacteria',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200156212': {'inp_pmids': ['38287646'],\n",
       "  'title': 'Effects of vitamin D on non-specific inflammatory response in aged skin.',\n",
       "  'summary': 'We have previously demonstrated older skin exhibits increased sterile inflammation 6 hours after saline-injection. In this work, we examined whether vitamin D in vivo would attenuate this non-specific inflammatory response towards saline in elder individuals (≥65 years).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200149097': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200148990': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Differential gene expression in miR-146a-/- vs. WT hematopoietic stem and progenitor cells [miR146aKO_vs_WT_RNA-seq_bulk]',\n",
       "  'summary': 'To identify pathways and processes driving the observed hematopoietic stem cell (HSC) aging-like phenotypes in miR-146a-/- vs. WT, we performed RNA-seq gene expression profiling of Lin- Sca-1+ c-Kit+ (LSK) cells isolated from miR-146a-/- or WT mouse bone marrow (BM). Differential expression analysis and EnrichmentMap network analysis identified cytokine signalling and immune pathways as potential drivers of aging-like alterations in miR-146a-/- HSC proliferation and differentiation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200148989': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy [miR146aKO_LSK_WGBS]',\n",
       "  'summary': 'Aging is associated with significant changes in the hematopoietic system, including increased inflammation, impaired hematopoietic stem cell (HSC) function, and increased incidence of myeloid malignancy. Inflammation of aging (“inflammaging”) has been proposed as a driver of age-related changes in HSC function and myeloid malignancy, but mechanisms linking these phenomena remain poorly defined. Here, we identify loss of miR-146a as driving aging-associated inflammation in AML patients. miR-146a expression declined in old wild-type mice, and loss of miR-146a promoted premature HSC aging and inflammation in young miR-146a-null mice, preceding development of aging-associated myeloid malignancy. Using single-cell assays of HSC quiescence, stemness, differentiation potential, and epigenetic state to probe HSC function and population structure, we found that loss of miR-146a depleted a subpopulation of primitive, quiescent HSCs. DNA methylation and transcriptome profiling implicated NF-κB, IL6, and TNF as potential drivers of HSC dysfunction, activating an inflammatory signaling relay promoting IL6 and TNF secretion from mature miR-146a-/- myeloid and lymphoid cells. Reducing inflammation by targeting Il6 or Tnf was sufficient to restore single-cell measures of miR-146a-/- HSC function and subpopulation structure, and reduced the incidence of hematological malignancy in miR 146a-/- mice. miR-146a-/- HSCs exhibited enhanced sensitivity to IL6 stimulation, indicating that loss of miR-146a affects HSC function via both cell-extrinsic inflammatory signals and increased cell-intrinsic sensitivity to inflammation. Thus, loss of miR 146a regulates cell-extrinsic and -intrinsic mechanisms linking HSC inflammaging to the development of myeloid malignancy.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200148988': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy [miR146aKO_ESLAM_SCBS]',\n",
       "  'summary': 'Aging is associated with significant changes in the hematopoietic system, including increased inflammation, impaired hematopoietic stem cell (HSC) function, and increased incidence of myeloid malignancy. Inflammation of aging (“inflammaging”) has been proposed as a driver of age-related changes in HSC function and myeloid malignancy, but mechanisms linking these phenomena remain poorly defined. Here, we identify loss of miR-146a as driving aging-associated inflammation in AML patients. miR-146a expression declined in old wild-type mice, and loss of miR-146a promoted premature HSC aging and inflammation in young miR-146a-null mice, preceding development of aging-associated myeloid malignancy. Using single-cell assays of HSC quiescence, stemness, differentiation potential, and epigenetic state to probe HSC function and population structure, we found that loss of miR-146a depleted a subpopulation of primitive, quiescent HSCs. DNA methylation and transcriptome profiling implicated NF-κB, IL6, and TNF as potential drivers of HSC dysfunction, activating an inflammatory signaling relay promoting IL6 and TNF secretion from mature miR-146a-/- myeloid and lymphoid cells. Reducing inflammation by targeting Il6 or Tnf was sufficient to restore single-cell measures of miR-146a-/- HSC function and subpopulation structure, and reduced the incidence of hematological malignancy in miR 146a-/- mice. miR-146a-/- HSCs exhibited enhanced sensitivity to IL6 stimulation, indicating that loss of miR-146a affects HSC function via both cell-extrinsic inflammatory signals and increased cell-intrinsic sensitivity to inflammation. Thus, loss of miR 146a regulates cell-extrinsic and -intrinsic mechanisms linking HSC inflammaging to the development of myeloid malignancy.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200146264': {'inp_pmids': ['33309739'],\n",
       "  'title': 'Single cell RNA-seq of psoriatic skin identifies pathogenic Tc17 subsets and reveals distinctions between CD8+ T cells in autoimmunity and cancer',\n",
       "  'summary': 'Psoriasis is an inflammatory, IL-17-driven skin disease in which autoantigen-induced CD8+ T cells have been identified as pathogenic drivers. In this study, we used single-cell RNA-seq to identify 11 transcriptionally diverse CD8+ T cell subsets in psoriatic and healthy skin. Among several inflammatory subsets enriched in psoriatic skin, we observed two Tc17 subsets that were metabolically divergent, developmentally related, and expressed CXCL13, which we found to be a biomarker of psoriasis severity. Despite high co-inhibitory receptor expression in the Tc17 clusters, a comparison of these cells with melanoma-infiltrating CD8+ T cells revealed upregulated cytokine, cytolytic, and metabolic transcriptional activity in the psoriatic cells that differed from an exhaustion program. Our findings provide a high-resolution view of cutaneous CD8+ T cells in psoriasis and healthy skin and shed light on their functional distinctions within the chronic pathologies of autoimmunity and cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200146026': {'inp_pmids': ['32572264'],\n",
       "  'title': 'single cell RNA-seq analysis of ovarian cancer ascites',\n",
       "  'summary': 'To understand the cellular diversity within malignant ascites, we profiled ~11,000 single cells from 22 samples by Smartseq2 and by 10x.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200145669': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Transcriptome analysis of HSPCs and T cells from aplastic anemia (AA) and healthy donors',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200145668': {'inp_pmids': ['33763704'],\n",
       "  'title': \"Single cell transcriptome from aplastic anemia (AA) and healthy donors [3' RNA-seq]\",\n",
       "  'summary': 'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) and T cells collected from bone marrow and peripheral blood of healthy donors and aplastic anemia patients untreated or response to immunosuppressive therapy.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200145531': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Single cell transcriptome from aplastic anemia (AA) and healthy donors [Full-length RNA-seq]',\n",
       "  'summary': 'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) from aplastic anemia (AA) and healthy donors',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200142259': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem [protophloem sieve element cells]',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200140977': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem II',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200140778': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'blank sample; Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200140493': {'inp_pmids': ['35419551'],\n",
       "  'title': 'Multi-omic profiling of single nuclei uncovers regulatory diversity of brain cell types and diseases',\n",
       "  'summary': 'Most existing single-cell techniques can only make one type of molecular measurements. Although computational approaches have been developed to integrate single-cell datasets, their efficacy still needs to be determined with reference to authentic single-cell multi-omic profiles. To address this challenge, we devised single-nucleus methylCytosine, Chromatin accessibility and Transcriptome sequencing (snmC2T-seq) and applied the approach to post-mortem human frontal cortex tissue. We developed a computational framework to evaluate the quality of finely defined cell types using multi-modal information and validated the efficacy of computational multi-omic integration methods. Correlation analysis in individual cells revealed gene groups showing distinct relations between methylation and expression. Integration of snmC2T-seq with other multi- and single- modal datasets enabled joint analyses of the methylome, chromatin accessibility, transcriptome, and chromatin architecture for 63 human cortical cell types. We reconstructed the regulatory lineage of these cortical cell types and found pronounced cell-type-specific enrichment of disease risks for neuropsychiatric traits, predicting causal cell types that can be targeted for treatment.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing'},\n",
       " '200137444': {'inp_pmids': ['38539015'],\n",
       "  'title': 'Stem cell derived human microglia transplanted in mouse brain to study human disease',\n",
       "  'summary': 'While genetics highlight the role of microglia in Alzheimer’s disease (AD), one third of putative AD-risk genes lack adequate mouse orthologs. Here, we successfully engraft human microglia derived from embryonic stem cells in the mouse brain. The cells recapitulate transcriptionally human primary microglia ex vivo and show expression of human specific AD-risk genes. Oligomeric Amyloid-β induces a divergent response in human vs. mouse microglia. This model can be used to study the role of microglia in neurological diseases.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200134646': {'inp_pmids': ['37958987'],\n",
       "  'title': 'Exploring the effects of rearing densities on epigenetic modifications in the zebrafish gonads',\n",
       "  'summary': 'The goal of this study was to determine whether there was a direct relationship between stress and DNA methylation changes in fish gonads. During sex differentiation, we studied the sensitive window to density. Sex ratios were studied and methylation of canonical reproduction-related genes was analysed.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200133922': {'inp_pmids': ['31801092'],\n",
       "  'title': 'Global transcriptomic profiling of the bone marrow stromal microenvironment during postnatal development, aging, and inflammation',\n",
       "  'summary': 'RNA-sequencing of a longitudinal comparison of four principal mesenchymal and endothelial stromal cell types (CXCL12-abundant reticular cells, PDGFR-α+ Sca-1+, sinusoidal and arterial endothelial cells), isolated from early postnatal, adult and aged mice. Additional transcriptional profiling of the response of CXCL12-abundant reticular cells and sinusoidal endothelial cells to infection-mimicking agents.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200130711': {'inp_pmids': ['31501549', '35419551'],\n",
       "  'title': 'Simultaneous profiling of 3D genome structure and DNA methylation in single human cells',\n",
       "  'summary': 'Dynamic 3D chromatin conformation is a critical mechanism for gene regulation during development and disease. Despite this, profiling of 3D genome structure from complex tissues with cell-type specific resolution remains challenging. Recent efforts have demonstrated that cell-type specific epigenomic features can be resolved in complex tissues using single-cell assays. However, it remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles can effectively identify cell types and reconstruct cell-type specific chromatin conformation maps. To address these challenges, we have developed single-nucleus methyl-3C sequencing (sn-m3C-seq) to capture chromatin organization and DNA methylation information and robustly separate heterogeneous cell types. Applying this method to >4,200 single human brain prefrontal cortex cells, we reconstruct cell-type specific chromatin conformation maps from 14 cortical cell types. These datasets reveal the genome-wide association between cell-type specific chromatin conformation and differential DNA methylation, suggesting pervasive interactions between epigenetic processes regulating gene expression.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other'},\n",
       " '200130633': {'inp_pmids': ['38287646'],\n",
       "  'title': 'Effects of losmapimod on antigen-specific response in aged skin',\n",
       "  'summary': 'Older humans are more susceptible to infections, even those to which they were immune to in their youth. Vaccination against many different infectious agents is also sub-optimal in these individuals. To investigate the mechanisms involved in the decline in immunity during ageing we inject safe derivatives of microorganisms into the skin of old and young volunteers. After injection of the agent, we can extract the white cells (leucocytes) that accumulate at the site of the immune response to test for their function. One of the responses we have been investigating is that to the varicella zoster virus (VZV) that induces chickenpox mainly in young individuals. Infected subjects control this persistent virus until an older age where reduced immunity leads to viral re-activation to cause shingles. We have found that this defective response is associated with increased baseline inflammation in this tissue. Although inflammatory responses are required to clear infections, excessive inflammation has previously been shown to interfere with the generation of specific immunity. We will therefore investigate ways of enhancing the immune response of older humans by blocking basal inflammation. To extend our initial observations of high background inflammation in the skin, we will investigate which cells in this tissue are responsible for the production of inflammatory mediators by using a second human experimental system that tests the response to a skin irritant known as cantharadin. Next we will block inflammation in the skin to determine whether this can lead to improved responses upon challenge with antigens. This will be achieved by pre-treating old subjects with a drug that has already been developed by GSK (Losmapimod, p38MAPkinase inhibitor) and another that is available off the shelf (vitamin D3). The GSK p38 inhibitors are currently being tested to prevent unwanted inflammation. A surprising observation we made was that p38 inhibition could also rejuvenate human T lymphocytes and enhance their ability to proliferate in vitro. Therefore the inhibition of p38 may block unwanted inflammatory response as well as enhance T lymphocyte reactivity. Vitamin D3 has also been shown to have anti-inflammatory effects and works in part by inhibiting p38MAPkinase activation. The use of 2 separate interventions in this project increases the likelihood of success in this study. While Losmapimod may be more specific in its anti-inflammatory action, the use of vitamin D3 is more cost effective as it is a cheap compound that is readily available. The desirable outcome of this study is that either one or other of these compounds, that older volunteers will be treated with, will boost their response to microbial antigen challenge in the skin. This will provide proof of concept data that will lead to the exciting possibility of enhancing immunity by inhibiting the increased baseline inflammatory responses that are found during ageing. This may be a strategy to enhance immune responses in general and specifically to enhance vaccination efficacy to various diseases that is less effective in older subjects.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200130051': {'inp_pmids': ['31076851'],\n",
       "  'title': 'Second-generation molecular subgrouping of medulloblastoma: an international meta-analysis of Group 3 and Group 4 subtypes',\n",
       "  'summary': 'We undertook a meta-analysis based on DNA methylation patterns of Group3/4 subgroup medulloblastoma from three published studies, alongside additional unpublished tumours (total n=1501).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200127893': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200127892': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (KW)',\n",
       "  'summary': 'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200127884': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (CDEP)',\n",
       "  'summary': 'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200127711': {'inp_pmids': ['31792364'],\n",
       "  'title': 'Blood Biomarkers for Memory: Towards Early Detection of Risk for Alzheimer Disease, Pharmacogenomics, and Repurposed Drugs',\n",
       "  'summary': 'Short-memory dysfunction is a key early feature of Alzheimer Disease. Psychiatric patients may be at higher risk for memory dysfunction and subsequent Alzheimer due to the negative effects of stress and depression on the brain. We carried out longitudinal within-subject studies in male and female psychiatric patients to identify blood gene expression biomarkers that track short term memory as measured by the retention measure in the Hopkins Verbal Learning Test. These biomarkers were prioritized with a convergent functional genomics approach using previous evidence in the field implicating them in Alzheimer Disease. The top candidate biomarkers were tested in an independent cohort for ability to predict state short-term memory, and trait future positive neuropsychological testing for cognitive impairment',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200124391': {'inp_pmids': ['31501549'],\n",
       "  'title': 'Single-cell multi-omic profiling of chromatin conformation and DNA methylome',\n",
       "  'summary': 'Recent advances in the development of single cell epigenomic assays have facilitated the analysis of the gene regulatory landscapes in complex biological systems. Single-cell variations of methods such as DNA methylation-sequencing and ATAC-seq hold tremendous promise for delineating distinct cell types and identifying their critical cis-regulatory sequences. Emerging evidence in recent years has shown that in addition to cis-regulatory sequences, dynamic regulation of 3D chromatin conformation is a critical mechanism for the modulation of gene expressions during development and disease. While assays for the investigation of single-cell 3D chromatin structure have been developed, cell-type specific chromatin conformation in primary human tissues has not been extensively explored. It remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles are suitable for cell type identification and allow the reconstruction of cell-type specific chromatin conformation maps. To address these challenges, we have developed a multi-omic method single-nucleus methyl-3C sequencing (sn-m3C) to profile chromatin conformation and DNA methylation from the same cell. We have shown that bulk m3C and sn-m3C accurately capture chromatin organization information and robustly separate mouse cell types. We have developed a fluorescent-activated nuclei sorting strategy based on DNA content that eliminates nuclei multiplets caused by crosslinking. The sn-m3C-seq method allows high-resolution cell-type classification using two orthogonal types of epigenomic information and the reconstruction of cell-type specific chromatin conformation maps.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other'},\n",
       " '200122497': {'inp_pmids': ['31125107'],\n",
       "  'title': 'Large-scale serum microRNA profiling for developing an esophageal squamous cell carcinoma detection model',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of esophageal squamous cell carcinoma.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200121750': {'inp_pmids': ['32084358'],\n",
       "  'title': 'Functional heterogeneity within the developing zebrafish epicardium',\n",
       "  'summary': 'The epicardium is essential during cardiac development, homeostasis and repair and yet fundamental insights into its underlying cell biology, notably epicardium formation, lineage heterogeneity and functional cross-talk with other cell types in the heart, is currently lacking. In this study, we investigated epicardial heterogeneity and the functional diversity of discrete epicardial subpopulations in the developing zebrafish heart. Single-cell RNA-sequencing uncovered three epicardial subpopulations with specific genetic programmes and distinctive spatial distribution within the developing heart. Perturbation of unique gene signatures uncovered distinct functions associated with each subpopulation and established novel epicardial roles in cell adhesion, migration, and chemotaxis as a mechanism for recruitment of leukocytes into the heart. This work elucidates the mutual spatiotemporal relationships between different epicardial subpopulations and assigns unique function to each during cardiac development. Understanding which mechanisms cells employ to establish a functional epicardium and to communicate with other cardiovascular cell types during development will bring us closer to repairing cellular relationships that are disrupted during cardiovascular disease.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200120584': {'inp_pmids': ['30820472', '31666070', '34686734'],\n",
       "  'title': 'Risk prediction models for dementia constructed by supervised principal component analysis using miRNA expression data',\n",
       "  'summary': 'Alzheimer’s disease (AD) is the most common subtype of dementia, followed by Vascular Dementia (VaD), and Dementia with Lewy Bodies (DLB). Recently, microRNAs (miRNAs) have received a lot of attention as the novel biomarkers for dementia. Here, using serum miRNA expression of 1,601 Japanese individuals, we investigated potential miRNA bio- markers and constructed risk prediction models, based on a supervised principal component analysis (PCA) logistic regression method, according to the subtype of dementia. The final risk prediction model achieved a high accuracy of 0.873 on a validation cohort in AD, when using 78 miRNAs: Accuracy = 0.836 with 86 miRNAs in VaD; Accuracy = 0.825 with 110 miRNAs in DLB. To our knowledge, this is the first report applying miRNA-based risk pre- diction models to a dementia prospective cohort. Our study demonstrates our models to be effective in prospective disease risk prediction; and with further improvement may contribute to practical clinical use in dementia.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200118723': {'inp_pmids': ['31002671'],\n",
       "  'title': 'Discovery and characterization of variance QTLs in human induced pluripotent stem cells',\n",
       "  'summary': 'Quantification of gene expression levels at the single cell level has revealed that gene expression can vary substantially even across a population of homogeneous cells. However, it is currently unclear what genomic features control variation in gene expression levels, and whether common genetic variants may impact gene expression variation. Here, we take a genome-wide approach to identify expression variance quantitative trait loci (vQTLs). To this end, we generated single cell RNA-seq (scRNA-seq) data from induced pluripotent stem cells (iPSCs) derived from 53 Yoruba individuals. We collected data for a median of 95 cells per individual and a total of 5,447 single cells, and identified 241 mean expression QTLs (eQTLs) at 10% FDR, of which 82% replicate in bulk RNA-seq data from the same individuals. We further identified 14 vQTLs at 10% FDR, but demonstrate that these can also be explained as effects on mean expression. Our study suggests that dispersion QTLs (dQTLs), which could alter the variance of expression independently of the mean, have systematically smaller effect sizes than eQTLs. We estimate that at least 300 cells per individual and 400 individuals would be required to have modest power to detect the strongest dQTLs in iPSCs. These results will guide the design of future studies on understanding the genetic control of gene expression variance.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Other'},\n",
       " '200117064': {'inp_pmids': ['31136284'],\n",
       "  'title': 'Serum miRNA-based risk prediction for stroke',\n",
       "  'summary': 'The serum miRNAs were correlated with predicted the incidence of stroke.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200116672': {'inp_pmids': ['30530648', '31820734'],\n",
       "  'title': 'In vivo molecular signatures of severe dengue infection revealed by viscRNA-Seq',\n",
       "  'summary': 'Dengue virus infection can result in severe symptoms including shock and hemorrhage, but an understanding of the molecular correlates of disease severity is lacking. Bulk transcriptomics on blood samples are difficult to interpret because the blood is composed of different cell types that may react differently to virus infection. Dengue virus RNA can be detected in human plasma, however identifying the cells carrying dengue virus through the bloodstream in vivo has proven challenging. Here we used our recently developed viscRNA-Seq approach to profile transcriptomes of thousands of single blood peripheral mononuclear cells from 6 human subjects with dengue fever and severe dengue, as well as to characterize the cell types associated with dengue virus in the human blood. We found that although no bulk transcriptome marker for severe dengue exists, the expression of MX2 in naive B cells, of CD163 in CD14+/CD16+ monocytes and of other genes in specific cell types is highly predictive for severe dengue. We detected virus-associated cells in the blood of two severe dengue patients with high viral load and discovered the majority of these to be B cells expressing germline IgM or IgD immunoglobulin chains and naive markers but also showing signs of activation and expression of CD69, CXCR4, and other surface receptors. In bystander B cells we detected signs of strong immune activation, parallel hypersomatic evolution and, in one severe degue subject, an anomalously large clone of highly mutated, IgG1 plasmablasts that could be reactive to dengue virus. This study presents a high-resolution molecular exploration into dengue virus infection in humans and can be generalized to any RNA virus.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200115513': {'inp_pmids': ['26740022'],\n",
       "  'title': 'miRNA and colorectal cancer: associations with tumor phenotype and survival',\n",
       "  'summary': 'MicroRNAs (miRNA) are a class of small regulatory RNAs that mediate post-transcriptional silencing of specific target mRNAs. Data suggest the importance of miRNAs to cancer development and possibly to survival. Our overall hypothesis is that miRNA expression is unique to tumor molecular phenotype; that miRNA expression levels at time of diagnosis predicts survival; and that miRNA expression is associated with inflammation-related genetic and lifestyle factors key to colorectal cancer (CRC). This study takes a two pronged approach to addressing our hypotheses. While we propose to validate previously identified miRNAs that have been identified as associated with CRC (either by differential expression or from assessment of mutations), we will add to the field through discovery of new and important associations that may be unique to specific molecular phenotypes, to polyp to cancer progression, and to survival. We have analyzed the expression of 2006 human miRNAs using data derived from tumor and paired normal tissue at time of diagnosis from: 1975 people with incident colon cancer or rectal cancer and 290 polyps from colon and rectal cases (included in this study) who reported a prior polyp. MiRNA was obtained from dissected paraffin-embedded tissue and assessed using an Agilent microarray platform. We intend to extend our validation of previously identified mutated miRNAs and differentially expressed miRNAs to determine if these alterations are associated with specific tumor molecular phenotype (TP53, KRAS2, CIMP+, and for colon tumors MSI+), inflammation-related factors, clinical factors and survival. We will identify associations with miRNAs that are related to specific molecular phenotypes, with drivers in the carcinogenic process, and with clinical features and survival. These miRNAs will be validated using targeted Agilent Platform. Associations will be tested based on differential expression for both individual and groups of miRNAs using recent extensions of several statistical methods including ANOVA, logistic regression, and Cox proportional hazards models. Our sample size allows for both a training and validation component, and provides sufficient statistical power to meet the study goals. MiRNAs that are differentially expressed in polyps and in subsequent tumors will provide new insights into targets for screening and treatment as well as provide support that miRNAs function as the “driver” in the carcinogenic process. Testing of mutated miRNAs identified from sequencing in conjunction with tumor phenotype, clinical, and survival data will further validate the importance of these miRNAs, and provide insight as to which CRC molecular pathway the miRNAs function. Our rich dataset of lifestyle, genetic, clinical and prognosis, and tumor molecular phenotype on 1975 CRC and paired normal tissue allows us to examine factors that are associated with miRNA expression and mutation in a large sample of population-based cases in the most cost-efficient manner possible. The miRNAs identified in these analyses will elucidate pathways important in the etiology of CRC and will provide insight into potential targets for screening and treatment. MiRNAs are small, non-protein-coding RNA molecules that regulate gene expression either by post-transcriptionally suppressing mRNA translation or by mRNA degradation. We examine differentially expressed miRNAs in colorectal carcinomas, adenomas, and normal colonic mucosa. Data come from population-based studies of colorectal cancer conducted in Utah and the Kaiser Permanente Medical Care Program. A total of 1893 carcinoma/normal paired samples and 290 adenoma tissue samples were run on the Agilent Human miRNA Microarray V19.0 which contained 2006 miRNAs. We tested for significant differences in miRNA expression between paired carcinoma/adenoma/normal colonic tissue samples. Fewer than 600 miRNAs were expressed in >80% of people for colonic tissue; of these 86.5% were statistically differentially expressed between carcinoma and normal colonic mucosa using a False Discovery Rate of 0.05. Roughly half of these differentially expressed miRNAs showed a progression in levels of expression from normal to adenoma to carcinoma tissue. Other miRNAs appeared to be altered at the normal to adenoma stage, while others were only altered at the adenoma to carcinoma stage or only at the normal to carcinoma stage. Evaluation of the Agilent platform showed a high degree of repeatability (r=0.98) and reasonable agreement with the NanoString platform. Our data suggest that miRNAs are highly dysregulated in colorectal tissue among individuals with colorectal cancer; the pattern of disruption varies by miRNA as tissue progresses from normal to adenoma to carcinoma.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200113740': {'inp_pmids': ['32025611'],\n",
       "  'title': 'High-sensitive circulating microRNA panel for early detection of hepatocellular carcinoma',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of hepatocellular carcinoma. Keywords: Non-coding RNA profiling by array',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200109159': {'inp_pmids': ['29576475'],\n",
       "  'title': 'Smart-seq2 analysis of larval zebrafish habenula from the gng8-GFP transgenic line',\n",
       "  'summary': 'The identification of cell types and marker genes is critical for dissecting neural development and function, but the size and complexity of the brain has hindered the comprehensive discovery of cell types. We combined single-cell RNA-seq with anatomical brain registration to create a comprehensive map of the zebrafish habenula, a conserved forebrain hub involved in pain processing and learning. Single-cell transcriptomes of ~13000 habenular cells (>4x coverage) identified 18 neuronal types and dozens of marker genes. Registration of marker genes onto a common reference atlas created a rich resource for anatomical and functional studies and enabled the mapping of active neurons onto neuronal types following aversive stimuli. Strikingly, despite brain growth and functional maturation, cell types were retained between the larval and adult habenula. This study provides a gene expression atlas to dissect habenular development and function and offers a general framework for the comprehensive characterization of other brain regions.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200109158': {'inp_pmids': ['29576475'],\n",
       "  'title': '10X genomics analysis of adult and larval habenula from the gng8-GFP transgenic line',\n",
       "  'summary': 'The identification of cell types and marker genes is critical for dissecting neural development and function, but the size and complexity of the brain has hindered the comprehensive discovery of cell types. We combined single-cell RNA-seq with anatomical brain registration to create a comprehensive map of the zebrafish habenula, a conserved forebrain hub involved in pain processing and learning. Single-cell transcriptomes of ~13000 habenular cells (>4x coverage) identified 18 neuronal types and dozens of marker genes. Registration of marker genes onto a common reference atlas created a rich resource for anatomical and functional studies and enabled the mapping of active neurons onto neuronal types following aversive stimuli. Strikingly, despite brain growth and functional maturation, cell types were retained between the larval and adult habenula. This study provides a gene expression atlas to dissect habenular development and function and offers a general framework for the comprehensive characterization of other brain regions.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200106817': {'inp_pmids': ['30333487', '39587714'],\n",
       "  'title': 'Integrated extracellular microRNA profiling for ovarian cancer screening',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detEsophageal Cancertion of patients with ovarian cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200105115': {'inp_pmids': ['29576475'],\n",
       "  'title': 'Comprehensive identification and spatial mapping of habenular neuronal types using single cell RNAseq',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200103708': {'inp_pmids': ['30333487'],\n",
       "  'title': 'The profile of exosomal and cellular miRNA in ovarian cancer cell lines',\n",
       "  'summary': 'To check the profile of exosomal and cellular miRNA in ovarian cancer cell lines, total RNA were extracted from exosomes and cells. Thirteen ovarian cancer cell lines (A2780, ES-2, CAOV3, SKOV3, OV-90, OAW42, MCAS, COV362, RMG-1, RMUG-S, KURAMOCHI, NIH-OVCAR3 and A2780cis) were investigated, and HOSE1, HOSE2 and HOSE3 (human ovarian surface epithelim cell lines) were used as control.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200102248': {'inp_pmids': ['29462153'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings growing in the presence of a bacterial synthetic community under full and low phosphate conditions',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings growing in the presence of a diversity of synthetic bacteria communities under different phosphate availability.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200099962': {'inp_pmids': ['29550329'],\n",
       "  'title': 'The biological age linked with oxidative estress modifies breast cancer aggressiveness',\n",
       "  'summary': 'The incidence of breast cancer increases with age until menopause, and breast cancer is more aggressive in younger women. The existence of epidemiological links between breast cancer and aging indicates that both processes possibly share some common mechanisms of development. Oxidative stress is associated with both cancer susceptibility and aging. Here we observed that ERBB2-positive breast cancer, developed from genetically different ERBB2-positive transgenic mice generated by a backcross, is more aggressive in younger than in older mice, similar to what occurs in humans. In this cohort, we estimated the oxidative biological age using a mathematical model that integrated several subphenotypes directly or indirectly related to oxidative stress. This model included the serum levels of HDL-cholesterol and magnesium that were determined at a disease-free stage, and total AKT1 and glutathione concentrations in the liver at the time of necropsy. Later we defined the grade of aging due to oxidative stress as the increment of aging, which was the difference between the predicted biological age and the chronological age. This comparison permitted the identification of the biologically younger mice with less oxidative stress and older mice with more oxidative stress than would be expected according to their chronological age. Interestingly, biologically older mice developed more aggressive breast cancer than the biologically younger mice. We identified genomic regions on chromosomes 2 and 15 that were linked to the grade of oxidative aging. The levels of expression of Zbp1 located on chromosome 2, a gene related to necroptosis and inflammation, positively correlated with the grade of oxidative aging and tumour aggressiveness. This study shows part of the complex interactions between breast cancer and aging.',\n",
       "  'organism': 'Mus musculus; Mus',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200097693': {'inp_pmids': ['30498128'],\n",
       "  'title': 'Single-cell Multi-omics Sequencing and Analyses of Human Colorectal Cancer',\n",
       "  'summary': 'Although genomic instability, epigenetic abnormality, and gene expression dysregulation are hallmarks of colorectal cancer, these features have not been simultaneously analyzed at single-cell resolution. Using optimized single-cell multi-omics sequencing together with multi-regional sampling of the primary tumor, lymphatic and distant metastases, we provide insights beyond intratumoral heterogeneity. Genome-wide DNA methylation levels were relatively consistent within a single genetic sub-lineage. The genome-wide DNA demethylation patterns of cancer cells were consistent in all 10 sequenced patients. Our work demonstrates the feasibility of reconstructing genetic lineages, and tracing their epigenomic and transcriptomic dynamics with single-cell multi-omics sequencing.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing'},\n",
       " '200097179': {'inp_pmids': ['35419551'],\n",
       "  'title': 'Single Cell Methylome Signatures Reveal Distinct Neuronal Populations and Regulatory Elements in the Mammalian Cortex',\n",
       "  'summary': 'The mammalian brain contains diverse neuron cell types with distinct physiological, morphological and molecular characteristics. However, a comprehensive assessment of the epigenetically distinct neuronal classes is currently missing. Cytosine DNA methylation is a stable epigenetic mark that distinguishes neuron types and marks gene regulatory elements. We developed an efficient single-nucleus methylome sequencing approach that allows robust high-throughput neuron-type classification. We generated >6,000 single-nucleus methylomes and identified 16 mouse and 21 human neuronal subpopulations in the frontal cortex. Both CG and non-CG methylation exhibited cell type-specific distributions that recapitulate and extend findings from single neuron transcriptome profiling. Moreover, we found approximately 500,000 neuron-type-specific regulatory elements showing strong differential methylation in mouse and human cortex. Distinct methylation signatures identified an unique human Parvalbumin-expressing inhibitory sub-type and a layer 6 specific excitatory sub-type in mouse. Comparative epigenomic analysis showed stronger conservation of gene regulatory elements in inhibitory compared with excitatory neurons. These findings demonstrate the utility of single nucleus methylome profiling for both expanding the atlas of brain cell types and identifying regulatory elements that potentially drive these differences.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other'},\n",
       " '200096066': {'inp_pmids': ['30320226'],\n",
       "  'title': 'Probing Plasmodium falciparum sexual differentiation at the single cell level',\n",
       "  'summary': 'Malaria parasites go through major transitions during their complex life cycle, yet the underlying differentiation pathways remain obscure. Here we apply single cell transcriptomics to unravel events that initiate sexual development in preparation for transmission of the parasite from human to mosquito. This proof-of-concept study provides a template to capture transcriptional diversity in heterogeneous parasite populations, with major implications for our understanding of parasite biology and the ongoing malaria elimination campaign.',\n",
       "  'organism': 'Plasmodium falciparum NF54',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200088798': {'inp_pmids': ['29794063'],\n",
       "  'title': 'Time-resolved transcriptome analysis with genetic perturbations reveals a critical time window for effective plant immunity',\n",
       "  'summary': 'Pathogen invasion in plants is associated with transcriptional reprogramming. Enigmatically, plants induce similar transcriptome responses upon infection by virulent or avirulent pathogens. This renders the importance of transcriptional reprogramming for immunity obscure. Here, using RNA-seq, we generate time-series transcriptome data coupled with genetic perturbations to reveal temporal dynamics upon infection by virulent or avirulent strains of a bacterial pathogen, Pseudomonas syringae, in Arabidopsis thaliana. Fast and sustained transcriptional reprogramming occurs upon infection with avirulent strains while virulent strain infection leads to a slower response with comparable gene expression patterns and magnitudes. Importantly, transcriptome analysis of resistant and susceptible mutants responding to avirulent strains links delayed transcriptional reprogramming to compromised immunity. Taken together, our results pinpoint the early critical time window of transcriptional reprogramming for establishing effective immunity against the bacterial pathogen.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200087162': {'inp_pmids': ['30322904'],\n",
       "  'title': 'RNA-seq of Solanum lycopersicum cv. M82 and S. pennellii introgression line 3-5 day old root tip tissue',\n",
       "  'summary': 'The goal of this study was to perform RNA-seq expression analysis on Solanum lycopersicum cv. M82 X S. pennellii introgression lines, deriving expression Quantitative Trait Loci which were analyzed together with pre-existing genomic and phenotypic data to define genes and regulatory pathways controlling tomato root development and observed natural variation. We completed the RNAseq expression profiling analysis and developed a tool to display this information graphically in collaboration with Nicholas Provart at the University of Toronto: http://bar.utoronto.ca/efp_tomato/cgi-bin/efpWeb.cgi?dataSource=ILs_Root_Tip_Brady_Lab To identify candidate genes and pathways we focussed on one root growth trait, root growth angle, and identified two statistically significant genomic regions within tomato root growth angle QTL containing two candidate genes that likely control the gravitropic setpoint angle (CDC73 and PAP27), both of which are conserved between Arabidopsis and tomato, and which we tested using transgenic lines of the Arabidopsis orthologs. A possible regulatory role for suberin in root growth angle control was also identified.',\n",
       "  'organism': 'Solanum lycopersicum; Solanum lycopersicum x Solanum pennellii; Solanum pennellii',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200084422': {'inp_pmids': ['27799057'],\n",
       "  'title': \"Molecular Signatures Underlying Selective Regional Vulnerability to Alzheimer's Disease\",\n",
       "  'summary': \"Alzheimer's disease (AD) is the most common form of dementia, characterized by progressive cognitive impairment and neurodegeneration as a result of abnormal neuronal loss. To elucidate the molecular systems associated with AD, we characterized the gene expression changes associated with multiple clinical and neuropathological traits in 1,053 postmortem brain samples across 19 brain regions from 125 persons dying with varying severities of dementia and variable AD-neuropathology severities.\",\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200080744': {'inp_pmids': ['35440059'],\n",
       "  'title': 'Epigenomic and genome structural diversity in a worldwide collection of Arabidopsis thaliana',\n",
       "  'summary': 'Epigenetic variation can impact gene transcription and may play roles in phenotypic diversity and adaptation. Here we report 1,107 high quality single-base resolution methylomes, and 1,210 transcriptomes from the 1001 Arabidopsis Genomes population. Analyses reveal strong effects of geographic origin on average DNA methylation levels, alterations of gene expression by epialleles and a highly complex genetic basis for DNA methylation. Physical genome maps for nine of the most diverse accessions revealed how transposable elements and other structural variations shaped the epigenome to allow rapid adaptation to environmental changes, with strong emphasis on disease resistance. Analysis of the cistromes and epicistromes in these accessions revealed a significant association between both methylation and nucleotide variation and the conservation of transcription factor binding sites. The Arabidopsis thaliana 1001 Epigenomes Project now provides a comprehensive resource to help further understand how epigenetic variation contributes to both molecular and phenotypes in natural populations of the most widely studied reference plant.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200076381': {'inp_pmids': ['27716510'],\n",
       "  'title': 'Single Cell RNA-seq Study of Midbrain and Dopaminergic Neuron Development in Mouse, Human, and Stem Cells',\n",
       "  'summary': 'In order to get a better molecular understanding of human midbrain development, this study defines cell types of the ventral midbrain in both human and mouse as well as stem cell-derived dopaminergic neuron preparations, and reveals the temporal dynamics of key lineages across development and the adult.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200073002': {'inp_pmids': ['26749252'],\n",
       "  'title': 'A novel combination of serum microRNAs for detecting breast cancer in the early stage',\n",
       "  'summary': 'MicroRNAs (miRNAs), which are stably present in serum, have been reported to be potentially useful for detecting cancer. In the present study, we examined the expression profiles of serum miRNAs in large cohorts to identify the miRNAs that can be used to detect breast cancer in the early stage. We comprehensively evaluated serum miRNA expression profiles using highly sensitive microarray analysis. A total of 1,280 serum samples of breast cancer patients stored in the National Cancer Center Biobank were used. Additionally, 2,836 serum samples were obtained from non-cancer controls and 514 from patients with other types of cancers or benign diseases. The samples were divided to a training cohort including non-cancer controls, other cancers and breast cancer and a test cohort including non-cancer controls and breast cancer. The training cohort was used to identify a combination of miRNAs that detect breast cancer, and the test cohort was used to validate that combination. miRNA expression was compared between breast cancer and non-breast cancer serum , and a combination of five miRNAs (miR-1246, miR-1307-3p, miR-4634, miR-6861-5p, and miR-6875-5p) was found to detect breast cancer. This combination had a sensitivity of 97.3%, specificity of 82.9%, and accuracy of 89.7% for breast cancer in the test cohort Additionally, the combination could detect breast cancer in the early stage (sensitivity of 98.0% for T0).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array'},\n",
       " '200072698': {'inp_pmids': ['26566685'],\n",
       "  'title': 'Site-specific methylation and acetylation of lysine residues in the C-terminal domain (CTD) of RNA polymerase II',\n",
       "  'summary': 'Dynamic modification of the carboxy-terminal domain (CTD) of RNA polymerase II (RNAPII) regulates transcription-coupled processes in eukaryotes. The CTD in mammals is composed of 52 heptad-repeats with the consensus sequence Y1-S2-P3-T4-S5-P6-S7. Repeats in the distal part of CTD deviate from the consensus sequence and have frequently replaced serine at position 7 by lysine residues (K7). Mass spectrometry analysis revealed modification of K7 residues in heptad-repeats 39, 42, and 47/49 by acetylation and in heptad-repeats 38, 39, 40, 42, and 47/49 by mono-, di-, or trimethylation. Notably, acetylated as well as di- and tri-methylated K7 residues were found exclusively in phosphorylated CTD peptides, while mono-methylated K7 residues occurred also in non-phosphorylated CTD peptides. Methylation of K7 residues was further studied with the monoclonal antibody (mAb) 1F5, which recognizes mono- and di-methylated K7 in CTD. ChIP experiments revealed high levels of K7 methylation at the transcriptional start site of genes. The low levels of K7 methylation in the body of genes results from impairment of epitope recognition in hyper-phosphorylated CTD by mAb 1F5. In agreement with this notion, phosphatase treatment of hyper-phosphorylated CTD or treatment of cells with kinase inhibitor flavopiridol restored the reactivity of mAb 1F5 towards methylated K7 residues in CTD. We conclude that methylation and acetylation of K7 residues further expand the mammalian CTD code and potentially contribute to regulation of gene expression.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200066017': {'inp_pmids': ['25939354'],\n",
       "  'title': 'Effect of Chromomethylase-2 and Dicer-like-3 knockouts on DNA methylation',\n",
       "  'summary': 'Whole genome bisulfite sequencing was performed on Chromomethylase-2 and Dicer-like-3 knockout mutants in order to confirm the results from genome wide association mapping and to identify the respective genomic regions that they target.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200065685': {'inp_pmids': ['25939354'],\n",
       "  'title': 'Population epigenomics of Swedish Arabidopsis',\n",
       "  'summary': 'Plants display remarkable developmental and phenotypic plasticity in order to adapt to their environment. It has long been postulated that epigenetics plays a key role in these processes, but with one or two exceptions, solid evidence for the role of epigenetic variation in these processes is lacking. A key impediment to understanding these processes is the lack of information on the extent of epigenetic variation and how it relates to genetic and phenotypic variation in natural population, both over the lifecycle of an individual, and over evolutionary time. Here we show that genetic variants under selection in the north of Sweden appear to drive variation in DNA methylation, which in turn is highly correlated with local climate. Selective sweeps and genetic variants associated with adaptation to the local environment have previously been identified within the Swedish Arabidopsis population. Our finding that they harbour variants responsible for climate associated epigenetic variation strongly supports the role of epigenetic processes in local adaptation. These findings provide a basis for further dissecting the role of epigenetics in local adaptation at the molecular level',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200063945': {'inp_pmids': ['26280576'],\n",
       "  'title': 'Reinstating transcriptome plasticity and memory function in mouse models for cognitive decline',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing; Non-coding RNA profiling by high throughput sequencing'},\n",
       " '200063944': {'inp_pmids': ['26280576'],\n",
       "  'title': 'small RNA-Seq of the CA1 hippocampal subregion from 3 month and 20 month old animals treated orally with vehicle or SAHA',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing'},\n",
       " '200063943': {'inp_pmids': ['26280576'],\n",
       "  'title': 'mRNA-Seq of the CA1 hippocampal subregion and liver from 3 month and 20 month old animals treated orally with vehicle or SAHA and mRNA-Seq of CA1 of 10 month old WT or APP/PS1-21 transgenic animals treated orally with vehicle or SAHA',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200063942': {'inp_pmids': ['26280576'],\n",
       "  'title': 'Genome-wide analysis of H4K12 acetylation in neuronal and non-neuronal nuclei in young animals treated with vehicle, WT littermates of APP/PS1-21 animals treated with vehicle and aged and APP/PS1-21 animals treated with oral vehicle or SAHA for 4 weeks',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing'},\n",
       " '200063203': {'inp_pmids': ['35172154'],\n",
       "  'title': 'A restricted period for formation of Outer Subventricular Zone defined by Cdh1 and Trnp1 levels',\n",
       "  'summary': 'The Outer Subventricular Zone (OSVZ) is a germinal layer playing key roles in the development of the neocortex, with particular relevance in gyrencephalic species like human and ferret where it contains abundant basal Radial Glia Cells (bRGCs) that promote cortical expansion. Here we identify a brief period in ferret embryonic development when apical RGCs generate a burst of bRGCs that become founders of the OSVZ. After this period, bRGCs in the OSVZ proliferate and self-renew exclusively locally, thereby forming a self-sustained lineage independent from the other germinal layers. The time window for the brief period of OSVZ bRGC production is delineated by the coincident down-regulation of Cdh1 and Trnp1, and their up-regulation reduces bRGC production and prevents OSVZ seeding. This mechanism in cortical development may have key relevance in brain evolution and disease',\n",
       "  'organism': 'Mustela putorius furo',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200060687': {'inp_pmids': ['35172154'],\n",
       "  'title': 'Sharp changes in gene expression levels along germinal layers distinguish the development of gyrencephaly',\n",
       "  'summary': 'Gyrencephalic species develop folds in the cerebral cortex in a stereotypic manner, but the genetic mechanisms underlying this unique patterning process are unknown. We present a large-scale transcriptomic analysis of individual germinal layers in the developing cortex of the gyrencephalic ferret, comparing between regions prospective of fold and fissure. We find unique transcriptional signatures in each germinal compartment, indicating that thousands of genes are differentially expressed between regions, including ~80% of genes mutated in human cortical malformations. Regional differences result from sharp changes in gene expression levels, which occur at multiple locations across the developing cortex of ferret and human, but not in the lissencephalic mouse. These complex expression patterns emerge sequentially during development and map the eventual location of specific folds and fissures. Our findings demonstrate that step-wise gene expression maps within germinal layers distinguish the development of gyrencephaly from lissencephaly, and may contribute to identify novel genes responsible for human malformations.',\n",
       "  'organism': 'Mustela putorius furo',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200060202': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Fathead Minnow (Pimephales promelas) and Zebrafish (Danio rerio)',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Pimephales promelas',\n",
       "  'exp_type': 'Expression profiling by array; Third-party reanalysis'},\n",
       " '200060167': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Zebrafish (Danio rerio)',\n",
       "  'summary': 'Fathead minnow and zebrafish are among the most intensively studied fish species in environmental toxicogenomics. To aid the assessment and interpretation of subtle transcriptomic effects from treatment conditions of interest, there needs to be a better characterization and understanding of the natural variation in gene expression among fish individuals within populations. Little effort, however, has been made in this area. Leveraging the transcriptomics data from a number of our toxicogenomics studies conducted over the years, we conducted a meta-analysis of nearly 600 microarrays generated from the ovary tissue of untreated, reproductively mature fathead minnow and zebrafish samples. As expected, there was considerable batch-to-batch transcriptomic variation; this “batch-effect” appeared to impact the fish transcriptomes randomly. The overall level of variation within-batch was quite low in fish ovary tissue, making it a suitable system for studying chemical stressors with subtle biological effects. The within-batch variation, however, differed considerably among individual genes and molecular pathways. This difference in variability is probably both technical and biological, thus suggesting a need to take into account both the expression levels and variance in evaluating and interpreting the transcriptional impact on genes and pathways by experimental conditions. There was significant conservation of both the genomes and transcriptomes between fathead minnow and zebrafish. The conservation to such a degree would enable not only a comparative biology approach in studying the mechanisms of action underlying environmental stressors, but also effective sharing of a large amount of existing public transcriptomics data for future development of toxicogenomics applications.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Third-party reanalysis; Expression profiling by array'},\n",
       " '200060126': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Fathead Minnow (Pimephales promelas)',\n",
       "  'summary': 'Fathead minnow and zebrafish are among the most intensively studied fish species in environmental toxicogenomics. To aid the assessment and interpretation of subtle transcriptomic effects from treatment conditions of interest, there needs to be a better characterization and understanding of the natural variation in gene expression among fish individuals within populations. Little effort, however, has been made in this area. Leveraging the transcriptomics data from a number of our toxicogenomics studies conducted over the years, we conducted a meta-analysis of nearly 600 microarrays generated from the ovary tissue of untreated, reproductively mature fathead minnow and zebrafish samples. As expected, there was considerable batch-to-batch transcriptomic variation; this “batch-effect” appeared to impact the fish transcriptomes randomly. The overall level of variation within-batch was quite low in fish ovary tissue, making it a suitable system for studying chemical stressors with subtle biological effects. The within-batch variation, however, differed considerably among individual genes and molecular pathways. This difference in variability is probably both technical and biological, thus suggesting a need to take into account both the expression levels and variance in evaluating and interpreting the transcriptional impact on genes and pathways by experimental conditions. There was significant conservation of both the genomes and transcriptomes between fathead minnow and zebrafish. The conservation to such a degree would enable not only a comparative biology approach in studying the mechanisms of action underlying environmental stressors, but also effective sharing of a large amount of existing public transcriptomics data for future development of toxicogenomics applications.',\n",
       "  'organism': 'Pimephales promelas',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200054680': {'inp_pmids': ['25939354'],\n",
       "  'title': 'DNA methylation variation in Arabidopsis has a genetic basis and appears to be involved in local adaptation',\n",
       "  'summary': 'We investigated DNA methylation variation in Swedish Arabidopsis thaliana accessions. We found that methylation of transposable elements is temperature sensitive and associated with genetic polymorphism in both cis and trans, whereas gene body methylation is associated with genetic polymorphism in trans. Additionally, complementary RNA-Seq data for the Arabidopsis accessions were used to correlate methylation changes with gene expression across environments.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing'},\n",
       " '200054362': {'inp_pmids': ['25340342'],\n",
       "  'title': 'Effects of Changes in Food Supply at the Time of Sex Differentiation on the Gonadal Transcriptome of Juvenile Fish. Implications for Natural and Farmed Populations',\n",
       "  'summary': 'Background: Food supply is a major factor influencing growth rates in animals. This has important implications for both natural and farmed fish populations, since food restriction may difficult reproduction. However, a study on the effects of food supply on the development of juvenile gonads has never been transcriptionally described in fish. Methods and Findings: This study investigated the consequences of growth on gonadal transcriptome of European sea bass in: 1) 4-month-old sexually undifferentiated fish, comparing the gonads of fish with the highest vs. the lowest growth, to explore a possible link between transcriptome and future sex, and 2) testis from 11-month-old juveniles where growth had been manipulated through changes in food supply. The four groups used were: i) sustained fast growth, ii) sustained slow growth, iii) accelerated growth, iv) decelerated growth. The transcriptome of undifferentiated gonads was not drastically affected by initial natural differences in growth. Further, changes in the expression of genes associated with protein turnover were seen, favoring catabolism in slow-growing fish and anabolism in fast-growing fish. Moreover, while fast-growing fish took energy from glucose, as deduced from the pathways affected and the analysis of protein-protein interactions examined, in slow-growing fish lipid metabolism and gluconeogenesis was favored. Interestingly, the highest transcriptomic differences were found when forcing initially fast-growing fish to decelerate their growth, while accelerating growth of initially slow-growing fish resulted in full transcriptomic convergence with sustained fast-growing fish. Conclusions: Food availability during sex differentiation shapes the juvenile testis transcriptome, as evidenced by adaptations to different energy balances. Remarkably, this occurs in absence of major histological changes in the testis. Thus, fish are able to recover transcriptionally their testes if they are provided with enough food supply during sex differentiation; however, an initial fast growth does not represent any advantage in terms of transcriptional fitness if later food becomes scarce.',\n",
       "  'organism': 'Dicentrarchus labrax',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200054292': {'inp_pmids': ['25939354'],\n",
       "  'title': 'DNA methylation variation in Arabidopsis has a genetic basis and appears to be involved in local adaptation',\n",
       "  'summary': 'Here we investigate DNA methylation variation in Swedish Arabidopsis thaliana accessions, demonstrating that methylation of transposable elements is temperature sensitive and associated with genetic polymorphism in both cis and trans, whereas gene body methylation is highly correlated with climate of origin and associated with genetic polymorphism in trans that shows evidence of local adaptation. While genome-wide surveys of naturally occurring DNA methylation have been published previously, the degree of genetic control revealed here is unprecedented. Furthermore, the observation that DNA methylation is associated with climate, and is apparently adaptively important, is completely novel.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200047798': {'inp_pmids': ['24223949'],\n",
       "  'title': 'Perinatal-Estrogen-Induced Changes in Gene Expression Related to Brain Sexual Differentiation in Mice',\n",
       "  'summary': 'Sexual dimorphism of the behaviors or physiological functions in mammals is mainly due to the sex difference of the brain. The goal of this study is to identify genes mediating sexaul dimorphism of the brain. The large-scale analysis with microarray in the present study is an attempt to obtain the candidate gene(s) mediating the perinatal estrogen effect causing the brain sexual differentiation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200040539': {'inp_pmids': ['23042784'],\n",
       "  'title': 'Genome-wide DNA hydroxymethylation changes are associated with neurodevelopmental genes in the developing human brain',\n",
       "  'summary': '5-hydroxymethylcytosine (5-hmC) is a newly discovered modified form of cytosine that has been suspected to be an important epigenetic modification in neurodevelopment. While DNA methylation dynamics have already been implicated during neurodevelopment, little is known about hydroxymethylation in this process. Here we report DNA hydroxymethylation dynamics during cerebellum development in the human brain. Overall, we find a positive correlation between 5-hmC levels and cerebellum development. Genome-wide profiling reveals that 5-hmC is highly enriched on specific gene regions, including exons and especially the untranslated regions (UTRs), but it is depleted on introns and intergenic regions. Furthermore, we have identified fetus-specific and adult-specific differentially hydroxymethylated regions (DhMRs), most of which overlap with genes and CpG island shores. Surprisingly, during development DhMRs are highly enriched in genes encoding mRNAs that can be regulated by fragile X mental retardation protein (FMRP), some of which are disrupted in autism, as well as in many known autism genes. Our results suggest that 5-hmC-mediated epigenetic regulation may broadly impact the development of the human brain, and its dysregulation could contribute to the molecular pathogenesis of neurodevelopmental disorders.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200034429': {'inp_pmids': ['22219169'],\n",
       "  'title': 'Expression and DNA methylation of colon cancer cell lines',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array; Methylation profiling by genome tiling array'},\n",
       " '200034427': {'inp_pmids': ['22219169'],\n",
       "  'title': 'DNA methylation analysis of Colon Cancer cell line SW48',\n",
       "  'summary': 'We combined gene expression experiments in response to HDAC inhibitor Depsipeptide with DNA methylation level to assess gene reactivation arising from hypermethylated promoters.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200034077': {'inp_pmids': ['22219169'],\n",
       "  'title': 'Colon Cancer cell line YB5 untreated cells vs YB5 treated 24h with Depsipeptide at 20 nM',\n",
       "  'summary': 'Study the effect of HDACi Depsipeptide on gene expression. We combined these data with whole genome DNA methylation microarray to test the reactivation of genes that have an hypermethylated promoter region.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200033528': {'inp_pmids': ['22384383'],\n",
       "  'title': 'Genome-wide survey of large rare copy number variations in Alzheimer’s disease among Caribbean Hispanics',\n",
       "  'summary': 'Recently genome-wide association studies have identified significant association between Alzheimer’s disease and variations in CLU, PICALM, BIN1, CR1, MS4A4/MS4A6E, CD2AP, CD33, EPHA1 and ABCA7. However, the pathogenic variants in these loci have not yet been found. We conducted a genome-wide scan for large copy number variations (CNVs) in a dataset of Caribbean Hispanic origin (554 controls and 559 cases with late-onset Alzheimer’s disease) that was previously investigated in a SNP-based genome-wide association study using Illumina HumanHap 650Y platform. We ran four CNV calling algorithms and analyzed rare large CNVs (>100 Kb) to obtain high-confidence calls that were detected by at least two algorithms. In total, 734 such CNVs were observed in our dataset. Global burden analyses did not reveal significant differences between cases and controls in CNV rate, distribution of deletions or duplications, total or average CNV size; and number of genes affected by CNVs. However, we observed a nominal association between Alzheimer’s disease and a ~470 Kb duplication on chromosome15q11.2 (P=0.037). This duplication, encompassing up to five genes (TUBGCP5, CYFIP1, NIPA2, NIPA1 and WHAMML1) was present in 10 cases (2.6%) and 3 controls (0.8%). The dosage increase of CYFIP1 and NIPA1 genes was further confirmed by quantitative PCR. The current study did not detect CNVs (including common CNVs) that affect novel Alzheimer’s disease loci reported by large genome-wide association studies. However, since the array technology used in our study has limitations in detecting small CNVs, future studies must carefully assess novel AD associated genes for the presence of disease related CNVs.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome variation profiling by SNP array'},\n",
       " '200028094': {'inp_pmids': ['21613409'],\n",
       "  'title': 'A DNA Methylation Fingerprint of 1,628 Human Samples',\n",
       "  'summary': 'DNA methylation is the best characterized of the different layers that make up the epigenetic setting. Most of the studies characterizing DNA methylation patterns have been restricted to particular genomic loci in a limited number of human samples and pathological conditions. The recently arrived single-base-resolution technologies for DNA methylation are extremely helpful tools, but are not yet applicable and affordable for studying large groups of subjects. Herein, we present a compromise between an extremely comprehensive study of a human sample population with an intermediate level of resolution of CpGs at the genomic level. We obtained a DNA methylation fingerprint of 1,628 human samples where we interrogated 1,505 CpG sites. The DNA methylation patterns revealed show this epigenetic mark to be critical in tissue-type definition and stemness, particularly around transcription start sites that are not within a CpG island. For disease, the generated DNA methylation fingerprints show that, during tumorigenesis, human cancer cells underwent a progressive gain of promoter CpG island hypermethylation and a loss of CpG methylation in non-CpG island promoters. Although transformed cells are those where DNA methylation disruption is more obvious, we observed that other common human diseases, such as neurological and autoimmune disorders, had their own distinct DNA methylation profiles. Most importantly, we provide proof of principle that the obtained DNA methylation fingerprints might be useful for translational purposes by showing that are able to identify the tumor type origin of Cancers of Unknown Primary (CUPs). Thus, the DNA methylation patterns identified across the largest spectrum of samples, tissues and diseases reported to date constitute a baseline for developing higher-resolution DNA methylation maps, and provide important clues concerning the contribution of CpG methylation to tissue identity and its changes in the most prevalent human diseases.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by array'},\n",
       " '200023694': {'inp_pmids': ['21057496'],\n",
       "  'title': 'Analysis and design of RNA sequencing experiments for identifying mRNA isoform regulation',\n",
       "  'summary': 'Through alternative splicing, most human genes express multiple isoforms that may have distinct or even antagonistic functions. To infer isoform regulation based on data from high-throughput sequencing of cDNA fragments (RNA-Seq), we have developed MISO, a computational model that estimates the expression level of alternatively spliced exons and mRNA isoforms and provides intuitive measures of conﬁdence in these estimates. Incorporation of the length distribution of inserted cDNA fragments in paired-end RNA-Seq analysis in MISO enables dramatic improvements in estimation of alternative splicing levels relative to previous methods. We show that one lane of paired-end RNA-Seq data can provide far more information about splicing than two lanes of single-end data, depending critically on properties of the distribution of cDNA fragment lengths in the sequenced library. MISO also leads to an intuitive method to detect diﬀerentially regulated exons or isoforms. Application of this method implicates the RNA splicing factor hnRNP H in regulation of alternative cleavage and polyadenylation, a role that is supported by UV crosslinking/immunoprecipitation/high-throughput sequencing (CLIP-Seq) analysis. Together, our results provide a probabilistic framework for RNA-Seq analysis, derive functional insights into pre-mRNA processing, and yield guidelines for the optimal design of RNA-Seq experiments for studies of gene and isoform expression.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Other'},\n",
       " '200022872': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating H3K4me2 methylation',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by genome tiling array; Expression profiling by array'},\n",
       " '200022870': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating intragenic H3K4me2 methylation (ChIP-chip)',\n",
       "  'summary': 'Dynamic histone H3K4 methylation is an important epigenetic component of transcriptional regulation. However, most of our current understanding of this histone mark is confined to regulation of transcriptional initiation. We now show that human LSD2/KDM1b/AOF1, the human homolog of LSD1, is a novel H3K4me1/2 demethylase that specifically regulates histone H3K4 methylation within intragenic regions of its target genes. Genome-wide mapping reveals that LSD2 associates predominantly with the gene bodies of actively transcribed genes, but is markedly absent from promoters. Depletion of endogenous LSD2 results in an increase of H3K4me2 as well as a decrease of H3K9me2 at LSD2 binding sites, and a consequent dysregulation of target gene transcription. Furthermore, characterization of LSD2 complex revealed that LSD2 forms active complexes with euchromatic histone methyltransferases EHMT1/2 and NSD3 as well as cellular factors involved in active transcription elongation. These data provide a possible molecular mechanism linking LSD2 to transcriptional regulation post initiation.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by genome tiling array'},\n",
       " '200022859': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating H3K4me2 methylation (Affymetrix)',\n",
       "  'summary': 'Dynamic histone H3K4 methylation is an important epigenetic component of transcriptional regulation. However, most of our current understanding of this histone mark is confined to regulation of transcriptional initiation. We now show that human LSD2/KDM1b/AOF1, the human homolog of LSD1, is a novel H3K4me1/2 demethylase that specifically regulates histone H3K4 methylation within intragenic regions of its target genes. Genome-wide mapping reveals that LSD2 associates predominantly with the gene bodies of actively transcribed genes, but is markedly absent from promoters. Depletion of endogenous LSD2 results in an increase of H3K4me2 as well as a decrease of H3K9me2 at LSD2 binding sites, and a consequent dysregulation of target gene transcription. Furthermore, characterization of LSD2 complex revealed that LSD2 forms active complexes with euchromatic histone methyltransferases EHMT1/2 and NSD3 as well as cellular factors involved in active transcription elongation. These data provide a possible molecular mechanism linking LSD2 to transcriptional regulation post initiation. We used microarray analysis to identify the subset of genes that are differentially expression after depletion of endogenous LSD2/KDM1b/AOF1.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '200021662': {'inp_pmids': ['20385583'],\n",
       "  'title': 'A tripartite paternally methylated region within the Gpr1-Zdbf2 imprinted domain on mouse chromosome 1 identified by meDIP-on-chip',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200018178': {'inp_pmids': ['20385583'],\n",
       "  'title': 'Methylation status in human sperm',\n",
       "  'summary': 'Genomic imprinting describes the expression of a subset of mammalian genes from one parental chromosome. The parent-of-origin specific expression of imprinted genes relies on DNA methylation of CpG-dinucleotides at differentially methylated regions (DMRs) during gametogenesis. We identified the paternally methylated DMR at human chromosome 2 near the imprinted ZDBF2 gene using a methylated-DNA immunoprecipitation-on-chip (meDIP-on-chip) method applied to DNA from sperm.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200018126': {'inp_pmids': ['20385583'],\n",
       "  'title': 'Methylation status in AG- and PG-derived cells and sperm',\n",
       "  'summary': 'Genomic imprinting describes the expression of a subset of mammalian genes from one parental chromosome. The parent-of-origin specific expression of imprinted genes relies on DNA methylation of CpG-dinucleotides at differentially methylated regions (DMRs) during gametogenesis. We identified the paternally methylated DMRs at mouse chromosome 1 near the imprinted Zdbf2 gene using a methylated-DNA immunoprecipitation-on-chip (meDIP-on-chip) method applied to DNA from parthenogenetic (PG)- and androgenetic (AG)-derived cells and sperm.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array'},\n",
       " '200017554': {'inp_pmids': ['19723310', '20602769'],\n",
       "  'title': 'Nucleosome deposition and DNA methylation at coding region boundaries',\n",
       "  'summary': 'We profiled CpG methylation for human T cells and compared this pattern with public T cell nucleosome (H2A.Z) and polymerase II profiles (SRA000234). Focusing on DNA regions surrounding the start codon and stop codon, instead of the transcription start and end sites, we discovered a very intriguing pattern, namely methylation and nucleosomal peaks at those regions, more prominent than peaks near transcript boundaries. Elongating polymerases tend to pause near both coding ends immediately upstream of the epigenetic peaks, causing a significant reduction in elongation efficiency, suggesting their role in control of RNA splicing. Conserved features in underlying protein coding sequences seem to dictate their evolutionary conservation across multiple species. The nucleosomal and methylation marks are commonly associated with high sequence-encoded DNA-bending propensity but differentially with CpG density. Keywords: Epigenetics',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing'},\n",
       " '200015745': {'inp_pmids': ['20485568'],\n",
       "  'title': 'Abundant Quantitative Trait Loci for CpG Methylation and Expression Across Human Brain Tissues',\n",
       "  'summary': 'A fundamental challenge in the post-genome era is to understand and annotate the consequences of genetic variation, particularly within the context of human tissues. We describe a set of integrated experiments designed to investigate the effects of common genetic variability on DNA methylation, mRNA expression and microRNA (miRNA) expression in four distinct human brain regions. We show that brain tissues may be readily distinguished based on methylation status or expression profile. We find an abundance of genetic cis regulation mRNA expression and show for the first time abundant quantitative trait loci for DNA CpG methylation. We observe that the largest magnitude effects occur across distinct brain regions. We believe these data, which we have made publicly available, will be useful in understanding the biological effects of genetic variation. Authorized Access data: Mapping of GEO sample accessions to dbGaP subject/sample IDs is available through dbGaP Authorized Access, see http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000249.v1.p1',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array; Non-coding RNA profiling by array; Methylation profiling by array'},\n",
       " '200009990': {'inp_pmids': ['19211887'],\n",
       "  'title': 'Transcriptional profiles of rodent hippocampal CA1 tissue during aging and cognitive decline',\n",
       "  'summary': 'Recent microarray studies in the hippocampus of rodents or Alzheimer’s disease (AD) subjects have identified a substantial number of cellular pathways/processes correlated with aging and cognitive decline. However, the temporal relationships among these expression changes or with cognitive impairment have not been studied in depth. Here, using Affymetrix microarrays, immunohistochemistry and Morris water maze cognitive testing across 5 age groups of male F344 rats (n=9-15/group, one microarray per animal), we systematically analyzed the temporal sequence and cellular localization of aging changes in expression. These were correlated with performance scores on the hippocampus-dependent Morris Water Maze task. Significant microarray results were sorted in to Early, Intermediate, Midlife, and Late patterns of expression, and functionally categorized (Early- downregulated neural development, lipid synthesis and energy-utilization; upregulated ribosomal synthesis, growth, stress/inflammatory, lysosome and protein/lipid degradation. Intermediate- increased defense/inflammatory activation and decreased transporter activity; Midlife- downregulated energy-dependent signaling and neurite growth, upregulated astroglial activation, Ca2+-binding, cholesterol/lipid trafficking, myelinogenic processes and additional lysosome/inflammation; Late- further recruitment of genes in already-altered pathways). Immunohistochemistry revealed a primarily astrocytic localization of the processes upregulated in midlife, as well as increased density of myelin proteins. Evidence of cognitive impairment first appeared in the 12-month-old group (midlife) and was increased further in the 23-month-old group, exhibiting the highest correlations with some upregulated genes related to cholesterol transport (e.g., Apoe, Abca2), protein management and ion binding. Some upregulated genes for inflammation (Il6st) and myelinogenesis (Pmp22) also correlated with impairment. Together, the data are consistent with a novel sequential cascade model of brain aging in which metabolic alterations early in maturity are followed by inflammation and midlife activation of an astrocyte-centered cholesterol trafficking pathway that stimulates oligodendrocyte remyelination programs. Importantly, this cholesterol trafficking pathway also may compete for astroglial bioenergetic support of neurons, in turn, leading to downregulation of energy-dependent pathways needed to sustain cognitive functions. Keywords: aging time course',\n",
       "  'organism': 'Rattus norvegicus',\n",
       "  'exp_type': 'Expression profiling by array'},\n",
       " '3915': {'inp_pmids': ['19211887'],\n",
       "  'title': 'Aging and cognitive decline: hippocampus',\n",
       "  'summary': 'Analysis of hippocampi from F344 males, at 5 age points across the adult lifespan, tested on the Morris water maze (MWM) task. MWM is a dorsal hippocampal-dependent, spatial memory task. MWM performance showed an age effect. Results provide insight into the molecular basis of cognitive aging.',\n",
       "  'organism': 'Rattus norvegicus',\n",
       "  'exp_type': 'Expression profiling by array'}}"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gds_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "b0001585-e3bd-494f-914d-71a9bc5fd5af",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Creating a corpus for tf-idf encoding and a gds_ids list with corresponding indices\n",
    "corpus = []\n",
    "gds_ids = []\n",
    "\n",
    "for gds_id, data in gds_dict.items():\n",
    "    doc = f\"{data['title']} {data['summary']} {data['organism']} {data['exp_type']}\"\n",
    "    corpus.append(doc)\n",
    "    gds_ids.append(gds_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "45ac4b3c-a22c-468d-8484-f5a2fdae0805",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Setting u the vectorizer and fitting\n",
    "vectorizer = TfidfVectorizer(\n",
    "    stop_words=\"english\",\n",
    "    max_df=0.95,\n",
    "    min_df=2\n",
    ")\n",
    "X = vectorizer.fit_transform(corpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "8eee3202",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Setting up clustering parameters\n",
    "k = 8\n",
    "model = KMeans(n_clusters=k, random_state=42)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "0fa1c47a",
   "metadata": {},
   "outputs": [],
   "source": [
    "clusters = model.fit_predict(X)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "7976557c",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Adding a cluster_id to the main dictionary\n",
    "for gds_id, cluster_id in zip(gds_ids, clusters):\n",
    "    gds_dict[gds_id][\"cluster_id\"] = int(cluster_id)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "12985231",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Inspecting clusters\n",
    "cluster_groups = defaultdict(list)\n",
    "\n",
    "for gds_id, cluster_id in zip(gds_ids, clusters):\n",
    "    cluster_groups[cluster_id].append(gds_id)\n",
    "\n",
    "for cluster_id, gds_list in cluster_groups.items():\n",
    "    print(f\"\\nCluster {cluster_id} ({len(gds_list)} datasets):\")\n",
    "    for gds in gds_list[:3]:  # pokaż tylko kilka\n",
    "        print(f\" - {gds_dict[gds]['title']}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "2b0f582a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Silhouette score: 0.027\n"
     ]
    }
   ],
   "source": [
    "score = silhouette_score(X, clusters)\n",
    "print(f\"Silhouette score: {score:.3f}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "dd2cbb25",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2 clusters → Silhouette score: 0.020\n",
      "3 clusters → Silhouette score: 0.020\n",
      "4 clusters → Silhouette score: 0.027\n",
      "5 clusters → Silhouette score: 0.027\n",
      "6 clusters → Silhouette score: 0.032\n",
      "7 clusters → Silhouette score: 0.036\n",
      "8 clusters → Silhouette score: 0.044\n",
      "9 clusters → Silhouette score: 0.049\n"
     ]
    }
   ],
   "source": [
    "# experimenting with different numbers of clusters\n",
    "for n_clusters in range(2, 10):  # Możesz zmienić zakres\n",
    "    kmeans = KMeans(n_clusters=n_clusters, random_state=42)\n",
    "    labels = kmeans.fit_predict(X)\n",
    "    score = silhouette_score(X, labels)\n",
    "    print(f\"{n_clusters} clusters → Silhouette score: {score:.3f}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "9ad13c5a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'200272793': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development (Mouse embryo RNA-seq)',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200268524': {'inp_pmids': ['39838364'],\n",
       "  'title': 'A robust circulating microRNA signature for diagnosis and early detection in pancreatobiliary cancer',\n",
       "  'summary': 'Purpose: Oncologists for pancreatobiliary cancer (PBca) demand a new circulating biomarker superior to carbohydrate antigen 19-9 (CA19-9). This study aimed to identify the serum microRNA signature composed of reproducible and disease-related microRNAs with clinical bioinformatics. Methods: This multicenter study enrolled patients with treatment-naïve PBca and healthy participants. Optimized serum processing condition was evaluated with t-distributed stochastic neighbor embedding (t-SNE) visualization. The serum microRNA candidates were selected in disease association with Weighted Gene Coexpression Network Analysis (WGCNA). A microRNA signature, combination of multiple serum microRNAs, was examined in exploratory, validation and independent validation set. Biology of the diagnostic miRNAs was evaluated using human pancreatic cancer cells. Results: The 284 (healthy 150, PBca 134) of 827 serum samples were processed within 2hr of time-to serum collection, distributed at the same area of t-SNE map, assigned to exploratory set. The 193 optimized samples were assigned to validation (healthy 50, PBca 47) or independent validation set (healthy 50, PBca 46). Index-1 was a combination of the 5 serum microRNAs having disease-association on WGCNA, showed a sensitivity/specificity >80% and an AUC outperforming CA19-9 in exploratory, validation and independent validation set. The AUC of Index-1 for detecting T1 tumor was superior to CA19-9 (0.856 vs. 0.649, p = 0.038). Human pancreatic cancer cells had intra- and extracellular expression of miR-665, a component of Index-1. Transfection of miR-665 to a human pancreatic cancer cell inhibited cell growth. Conclusions: A serum microRNA signature, Index-1, was useful for detecting PBca, which would improve the early diagnosis of PBca.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200263013': {'inp_pmids': ['38641753'],\n",
       "  'title': 'Repetitive element transcript accumulation is related to inflammaging in humans',\n",
       "  'summary': 'Inflammation is an established feature of many age-related diseases, and growing evidence implicates transcripts from repetitive elements in inflammatory signaling with aging. We generated transcriptomic (RNA-seq) and whole-genome bisulfite sequencing (WGBS, for methylation) data on frozen/biobanked peripheral blood mononuclear cells from middle-aged and older adults, as well as younger and older adults before and after an exercise intervention, to evaluate correlations among repetitive elements and inflammation, along with methylation of repetitive elements.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200261729': {'inp_pmids': ['39762647'],\n",
       "  'title': 'Investigating the transcriptional heterogeneity of the beta cells experiencing Ca2+ excitotoxicity',\n",
       "  'summary': 'Beta cells are remarkably heterogeneous. However, beta cells response to chronic stress at a single cell level remains largely unknown. Here, we used a chemo-genetic model of beta cell excitotoxcity coupled with single cell RNA sequencing to understand beta cells response to chronic stress. Remarkably, we found that a subset of stressed beta cells progressed towards death while the other group evaded the destruction by dedifferentiation. Vulnerable beta cells that progressed towards death showed upregulation of genes implicated in apoptosis, inflammation and immune system activation. The resilient beta cells that evaded the stress showed increased expression aldh1a3, a canonical marker of mammalian beta cell dedifferentiation.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200254970': {'inp_pmids': ['38909241'],\n",
       "  'title': 'Age, Sex and Alzheimer’s disease: a longitudinal study of 3xTg-AD mice reveals sex-specific disease trajectories and inflammatory responses mirrored in postmortem brains from Alzheimer’s patients',\n",
       "  'summary': 'Aging and sex are major risk factors for developing late-onset Alzheimer’s disease. Compared to men, women are not only nearly twice as likely to develop Alzheimer’s, but they also experience worse neuropathological burden and cognitive decline despite living longer with the disease. It remains unclear how and when sex differences in biological aging emerge and contribute to Alzheimer’s disease pathogenesis. We hypothesized that these differences lead to distinct molecular Alzheimer’s disease signatures in males and females, which could be harnessed for therapeutic and biomarker development. We aged male and female, 3xTg-AD and B6129 control mice across their respective lifespans while longitudinally collecting brain samples. We conducted RNA sequencing analysis on bulk brain tissue and examined differentially expressed genes between 3xTg-AD and B6129 samples and across ages in each sex. 3xTg-AD males experienced an accelerated upregulation of immune-related gene expression in the brain relative to females, especially in genes involved in complement system activation, suggesting distinct inflammatory disease trajectories between the sexes. Our data demonstrate that chronic inflammation and complement activation are associated with increased mortality, revealing that age-related changes in immune response act as a primary driver of sex differences in Alzheimer’s disease trajectories.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200250598': {'inp_pmids': ['39637179'],\n",
       "  'title': 'Epigenetic aging of seminal fluid is associated with inflammation',\n",
       "  'summary': 'Male infertility has been a primary cause of global infertility, affecting 8-12% of couples worldwide. Previous studies have shown that semen quality decreases with advanced aging with an increased presence of inflammatory cells. In this study, we examined the DNA methylation of seminal fluid to understand how the methylome changes with age and infertility. We also compare the age associated changes in semen to those observed in buccal swabs in order to characterize differences in epigenetic aging across diverse tissues. We found that infertility associated changes in the DNA methylation of semen are driven by changes in cell composition, while those associated with aging are linked to inflammatory genes. Many age associated sites are demethylated with advanced aging and are associated with the activation of inflammatory pathways. By contrast, we do not observe age associated changes in inflammatory genes in buccal swab methylomes, which instead are characterized by changes to bivalent promoters. Our study suggests that DNA methylation could serve as a biomarker for male infertility assessment in addition to the standard semen analysis.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200249713': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Non-coding RNA profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200249708': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development [Small-Seq]',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200249493': {'inp_pmids': ['39367016'],\n",
       "  'title': 'An atlas of small non-coding RNAs in Human Preimplantation Development [RNA-seq]',\n",
       "  'summary': 'Our current understanding of the molecular circuitries that govern early embryogenesis remains limited, particularly in the human. Small non-coding RNAs (sncRNAs) regulate gene expression transcriptionally and post-transcriptionally, however, the expression of specific biotypes and their dynamics during preimplantation development remains to be determined. Using Small-seq, we identified the abundance of and dynamic expression of piRNA, rRNA, snoRNA, tRNA, and miRNA in human embryos from day 3 to 7. Among the biotypes, both miRNA and snoRNA displayed distinct associations with developmental time and lineage formation, many of which were found to target key gene expression programs in pluripotency and lineage establishment. We observed an enrichment of the chromosome 19 miRNA cluster (C19MC) in the trophectoderm (TE), and the chromosome 14 miRNA cluster (C14MC) and MEG8-related snoRNAs in the inner cell mass (ICM). Additionally, isomiR analyses suggested miRNAs are modified differentially during blastulation. Finally, six novel miRNAs and their embryo gene targets we identified. Our analyses provide the first comprehensive measure of sncRNA biotypes and their corresponding dynamics throughout human preimplantation development, providing an extensive resource. Elucidating the functional roles of sncRNAs during preimplantation development will provide insight into the establishment of the blastocyst and exit from pluripotency. Further, better understanding the miRNA regulatory programs in human embryogenesis will inform strategies to improve embryo development and outcomes of assisted reproductive technologies. We anticipate broad usage of our data as a resource for studies aimed at understanding embryogenesis, optimising stem cell-based models, assisted reproductive technology, and stem cell biology.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200243256': {'inp_pmids': ['38977847'],\n",
       "  'title': 'Mapping the chromatin accessibility landscape of zebrafish embryogenesis at single-cell resolution by SPATAC-seq',\n",
       "  'summary': 'Zebrafish is a popular model for studies of embryonic development. Recent surveys of zebrafish embryogenesis with single-cell transcriptomics have uncovered hundreds of cell types, but the dynamic accessible regions of the genome that determine regulatory programs responsible for the unique identity and function of each cell type during embryogenesis lack detailed study. Here, we present SPATAC-seq: a split-pool ligation-based high-throughput single-cell assay for transposase-accessible chromatin using sequencing. Using SPATAC-seq, we profiled chromatin accessibility in more than 808,046 individual nuclei across 20 developmental stages spanning the sphere stage to the early larval protruding mouth stages in a single experiment. Using this chromatin accessibility map, we identified 604 cell types and inferred their developmental relationships. We also identified \\ufeff959,040 candidate cis-regulatory elements (cCREs) and delineate development-specific cCREs, as well as transcription factors defining diverse cell identities. Importantly, transient reporter assays confirmed that a majority of tested cCREs exhibited robust \\ufeffenhanced EGFP expression in restricted cell types or tissues. Finally, we explored gene regulatory programs that drive pigment and notochord cell differentiation. Our work provides a comprehensive and valuable open resource for exploring transcriptional regulators that determine cell fate specification in zebrafish embryogenesis.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200242202': {'inp_pmids': ['38379415'],\n",
       "  'title': 'Age-related changes in human skeletal muscle transcriptome and proteome are more affected by chronic inflammation and physical inactivity than primary aging',\n",
       "  'summary': 'Evaluation of the influence of primary and secondary aging on the manifestation of molecular and cellular hallmarks of aging is a challenging and currently unresolved issue. Our study represents the first demonstration of the distinct role of primary aging and both chronic inflammation and physical inactivity – the most important drivers of secondary aging, in the regulation of transcriptomic and proteomic profiles in human skeletal muscle. To achieve this purpose, young healthy people (n=15), young (n=8) and older (n=37) patients with knee/hip osteoarthritis, a model to study the effect of long-term inactivity and chronic inflammation on the vastus lateralis muscle, were included in the study. It was revealed that widespread and substantial age-related changes in gene expression (~4,000 genes regulating mitochondrial function, proteostasis, cell membrane, secretory and immune response) were related to the long-term physical inactivity and chronic inflammation rather than primary aging. Primary aging contributed mainly to the regulation of genes (~200) encoding nuclear proteins (regulators of DNA repair, RNA processing, and transcription), mitochondrial proteins (genes encoding respiratory enzymes, mitochondrial complex assembly factors, regulators of cristae formation and mitochondrial reactive oxygen species production), as well as regulators of proteostasis. It was found that proteins associated with aging were regulated mainly at the post-transcriptional level. The set of putative primary aging genes and their potential transcriptional regulators can be used as a resource for further targeted studies investigating the role of individual genes and related transcription factors in the emergence of a senescent cell phenotype.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200231588': {'inp_pmids': ['38030723'],\n",
       "  'title': 'BL6Mousei: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Allele-specific gene expression have been observed in genomic imprinting, X chromosome inactivation, allelic exclusion of immunoglobulin loci and non-canonical random monoallelic expression. Here we show that MSL2 can regulate allele specific gene expression and we elucidate its mechanism of action. MSL2 is known to regulate gene dosage up to two fold in drosophila and mammals. In addition to its canonical role as a milder modulator, we found that the loss of MSL2 switches off gene expression completely in an allele specific way and results in biallelic to monoallelic gene expression, which will be ignored as classical down regulated genes without applying the mouse hybrid F1 system. It indicates that MSL2 can activate the expression of the allele which is otherwise inactive intrinsically. We further found this is achieved by the interplay between MSL and KANSL complex. MSL2 acts as a methylation-insensitive factor, which can prevent DNA methylation of the inactive allele, thus facilitating occupancy of methylation-sensitive factor KANSL3 and activating the gene expression. The novel mechanism of MSL2 provides a safeguarding system to warrant biallelic gene expression of intrinsically monoallelic expressed genes including X-inactivated and imprinting genes. These genes are further discovered as sex bias genes such as X-inactivation escaped genes (escapees) are expressed higher in females. Furthermore, the biallelic expression of MSL2 regulated escapees may protect females from single loss of function mutation and contribute to male severer or predominance neurodevelopmental diseases.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200228348': {'inp_pmids': ['37989753'],\n",
       "  'title': 'Transcriptome analysis of human iPSC-neurons treated with cardiac glycosides',\n",
       "  'summary': 'To investigate the molecular mechanisms of cardiac glycosides’ neuroprotection, we profiled transcriptomes of human iPSC-neurons after 72h of treatment with increasing doses of digoxin, oleandrin, proscillaridin A or vehicle alone (0.1% DMSO) using RNA sequencing.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200227741': {'inp_pmids': ['38030723'],\n",
       "  'title': 'CaBlCaXi: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Allele-specific gene expression have been observed in genomic imprinting, X chromosome inactivation, allelic exclusion of immunoglobulin loci and non-canonical random monoallelic expression. Here we show that MSL2 can regulate allele specific gene expression and we elucidate its mechanism of action. MSL2 is known to regulate gene dosage up to two fold in drosophila and mammals. In addition to its canonical role as a milder modulator, we found that the loss of MSL2 switches off gene expression completely in an allele specific way and results in biallelic to monoallelic gene expression, which will be ignored as classical down regulated genes without applying the mouse hybrid F1 system. It indicates that MSL2 can activate the expression of the allele which is otherwise inactive intrinsically. We further found this is achieved by the interplay between MSL and KANSL complex. MSL2 acts as a methylation-insensitive factor, which can prevent DNA methylation of the inactive allele, thus facilitating occupancy of methylation-sensitive factor KANSL3 and activating the gene expression. The novel mechanism of MSL2 provides a safeguarding system to warrant biallelic gene expression of intrinsically monoallelic expressed genes including X-inactivated and imprinting genes. These genes are further discovered as sex bias genes such as X-inactivation escaped genes (escapees) are expressed higher in females. Furthermore, the biallelic expression of MSL2 regulated escapees may protect females from single loss of function mutation and contribute to male severer or predominance neurodevelopmental diseases.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200216999': {'inp_pmids': ['38539015'],\n",
       "  'title': 'Xenografted human microglia display diverse transcriptomic states in response to Alzheimer’s disease-related Aβ pathology',\n",
       "  'summary': 'Microglia are central players in Alzheimer’s Disease (AD) pathology, but analyzing microglia states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the AppNL-G-F model of amyloid pathology and wild type controls. Xenografted human microglia adopt a disease-associated (DAM) profile similar to that seen in mouse microglia, but display a more pronounced HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine CRM response to oligomeric Aβo. Genetic deletion of TREM2 or APOE, as well as APOE polymorphisms and TREM2R47H expression in the transplanted microglia modulate these responses differentially. The expression of other AD risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in response to AD-related pathology, which should be taken into account in translational studies.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200216991': {'inp_pmids': ['37989753'],\n",
       "  'title': 'Small Molecule Inducers of Neuroprotective miR-132 Identified by HTS-HTS in Human iPSC-derived Neurons',\n",
       "  'summary': 'MicroRNAs (miRNAs) are short RNAs that regulate fundamental biological processes. miR-132, a key miRNA with established functions in Tau homeostasis and neuroprotection, is consistently downregulated in Alzheimer’s disease (AD) and other tauopathies. miR-132 overexpression rescues neurodegenerative phenotypes in several AD models. To complement research on miRNA-mimicking oligonucleotides targeting the central nervous system, we developed a high-throughput-screen coupled high-throughput-sequencing (HTS-HTS) in human induced pluripotent stem cell (iPSC)-derived neurons to identify small molecule inducers of miR-132. We discovered that cardiac glycosides, which are canonical sodium-potassium ATPase inhibitors, selectively upregulated miR-132 in the sub-μM range. Coordinately, cardiac glycoside treatment downregulated total and phosphorylated Tau in rodent and human neurons and protected against toxicity by glutamate, N-methyl-D-aspartate, rotenone, and Aβ oligomers. In conclusion, we identified small-molecule drugs that upregulated the neuroprotective miR-132 and ameliorated neurodegenerative phenotypes. Our dataset also represents a comprehensive resource for discovering small molecules that modulate specific miRNAs for therapeutic purposes.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing',\n",
       "  'cluster_id': 6},\n",
       " '200216375': {'inp_pmids': ['36803569'],\n",
       "  'title': 'The essential roles of FXR in diet and aging affected metabolic changes and liver disease development: a multi-omics study',\n",
       "  'summary': 'Aging and unhealthy diets are risks for metabolic diseases including liver cancer. Bile acid receptor farnesoid X receptor (FXR) knockout (KO) mice develop metabolic liver diseases and progress into liver cancer as they age, and Western diet (WD) intake facilitates liver carcinogenesis in those mice. This study aimed to uncover molecular signatures within the gut-liver axis for diet and age-linked liver diseases in FXR-dependent or independent manners. Many more transcripts were changed due to WD intake and aging in WT mice than those in FXR KO mice. In other words, WD intake and aging impact the hepatic transcriptomes and metabolomes in an FXR-dependent manner. In WT mice, WD/aging upregulated inflammation-related genes and downregulated genes involved in oxidative phosphorylation (OXPHOS). Urine metabolomes provided clear distinguishing for differential dietary intake. By contrast, the metabolomes of the liver, serum, or urine could reflect age differences. Further, irrespective of differential diets intake or ages, transcriptomes, metabolomes, and cecal microbiota distinguished WT and FXR KO. Western dietary patterns and aging share molecular commonality with FXR deactivation. Notably, WD, aging, and FXR deactivation commonly altered hepatic cell division-related transcripts (Cenpe, Ect2, Top2a, Kif20a, Tpx2, Nuf2, Kif18b, Aspm, E2f8, and Hmmr), which are associated with overall survival rate in HCC patients. In conclusion, FXR is essential for maintaining metabolic homeostasis in response to WD intake and aging. FXR activation helps to alleviate diet and/or aging-induced metabolic health issues.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200215954': {'inp_pmids': ['37871105'],\n",
       "  'title': 'Precision pharmacological reversal of strain-specific diet-induced metabolic syndrome in mice informed by epigenetic and transcriptional regulation',\n",
       "  'summary': 'We exposed three mouse strains to a high-fat high-carbohydrate diet, leading to varying degrees of metabolic syndrome. Comprehensive methylation and transcriptomic analysis found overlapping but also highly divergent changes in DNA methylation and gene expression, with pathway analysis suggesting a strategy for strain-specific targeted pharmacologic intervention of upstream regulators. Administeration of the drug GW4064 to target one of these genotype-dependent networks, the farnesoid X receptor pathway resulted in genotype-specific protection against dietary effects in BL6 as predicted by our epigenetic analysis.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200215901': {'inp_pmids': ['38977847'],\n",
       "  'title': 'SPATAC-seq: split-pool ligation-based single cell ATAC-seq method',\n",
       "  'summary': \"To establish an ultra-high-throughput single cell chromatin accessibility profiling method that is cost-effective and widely accessible, we built on sci-ATAC-seq (Cusanovich, D. A. et al. Science. 2015; Amini, S. et al. Nature Genetics. 2014) and SPLIT-seq (Rosenberg, A. B. et al. Science. 2018) to design SPATAC-seq, which in situ label chromatin fragment in the same single cell through combinatorial barcoding. Briefly, in SPATAC-seq, (1) fixed nuclei are transposed in 48 different wells by 48 unique Tn5 transposase, which containing barcoded adaptors and 5'-phosphorylation; (2) the nuclei from all wells are collected and redistributed into second and third 48-well plate in turn, where the next two rounds of indexing are achieved through into either end of the custom transposome, which result in the generation of more than 110,000 (48^3) unique barcode combinations. (3) the nuclei are pooled, split into sublibraries and lysed, and the DNA was amplified by polymerase chain reaction (PCR), which introduce illumina sequencing barcodes and complete libraries construction. (4) After sequencing, fastq files were demultiplexed according to the same four-barcode combinations. For profiling more cells in one sublibrary, we can increase the number of barcode combinations by increase the number of indexing of each round to 96, which can produce about 1 million combinations. To assess the fidelity of SPATAC-seq, we performed a species-mixing experiment with cultured human (K562) and mouse (Hepa) cells. Here, we tagged mixed permeabilized nuclei with only 8 barcoded transposome. In round 4, we generated eight sublibraries with different cell-recovery targets, which can be used to evaluating the stability of this method and the correlation between real doublet rates and theoretical value.\",\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200214496': {'inp_pmids': ['38177678'],\n",
       "  'title': 'DNA methylation restricts default deterministic fates towards neural and germline lineages',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200214458': {'inp_pmids': ['38177678'],\n",
       "  'title': 'DNA methylation restricts default deterministic fates towards neural and germline lineages during early development',\n",
       "  'summary': 'We investigate the impact of DNA methylation on chromatin accessibility during epiblast differentiation. To this end, we used in vitro Epiblast-like cell (EpiLC) differentiation on WT and DNA-methylation free TKO murine ESCs. We next performed ATAC seq using TDE1 Tn5 transposase (Nextera).',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200210256': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 0},\n",
       " '200210255': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study with Infinium HumanMethylationEPIC BeadChip',\n",
       "  'summary': 'African-American individuals of the GENOA study',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200210254': {'inp_pmids': ['37169753', '39800688'],\n",
       "  'title': 'Methylation data from stored peripheral blood leukocytes from African American participants in the GENOA study with Infinium HumanMethylation450 BeadChip',\n",
       "  'summary': 'African-American individuals of the GENOA study',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200208265': {'inp_pmids': ['35920937'],\n",
       "  'title': 'Characterization and analysis of the promoter region of Monodehydroascorbate reductase 4 (CpMDAR4) in papaya',\n",
       "  'summary': 'The aim of this research was to study the effects of cis-element differences between the X, Y and Yh alleles on the expression of CpMDAR4, a potential candidate gene for sex differentiation in papaya, using a transcriptional reporter system in a model species Arabidopsis thaliana. Possible effects of a retrotransposon insertion in the Y and Yh alleles on the transcription and expression of CpMDAR4 alleles in papaya flowers were also examined. When comparing promoters and cis-regulatory elements among genes in the non-recombining region of the sex chromosomes, paired genes exhibited differences. Our results showed that differences in the promoter sequences of the CpMDAR4 alleles drove the expression of a reporter gene to different flower tissues in Arabidopsis. β-glucuronidase staining analysis of T2 and T3 lines for constructs containing 5’ deletions of native Y and Yh allele promoters showed the loss of specific expression of the reporter gene in the anthers, confirming the existence and location of cis-regulatory element POLLEN1LELAT52. The expression analysis of CpMDAR4 alleles in papaya flowers also showed that all alleles are actively expressed in different flower tissues, with the existence of a shorter truncated isoform, with unknown function, for the Y and Yh alleles due to an LTR-RT insertion in the Y and Yh chromosomes. The observed expression patterns in Arabidopsis thaliana flowers and the expression patterns of CpMDAR4 alleles in papaya flowers, suggest that MDAR4 might have a role on development of reproductive organs in papaya, and that it constitutes an important candidate for sex differentiation.',\n",
       "  'organism': 'Carica papaya',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200199806': {'inp_pmids': ['36510023'],\n",
       "  'title': 'DNA methyltransferases 3A and 3B target specific sequences during mouse gastrulation',\n",
       "  'summary': 'We performed single-cell transcriptome analysis (using MARS-seq) of mouse embryonic stem cells, embryiod bodies and mouse embryos. Additionally, we performed methylation analysis of bulk and single cells of this source (using whole-genome PBAT and PBAT capture).',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200197916': {'inp_pmids': ['37277533'],\n",
       "  'title': 'Metabolic competition between lipid metabolism and histone methylation regulates sexual differentiation in human malaria parasites',\n",
       "  'summary': 'Like most pathogens, malaria parasites balance persistence in the current host with transmission to the next. For Plasmodium falciparum, the most widespread and virulent human malaria parasite, persistence depends on continuous asexual replication in red blood cells while transmission requires differentiation into non-replicating gametocytes, the male and female cells able to infect the mosquito vector. This developmental decision is controlled via stochastic derepression of a heterochromatin-silenced locus encoding the transcription factor PfAP2-G, which acts as the master switch for sexual differentiation. Recent work showed that the frequency of pfap2-g derepression is responsive to extracellular levels of phospholipid precursors. However, the regulatory mechanisms linking these metabolites to epigenetic regulation of pfap2-g was hitherto unknown. Using chemical, metabolomic, and genetic approaches, we show that this response is mediated by metabolic competition for the methyl donor S-adenosylmethionine between histone methyltransferases and phospho-ethanolamine methyltransferase, a critical enzyme in the parasite’s pathway for de novo phosphatidyl-choline synthesis. When phosphatidyl-choline precursors are scarce, increased consumption of S-adenosylmethionine for de novo synthesis of phosphatidyl-choline results in a decrease in histone methylation marks that mediate silencing of pfap2-g, thereby up-regulating its transcription and increasing the frequency of sexual differentiation. Our findings reveal a key connection between metabolite utilization and gene expression in malaria parasites that forms the mechanistic link between phosphatidyl-choline metabolism and the frequency of sexual commitment.',\n",
       "  'organism': 'Plasmodium falciparum',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200197063': {'inp_pmids': ['36879017'],\n",
       "  'title': 'A gene silencing screen uncovers diverse tools for targeted gene repression in Arabidopsis',\n",
       "  'summary': 'DNA methylation has been utilized for target gene silencing in plants, however it’s not well-understood whether other silencing pathways can be also used to manipulate gene expression. Here we performed a gain-of-function screen for proteins that could silence a target gene when fused to an artificial zinc finger. We uncovered many proteins that suppressed gene expression through DNA methylation, histone H3K27me3 deposition, H3K4me3 demethylation, histone deacetylation, inhibition of RNA Polymerase II transcription elongation or Ser-5 dephosphorylation. These proteins also silenced many other genes with different efficacy, and a machine learning model could accurately predict the efficacy of each silencer based on various chromatin features of the target loci. Furthermore, some proteins were also able to target gene silencing when used in a dCas9-SunTag system. These results provide a more comprehensive understanding of epigenetic regulatory pathways in plants and provide an armament of tools for targeted gene manipulation.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200197017': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [scRNA-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200196613': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200196612': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [ATAC-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200196611': {'inp_pmids': ['36544018'],\n",
       "  'title': 'A skeletal-muscle senescence blueprint defines an aged-like inflamed niche that inhibits regeneration over lifetime [RNA-seq]',\n",
       "  'summary': 'A new sorting protocol was used to identify and isolate different senescent cell types from damaged muscles of young and geriatric mice. Analysis revealed conservation of two universal senescence hallmarks (inflammation and fibrosis) across cell type, regeneration time, and aging, while cell identity traits were preserved. Senescent cells create an “aged-like” inflamed niche, mirroring inflammation-associated with aging (inflammaging), that arrests stem cell proliferation and regeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200190939': {'inp_pmids': ['35511946'],\n",
       "  'title': 'Circadian alignment of early onset caloric restriction promotes longevity in male C57BL/6J mice',\n",
       "  'summary': 'Caloric restriction (CR) prolongs lifespan, yet the mechanisms by which it does so remain poorly understood. Under CR, mice self-impose chronic cycles of 2-hour-feeding and 22-hour-fasting, raising the question whether calories, fasting, or time of day are causal. We show that 30%-CR is sufficient to extend lifespan 10%; however, a daily fasting interval and circadian-alignment of feeding act together to extend lifespan 35% in male C57BL/6J mice. These circadian effects of CR are independent of fasting duration and body weight. Aging induces widespread increases in gene expression associated with inflammation and decreases in expression of genes encoding components of metabolic pathways in liver from ad lib fed mice. CR at night ameliorates these aging-related changes. Thus, circadian interventions promote longevity and provide a perspective to further explore mechanisms of aging.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200189217': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of blood aging [droplet-based scRNAseq - Batch 2]',\n",
       "  'summary': 'To investigate the role of IL1 signaling in hematopoietic aging, we performed droplet-based scRNAseq (10X Genomics) of hematopoietic stem and progenitor cells (LK/LSK) and unfractionated endosteal and central marrow stromal cells (Ter119-/CD45-) from young wild type and young and old wild type and IL1R1-/- C57BL/6 mice.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200186357': {'inp_pmids': ['36539615'],\n",
       "  'title': 'Dynamics of DNA hydroxymethylation and methylation during mouse embryonic and germline development',\n",
       "  'summary': 'In mammals, DNA 5-hydroxymethylcytosine (5hmC) is involved in methylation reprogramming during early embryonic development. Yet, to what extent 5hmC participates in genome-wide methylation reprogramming remains largely unknown. Here, we characterize the 5hmC landscapes in mouse early embryos and germ cells with parental allele specificity. DNA hydroxymethylation was most strongly correlated with DNA demethylation as compared with de novo or maintenance methylation in zygotes, while 5hmC was targeted to particular de novo methylated sites in postimplantation epiblasts. Surprisingly, DNA replication was also required for 5hmC generation, especially in the female pronucleus. More strikingly, aberrant nuclear localization of Dnmt1/Uhrf1 in mouse zygotes due to maternal deficiency of Nlrp14 led to defects in DNA-replication-coupled passive demethylation and impaired 5hmC deposition, revealing the divergency between genome-wide 5-methylcytosine (5mC) maintenance and Tet-mediated oxidation. In summary, our work provides insights and a valuable resource for the study of epigenetic regulation in early embryo development.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200183766': {'inp_pmids': ['38030723'],\n",
       "  'title': 'CaBl: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200183765': {'inp_pmids': ['38030723'],\n",
       "  'title': 'BlCa: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200183764': {'inp_pmids': ['38030723'],\n",
       "  'title': '9sCa: MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'Biallelic expression is important to ensure adequate mRNA levels as well as protect cells from deleterious heterozygous mutations. However, the factors determining biallelic expression at individual gene loci are unknown. Deletion of the MSL complex component MSL2 in hybrid mouse NPCs uncovers an unprecedented function for MSL2 in allelic regulation. While biallelically expressed in wildtype NPCs, a class of MSL2 targets switches from bi to monoallelic expression upon MSL2 loss.\\xa0The silenced allele loses nascent transcription, accompanied by dramatic changes in local chromatin landscape such as monoallelic gain of H3K27me3 and DNA methylation. Meanwhile, active histone modifications, RNA POL II, BRD4 and KANSL3 are retained on the active allele. MSL2 targets include several sex-biased genes on autosomes as well as X-chromosomal genes that escape X inactivation. Together, the allele-specific resolution allowed us to unravel an MSL2 mediated safeguarding mechanism to promote biallelic gene expression of sexually dimorphic and dosage-sensitive genes in mammals.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Other; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200183556': {'inp_pmids': ['38030723'],\n",
       "  'title': 'MSL2 is an allelic dosage sensor in mammals',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 0},\n",
       " '200180979': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200172402': {'inp_pmids': ['34033742'],\n",
       "  'title': 'microRNA-132 regulates adult neural stem cells and their progeny to restore adult hippocampal neurogenesis and memory deficits in Alzheimer’s disease',\n",
       "  'summary': 'Neural stem cells residing in the hippocampal neurogenic niche sustain life-long neurogenesis in the adult brain. Adult hippocampal neurogenesis (AHN) is functionally linked to mnemonic and cognitive plasticity in humans and rodents. In Alzheimer’s disease (AD), the process of generating new neurons at the hippocampal neurogenic niche is impeded, yet the mechanisms involved are unknown. Here we identify miR-132, one of the most consistently downregulated microRNAs in AD, as a potent regulator of AHN, exerting cell-autonomous pro-neurogenic effects in the adult neural stem cells and their progeny. Using distinct AD mouse models, cultured human primary and established neural stem cells, and human patient material, we demonstrate that AHN is directly impacted by AD pathology. miR-132 replacement in adult mouse AD hippocampus restores AHN and relevant memory deficits. Our findings corroborate the significance of AHN in AD and reveal the possible therapeutic significance of targeting miR-132 in neurodegeneration.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200171017': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure [RNA]',\n",
       "  'summary': 'Polychlorinated biphenyls (PCBs) are developmental neurotoxicants implicated as environmental risk factors for neurodevelopmental disorders (NDD). We examined the effects of prenatal exposure to a human-relevant mixture of PCBs on the DNA methylation profiles of fetal mouse brain and placenta. We found thousands of differentially methylated regions (DMRs) distinguishing placentas and brains from PCB-exposed embryos from sex-matched vehicle controls. In both placenta and brain, PCB-associated DMRs were enriched for functions related to neurodevelopment and cellular signaling and enriched within regions of bivalent chromatin. The placenta and brain PCB DMRs overlapped significantly and mapped to a shared subset of genes enriched for Wnt signaling, Slit/Robo signaling, and genes differentially expressed in NDD models. PBC DMRs also significantly overlapped with DMRs from the brains of humans with Rett syndrome and Dup15q syndrome. These results demonstrate that placenta can be used as a surrogate for embryonic brain DNA methylation changes relevant to an NDD.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200171016': {'inp_pmids': ['35235788'],\n",
       "  'title': 'Placenta and fetal brain share a neurodevelopmental disorder DNA methylation profile in a mouse model of prenatal PCB exposure [WGBS]',\n",
       "  'summary': 'Polychlorinated biphenyls (PCBs) are developmental neurotoxicants implicated as environmental risk factors for neurodevelopmental disorders (NDD). We examined the effects of prenatal exposure to a human-relevant mixture of PCBs on the DNA methylation profiles of fetal mouse brain and placenta. We found thousands of differentially methylated regions (DMRs) distinguishing placentas and brains from PCB-exposed embryos from sex-matched vehicle controls. In both placenta and brain, PCB-associated DMRs were enriched for functions related to neurodevelopment and cellular signaling and enriched within regions of bivalent chromatin. The placenta and brain PCB DMRs overlapped significantly and mapped to a shared subset of genes enriched for Wnt signaling, Slit/Robo signaling, and genes differentially expressed in NDD models. PBC DMRs also significantly overlapped with DMRs from the brains of humans with Rett syndrome and Dup15q syndrome. These results demonstrate that placenta can be used as a surrogate for embryonic brain DNA methylation changes relevant to an NDD.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200169162': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal niche inflammation mediated by IL-1 signaling is a targetable driver of hematopoietic aging',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200168661': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of hematopoietic aging [Microarray analysis]',\n",
       "  'summary': 'We performed integrated analyses of hematopoietic stem and progenitor cells as well as bone marrow stromal cells to better understand the underlying mechanisms of blood aging. To determine how the molecular state of multipotent progenitor (MPP) cells change with age we performed transcriptomic analyses of myeloid-biased MPP3 and lymphoid-biased MPP4 populations from young and old mice.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 6},\n",
       " '200168613': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Inflammation of the Aged Bone Marrow Niche Drives Blood Aging',\n",
       "  'summary': 'Hematopoietic aging is defined by a loss of regenerative capacity and skewed differentiation from hematopoietic stem cells (HSC) leading to dysfunctional blood production. Signals from the bone marrow (BM) microenvironment dynamically tailor hematopoiesis, but the effect of aging on the niche and the contribution of the aging niche to blood aging still remains unclear. Here, we show the development of an inflammatory milieu in the aged marrow cavity, which drives both niche and hematopoietic system remodeling. We find decreased numbers and functionality of osteogenic endosteal mesenchymal stromal cells (MSC), expansion of pro-inflammatory perisinusoidal MSCs, and deterioration of the central marrow sinusoidal endothelium, which together create a self-reinforcing inflamed BM milieu. Single cell molecular mapping of old niche cells further confirms disruption of cell identities and enrichment of inflammatory response genes. Inflammation, in turn, drives chronic activation of emergency myelopoiesis pathways in old HSCs and multipotent progenitors, which promotes myeloid differentiation at the expense of lymphoid and erythroid commitment, and hinders hematopoietic regeneration. Remarkably, both defective hematopoietic regeneration, niche deterioration and HSC aging can be improved by blocking inflammatory IL-1 signaling. Our results indicate that targeting the pro-inflammatory niche milieu can be instrumental in restoring blood production during aging.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 4},\n",
       " '200168586': {'inp_pmids': ['36650381'],\n",
       "  'title': 'Stromal inflammation is a targetable driver of hematopoietic aging [droplet-based scRNAseq]',\n",
       "  'summary': 'To investigate global changes in the structure of the bone marrow niche with age, we performed droplet-based scRNAseq (10X Genomics) of unfractionated endosteal and central marrow stromal cells (Ter119-/CD45-). 8,735 cells passed QC, and hematopoietic clusters of cells were excluded based on marker gene expression, leaving 2359 cells distributed across 9 distinct clusters for final presentation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200167559': {'inp_pmids': ['34686734'],\n",
       "  'title': 'Dementia subtype prediction models constructed by penalized regression methods for multiclass classification using serum microRNA expression data',\n",
       "  'summary': 'There are many subtypes of dementia, and identification of diagnostic biomarkers that are minimally-invasive, low-cost, and efficient is desired. Circulating microRNAs (miRNAs) have recently gained attention as easily accessible and non-invasive biomarkers. We conducted a comprehensive miRNA expression analysis of serum samples from 1348 Japanese dementia patients, composed of four subtypes—Alzheimer’s disease (AD), vascular dementia, dementia with Lewy bodies (DLB), and normal pressure hydrocephalus—and 246 control subjects. We used this data to construct dementia subtype prediction models based on penalized regression models with the multiclass classification. We constructed a final prediction model using 46 miRNAs, which classified dementia patients from an independent validation set into four subtypes of dementia. Network analysis of miRNA target genes revealed important hub genes, SRC and CHD3, associated with the AD pathogenesis. Moreover, MCU and CASP3, which are known to be associated with DLB pathogenesis, were identified from our DLB-specific target genes. Our study demonstrates the potential of blood-based biomarkers for use in dementia-subtype prediction models. We believe that further investigation using larger sample sizes will contribute to the accurate classification of subtypes of dementia.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200165870': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Bulk RNA-seq of aplastic anemia (AA) and healthy donors',\n",
       "  'summary': 'Bulk RNA-seq data of Lin-CD34+ hematopoietic stem and progenitor cells derived from bone marrow of healthy donors and untreated aplastic anemia patients',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200164788': {'inp_pmids': ['33589615'],\n",
       "  'title': \"Machine Learning Identifies Candidates for Drug Repurposing in Alzheimer's Disease\",\n",
       "  'summary': \"Clinical trials of novel therapeutics for Alzheimer's Disease (AD) have consumed a large amount of time and resources with largely negative results. Repurposing drugs already approved by the Food and Drug Administration (FDA) for another indication is a more rapid and less expensive option. In this study, we profile 80 FDA-approved and clinically tested drugs in neural cell cultures, with the goal of producing a ranked list of possible repurposing candidates.\",\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200164174': {'inp_pmids': ['33743111'],\n",
       "  'title': 'A novel combination of serum microRNAs for the detection of early gastric cancer',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of patients with early stage gastric cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200162803': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200162802': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention [RNA-seq]',\n",
       "  'summary': 'Aging is the result of an accumulation of molecular damages, driven by multiple factors like metabolic and oxidative stress, (epi)genetic alterations, and microbiome dysbiosis. In addition, aging is characterized by a chronic, low-grade inflammation known as inflammaging that represents a high significant risk factor for most, if not all, age-related diseases. Dietary restriction (DR) is the best-known non-invasive method to ameliorate the aging phenotype in many organisms. While it has been shown that DR reduces the inflammation in some tissues, the molecular mechanisms in old individuals are still largely unknown. Here we identify a multi-tissue gene network which regulates the inflammaging and that is ameliorated by DR. Using transcriptional profiling (RNA-seq), we found that aging upregulates the innate immune system receptor together with the activation of a systemic interferon (IFN) signaling through the Interferon Regulatory Factors (Irf), inflammatory cytokines and finally Stat1 transcription factor. Our results suggest that DR is able to ameliorate inflammaging by downregulating the expression and the activity of almost all the components of this gene network. In addition, chromatin accessibility genome-wide analysis showed that the binding site of Irf and Stat1 in the promoter of their target genes are more open in aging and become closer upon DR treatment indicating a transcriptional control of the inflammaging phenotype involving epigenetic modifications. Our results provide a comprehensive understanding of the molecular network regulating the inflammation in aging and DR and present novel anti-inflammaging therapeutic targets.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200162728': {'inp_pmids': ['35767948'],\n",
       "  'title': 'Stat1 activation, through systemic interferon signaling, and upregulation of innate immune system receptors drives inflammaging, that is ameliorated by dietary restriction intervention [ATAC-seq]',\n",
       "  'summary': 'Aging is the result of an accumulation of molecular damages, driven by multiple factors like metabolic and oxidative stress, (epi)genetic alterations, and microbiome dysbiosis. In addition, aging is characterized by a chronic, low-grade inflammation known as inflammaging that represents a high significant risk factor for most, if not all, age-related diseases. Dietary restriction (DR) is the best-known non-invasive method to ameliorate the aging phenotype in many organisms. While it has been shown that DR reduces the inflammation in some tissues, the molecular mechanisms in old individuals are still largely unknown. Here we identify a multi-tissue gene network which regulates the inflammaging and that is ameliorated by DR. Using transcriptional profiling (RNA-seq), we found that aging upregulates the innate immune system receptor together with the activation of a systemic interferon (IFN) signaling through the Interferon Regulatory Factors (Irf), inflammatory cytokines and finally Stat1 transcription factor. Our results suggest that DR is able to ameliorate inflammaging by downregulating the expression and the activity of almost all the components of this gene network. In addition, chromatin accessibility genome-wide analysis showed that the binding site of Irf and Stat1 in the promoter of their target genes are more open in aging and become closer upon DR treatment indicating a transcriptional control of the inflammaging phenotype involving epigenetic modifications. Our results provide a comprehensive understanding of the molecular network regulating the inflammation in aging and DR and present novel anti-inflammaging therapeutic targets.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200159798': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200159795': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains [RNA-seq]',\n",
       "  'summary': \"Immune system aging contributes significantly to declining health with age. It is unknown to what extent human and mice immune systems go through similar age-related changes and whether there are conserved biomarkers of aging. We characterized age-related changes in the immune system of long (C57BL/6J) and short-lived (NZO/HILtJ) strains and compared them with blood-driven human aging signatures. Peripheral blood lymphocytes (PBL), spleen cells and spleen-driven naive and memory CD8+ T cells were profiled using flow cytometry, RNA-seq and ATAC-seq from young (3 months) and old (18 months) mice. Pro-inflammatory myeloid genes were activated with age across strains and tissues, echoing human 'inflammaging' signatures. ATAC-seq footprinting analyses uncovered increased binding of pro-inflammatory transcription factors with age (e.g., AP1 complex, NFKB signaling pathway). Machine learning models identified inflammation, cytotoxic, and naive T cell derived genes to be strong signatures of immunosenescence. These data are publicly shared as a resource for immune aging (https://immune-aging.jax.org/mice).\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200159794': {'inp_pmids': ['36840360'],\n",
       "  'title': 'Age-related changes in blood and spleen-derived immune cells in long and short lived mice strains [ATAC-seq]',\n",
       "  'summary': \"Immune system aging contributes significantly to declining health with age. It is unknown to what extent human and mice immune systems go through similar age-related changes and whether there are conserved biomarkers of aging. We characterized age-related changes in the immune system of long (C57BL/6J) and short-lived (NZO/HILtJ) strains and compared them with blood-driven human aging signatures. Peripheral blood lymphocytes (PBL), spleen cells and spleen-driven naive and memory CD8+ T cells were profiled using flow cytometry, RNA-seq and ATAC-seq from young (3 months) and old (18 months) mice. Pro-inflammatory myeloid genes were activated with age across strains and tissues, echoing human 'inflammaging' signatures. ATAC-seq footprinting analyses uncovered increased binding of pro-inflammatory transcription factors with age (e.g., AP1 complex, NFKB signaling pathway). Machine learning models identified inflammation, cytotoxic, and naive T cell derived genes to be strong signatures of immunosenescence. These data are publicly shared as a resource for immune aging (https://immune-aging.jax.org/mice).\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200159596': {'inp_pmids': ['35172154'],\n",
       "  'title': 'Non-coding RNAs shape cortical neurons developmental trajectories',\n",
       "  'summary': 'A hallmark of cortical evolution is the high dynamic subventricular zone (SVZ) expansion, where basal progenitors (BPs) amplify and neuronal transcriptional programs unfold. How non-coding molecular factors such as microRNAs influence these developmental trajectories and regulate the acquisition of cortical type identities is largely unknown. Here we demonstrate that miR-137 and miR-122 regulate the positioning and identity features of superficial layer cortical neurons by acting at distinct steps of their developmental trajectories. MiR-137 sustains basal progenitor amplification by reverting their neurogenic commitment and inducing high proliferative state upregulating Cd63 and inhibiting Myt1l. Cd63 is an extra-cellular matrix (ECM) receptor which interacts with \\uf061\\uf06eb3- and \\uf0621-integrin pathways to promote proliferation, while Myt1l is a transcription factor that promotes and sustains neuronal fate. The BPs amplification by miR-137 is converted in the promotion of intracortical projecting neuron (ICPN) identity and L2/3 expansion. As opposed to miR-137, miR-122 acts postmitotically, affecting the bioelectrical properties, the calcium and cytoskeleton dynamics of newborn neurons as well as their transcriptional program, leading to a persistent molecular immaturity across time. Overall, these findings reveal that miR-137 and miR-122 are key regulators of the developmental trajectory of cortical neurons across evolution.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 6},\n",
       " '200157145': {'inp_pmids': ['34059805'],\n",
       "  'title': 'Transcriptional and imprinting complexity in Arabidopsis seeds at single-nucleus resolution',\n",
       "  'summary': 'Seeds are the basis of agriculture, yet their full transcriptional complexity has remained unknown. Here, we employ single-nucleus RNA-sequencing to characterize developing Arabidopsis thaliana seeds, with a focus on endosperm. Endosperm, the site of gene imprinting in plants, mediates the relationship between the maternal parent and embryo. We identify new cell types in the chalazal endosperm region, which interfaces with maternal tissue for nutrient unloading. We further demonstrate that the extent of parental bias of maternally expressed imprinted genes varies with cell cycle phase, and that imprinting of paternally expressed imprinted genes is strongest in chalazal endosperm. These data indicate imprinting in endosperm is heterogeneous and suggest that parental conflict, which is proposed to drive the evolution of imprinting, is fiercest at the boundary between filial and maternal tissues.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200157027': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings exposed to different communities of commensal bacteria [SSC]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings exposed to communities of suppressors (SynCom5-S1) and non-suppressors (SynCom5-NS1) of the plant immune system',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200156426': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of commensal bacteria',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200156425': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of a community of commensal bacteria [RST]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana roots exposed to the MAMP flg22 in the presence of a 35-member bacterial SynCom',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200156424': {'inp_pmids': ['33879573'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings exposed to the MAMP flg22 in the presence of individual commensal bacteria [MAE]',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings exposed to the MAMP flg22 in the presence of 35 different bacteria',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200156212': {'inp_pmids': ['38287646'],\n",
       "  'title': 'Effects of vitamin D on non-specific inflammatory response in aged skin.',\n",
       "  'summary': 'We have previously demonstrated older skin exhibits increased sterile inflammation 6 hours after saline-injection. In this work, we examined whether vitamin D in vivo would attenuate this non-specific inflammatory response towards saline in elder individuals (≥65 years).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200149097': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200148990': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Differential gene expression in miR-146a-/- vs. WT hematopoietic stem and progenitor cells [miR146aKO_vs_WT_RNA-seq_bulk]',\n",
       "  'summary': 'To identify pathways and processes driving the observed hematopoietic stem cell (HSC) aging-like phenotypes in miR-146a-/- vs. WT, we performed RNA-seq gene expression profiling of Lin- Sca-1+ c-Kit+ (LSK) cells isolated from miR-146a-/- or WT mouse bone marrow (BM). Differential expression analysis and EnrichmentMap network analysis identified cytokine signalling and immune pathways as potential drivers of aging-like alterations in miR-146a-/- HSC proliferation and differentiation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 6},\n",
       " '200148989': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy [miR146aKO_LSK_WGBS]',\n",
       "  'summary': 'Aging is associated with significant changes in the hematopoietic system, including increased inflammation, impaired hematopoietic stem cell (HSC) function, and increased incidence of myeloid malignancy. Inflammation of aging (“inflammaging”) has been proposed as a driver of age-related changes in HSC function and myeloid malignancy, but mechanisms linking these phenomena remain poorly defined. Here, we identify loss of miR-146a as driving aging-associated inflammation in AML patients. miR-146a expression declined in old wild-type mice, and loss of miR-146a promoted premature HSC aging and inflammation in young miR-146a-null mice, preceding development of aging-associated myeloid malignancy. Using single-cell assays of HSC quiescence, stemness, differentiation potential, and epigenetic state to probe HSC function and population structure, we found that loss of miR-146a depleted a subpopulation of primitive, quiescent HSCs. DNA methylation and transcriptome profiling implicated NF-κB, IL6, and TNF as potential drivers of HSC dysfunction, activating an inflammatory signaling relay promoting IL6 and TNF secretion from mature miR-146a-/- myeloid and lymphoid cells. Reducing inflammation by targeting Il6 or Tnf was sufficient to restore single-cell measures of miR-146a-/- HSC function and subpopulation structure, and reduced the incidence of hematological malignancy in miR 146a-/- mice. miR-146a-/- HSCs exhibited enhanced sensitivity to IL6 stimulation, indicating that loss of miR-146a affects HSC function via both cell-extrinsic inflammatory signals and increased cell-intrinsic sensitivity to inflammation. Thus, loss of miR 146a regulates cell-extrinsic and -intrinsic mechanisms linking HSC inflammaging to the development of myeloid malignancy.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 6},\n",
       " '200148988': {'inp_pmids': ['32384151'],\n",
       "  'title': 'Altered microRNA expression links IL6 and TNF-induced inflammaging with myeloid malignancy [miR146aKO_ESLAM_SCBS]',\n",
       "  'summary': 'Aging is associated with significant changes in the hematopoietic system, including increased inflammation, impaired hematopoietic stem cell (HSC) function, and increased incidence of myeloid malignancy. Inflammation of aging (“inflammaging”) has been proposed as a driver of age-related changes in HSC function and myeloid malignancy, but mechanisms linking these phenomena remain poorly defined. Here, we identify loss of miR-146a as driving aging-associated inflammation in AML patients. miR-146a expression declined in old wild-type mice, and loss of miR-146a promoted premature HSC aging and inflammation in young miR-146a-null mice, preceding development of aging-associated myeloid malignancy. Using single-cell assays of HSC quiescence, stemness, differentiation potential, and epigenetic state to probe HSC function and population structure, we found that loss of miR-146a depleted a subpopulation of primitive, quiescent HSCs. DNA methylation and transcriptome profiling implicated NF-κB, IL6, and TNF as potential drivers of HSC dysfunction, activating an inflammatory signaling relay promoting IL6 and TNF secretion from mature miR-146a-/- myeloid and lymphoid cells. Reducing inflammation by targeting Il6 or Tnf was sufficient to restore single-cell measures of miR-146a-/- HSC function and subpopulation structure, and reduced the incidence of hematological malignancy in miR 146a-/- mice. miR-146a-/- HSCs exhibited enhanced sensitivity to IL6 stimulation, indicating that loss of miR-146a affects HSC function via both cell-extrinsic inflammatory signals and increased cell-intrinsic sensitivity to inflammation. Thus, loss of miR 146a regulates cell-extrinsic and -intrinsic mechanisms linking HSC inflammaging to the development of myeloid malignancy.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 6},\n",
       " '200146264': {'inp_pmids': ['33309739'],\n",
       "  'title': 'Single cell RNA-seq of psoriatic skin identifies pathogenic Tc17 subsets and reveals distinctions between CD8+ T cells in autoimmunity and cancer',\n",
       "  'summary': 'Psoriasis is an inflammatory, IL-17-driven skin disease in which autoantigen-induced CD8+ T cells have been identified as pathogenic drivers. In this study, we used single-cell RNA-seq to identify 11 transcriptionally diverse CD8+ T cell subsets in psoriatic and healthy skin. Among several inflammatory subsets enriched in psoriatic skin, we observed two Tc17 subsets that were metabolically divergent, developmentally related, and expressed CXCL13, which we found to be a biomarker of psoriasis severity. Despite high co-inhibitory receptor expression in the Tc17 clusters, a comparison of these cells with melanoma-infiltrating CD8+ T cells revealed upregulated cytokine, cytolytic, and metabolic transcriptional activity in the psoriatic cells that differed from an exhaustion program. Our findings provide a high-resolution view of cutaneous CD8+ T cells in psoriasis and healthy skin and shed light on their functional distinctions within the chronic pathologies of autoimmunity and cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200146026': {'inp_pmids': ['32572264'],\n",
       "  'title': 'single cell RNA-seq analysis of ovarian cancer ascites',\n",
       "  'summary': 'To understand the cellular diversity within malignant ascites, we profiled ~11,000 single cells from 22 samples by Smartseq2 and by 10x.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200145669': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Transcriptome analysis of HSPCs and T cells from aplastic anemia (AA) and healthy donors',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200145668': {'inp_pmids': ['33763704'],\n",
       "  'title': \"Single cell transcriptome from aplastic anemia (AA) and healthy donors [3' RNA-seq]\",\n",
       "  'summary': 'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) and T cells collected from bone marrow and peripheral blood of healthy donors and aplastic anemia patients untreated or response to immunosuppressive therapy.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200145531': {'inp_pmids': ['33763704'],\n",
       "  'title': 'Single cell transcriptome from aplastic anemia (AA) and healthy donors [Full-length RNA-seq]',\n",
       "  'summary': 'Transcriptome analysis of hematopoietic stem and progenitor cells (HSPCs) from aplastic anemia (AA) and healthy donors',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200142259': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem [protophloem sieve element cells]',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200140977': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem II',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200140778': {'inp_pmids': ['34941412'],\n",
       "  'title': 'Cellular trajectory analysis links tissue maturation to cellular specialization in the plant meristem',\n",
       "  'summary': 'The mechanisms that allow cells in the plant meristem to coordinate tissue-wide maturation gradients with specialized cell networks are critical for indeterminate growth. Here, we combine long-term imaging, single-cell transcriptomics, genetic analysis, and network modeling to reconstruct the differentiation trajectory of the 20 protophloem cells in the Arabidopsis root. We found that lineage bifurcation and cellular specification are mediated near the stem cell niche by PEAR regulators. However, many PEAR downstream effects are repressed by a broad gradient of PLETHORA transcription factors, which directly inhibit PEARs’ own direct target APL until dissipating seven cells from the stem cell niche. Modeling and molecular epistasis suggest a so-called “seesaw” cascade of sharp transitions in which transcription factors regulating consecutive steps mutually inhibit each other’s targets, ending in programmed enucleation. This analysis provides a mechanistic understanding of how morphogen-like maturation gradients interface with cell-type specific transcriptional regulators to stage cellular differentiation',\n",
       "  'organism': 'blank sample; Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200140493': {'inp_pmids': ['35419551'],\n",
       "  'title': 'Multi-omic profiling of single nuclei uncovers regulatory diversity of brain cell types and diseases',\n",
       "  'summary': 'Most existing single-cell techniques can only make one type of molecular measurements. Although computational approaches have been developed to integrate single-cell datasets, their efficacy still needs to be determined with reference to authentic single-cell multi-omic profiles. To address this challenge, we devised single-nucleus methylCytosine, Chromatin accessibility and Transcriptome sequencing (snmC2T-seq) and applied the approach to post-mortem human frontal cortex tissue. We developed a computational framework to evaluate the quality of finely defined cell types using multi-modal information and validated the efficacy of computational multi-omic integration methods. Correlation analysis in individual cells revealed gene groups showing distinct relations between methylation and expression. Integration of snmC2T-seq with other multi- and single- modal datasets enabled joint analyses of the methylome, chromatin accessibility, transcriptome, and chromatin architecture for 63 human cortical cell types. We reconstructed the regulatory lineage of these cortical cell types and found pronounced cell-type-specific enrichment of disease risks for neuropsychiatric traits, predicting causal cell types that can be targeted for treatment.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Methylation profiling by high throughput sequencing; Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200137444': {'inp_pmids': ['38539015'],\n",
       "  'title': 'Stem cell derived human microglia transplanted in mouse brain to study human disease',\n",
       "  'summary': 'While genetics highlight the role of microglia in Alzheimer’s disease (AD), one third of putative AD-risk genes lack adequate mouse orthologs. Here, we successfully engraft human microglia derived from embryonic stem cells in the mouse brain. The cells recapitulate transcriptionally human primary microglia ex vivo and show expression of human specific AD-risk genes. Oligomeric Amyloid-β induces a divergent response in human vs. mouse microglia. This model can be used to study the role of microglia in neurological diseases.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200134646': {'inp_pmids': ['37958987'],\n",
       "  'title': 'Exploring the effects of rearing densities on epigenetic modifications in the zebrafish gonads',\n",
       "  'summary': 'The goal of this study was to determine whether there was a direct relationship between stress and DNA methylation changes in fish gonads. During sex differentiation, we studied the sensitive window to density. Sex ratios were studied and methylation of canonical reproduction-related genes was analysed.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200133922': {'inp_pmids': ['31801092'],\n",
       "  'title': 'Global transcriptomic profiling of the bone marrow stromal microenvironment during postnatal development, aging, and inflammation',\n",
       "  'summary': 'RNA-sequencing of a longitudinal comparison of four principal mesenchymal and endothelial stromal cell types (CXCL12-abundant reticular cells, PDGFR-α+ Sca-1+, sinusoidal and arterial endothelial cells), isolated from early postnatal, adult and aged mice. Additional transcriptional profiling of the response of CXCL12-abundant reticular cells and sinusoidal endothelial cells to infection-mimicking agents.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200130711': {'inp_pmids': ['31501549', '35419551'],\n",
       "  'title': 'Simultaneous profiling of 3D genome structure and DNA methylation in single human cells',\n",
       "  'summary': 'Dynamic 3D chromatin conformation is a critical mechanism for gene regulation during development and disease. Despite this, profiling of 3D genome structure from complex tissues with cell-type specific resolution remains challenging. Recent efforts have demonstrated that cell-type specific epigenomic features can be resolved in complex tissues using single-cell assays. However, it remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles can effectively identify cell types and reconstruct cell-type specific chromatin conformation maps. To address these challenges, we have developed single-nucleus methyl-3C sequencing (sn-m3C-seq) to capture chromatin organization and DNA methylation information and robustly separate heterogeneous cell types. Applying this method to >4,200 single human brain prefrontal cortex cells, we reconstruct cell-type specific chromatin conformation maps from 14 cortical cell types. These datasets reveal the genome-wide association between cell-type specific chromatin conformation and differential DNA methylation, suggesting pervasive interactions between epigenetic processes regulating gene expression.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 3},\n",
       " '200130633': {'inp_pmids': ['38287646'],\n",
       "  'title': 'Effects of losmapimod on antigen-specific response in aged skin',\n",
       "  'summary': 'Older humans are more susceptible to infections, even those to which they were immune to in their youth. Vaccination against many different infectious agents is also sub-optimal in these individuals. To investigate the mechanisms involved in the decline in immunity during ageing we inject safe derivatives of microorganisms into the skin of old and young volunteers. After injection of the agent, we can extract the white cells (leucocytes) that accumulate at the site of the immune response to test for their function. One of the responses we have been investigating is that to the varicella zoster virus (VZV) that induces chickenpox mainly in young individuals. Infected subjects control this persistent virus until an older age where reduced immunity leads to viral re-activation to cause shingles. We have found that this defective response is associated with increased baseline inflammation in this tissue. Although inflammatory responses are required to clear infections, excessive inflammation has previously been shown to interfere with the generation of specific immunity. We will therefore investigate ways of enhancing the immune response of older humans by blocking basal inflammation. To extend our initial observations of high background inflammation in the skin, we will investigate which cells in this tissue are responsible for the production of inflammatory mediators by using a second human experimental system that tests the response to a skin irritant known as cantharadin. Next we will block inflammation in the skin to determine whether this can lead to improved responses upon challenge with antigens. This will be achieved by pre-treating old subjects with a drug that has already been developed by GSK (Losmapimod, p38MAPkinase inhibitor) and another that is available off the shelf (vitamin D3). The GSK p38 inhibitors are currently being tested to prevent unwanted inflammation. A surprising observation we made was that p38 inhibition could also rejuvenate human T lymphocytes and enhance their ability to proliferate in vitro. Therefore the inhibition of p38 may block unwanted inflammatory response as well as enhance T lymphocyte reactivity. Vitamin D3 has also been shown to have anti-inflammatory effects and works in part by inhibiting p38MAPkinase activation. The use of 2 separate interventions in this project increases the likelihood of success in this study. While Losmapimod may be more specific in its anti-inflammatory action, the use of vitamin D3 is more cost effective as it is a cheap compound that is readily available. The desirable outcome of this study is that either one or other of these compounds, that older volunteers will be treated with, will boost their response to microbial antigen challenge in the skin. This will provide proof of concept data that will lead to the exciting possibility of enhancing immunity by inhibiting the increased baseline inflammatory responses that are found during ageing. This may be a strategy to enhance immune responses in general and specifically to enhance vaccination efficacy to various diseases that is less effective in older subjects.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200130051': {'inp_pmids': ['31076851'],\n",
       "  'title': 'Second-generation molecular subgrouping of medulloblastoma: an international meta-analysis of Group 3 and Group 4 subtypes',\n",
       "  'summary': 'We undertook a meta-analysis based on DNA methylation patterns of Group3/4 subgroup medulloblastoma from three published studies, alongside additional unpublished tumours (total n=1501).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200127893': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200127892': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (KW)',\n",
       "  'summary': 'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200127884': {'inp_pmids': ['31018141', '38539015'],\n",
       "  'title': 'The major risk factors for Alzheimer’s disease: Age, Sex and Genes, modulate the microglia response to Aβ plaques (CDEP)',\n",
       "  'summary': 'Microglia are involved in Alzheimer’s disease (AD) by adopting activated phenotypes. How ageing in the absence or presence of β-amyloid (Aβ) deposition in different brain areas affects this response and whether sex and AD risk genes are involved, remains however largely unknown. Here we analyzed the gene expression profiles of more than 10,000 individual microglia cells isolated from cortex and hippocampus of male and female AppNL-G-F at 4 different stages of Aβ deposition and in age-matched control mice. We demonstrate that microglia adopt two major activated states during normal aging and after exposure to amyloid plaques. One of the responses (activated response microglia, ARM) is enhanced in particular by amyloid plaques and is strongly enriched with AD risk genes. The ARM response is not homogeneous, as subgroups of microglia overexpressing MHC type II and tissue repair genes (Dkk2, Gpnmb, Spp1) are induced upon prolonged Aβ exposure. Microglia in female mice advance faster in the activation trajectories. Similar activated states were also found in a second AD model and in human brain. We demonstrate that abolishing the expression of Apoe, the major genetic risk factor for AD, impairs the establishment of ARMs, while the second microglia response type, enriched for interferon response genes, remains unaffected. Our data indicate that ARMs are the converging point of multiple AD risk factors.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200127711': {'inp_pmids': ['31792364'],\n",
       "  'title': 'Blood Biomarkers for Memory: Towards Early Detection of Risk for Alzheimer Disease, Pharmacogenomics, and Repurposed Drugs',\n",
       "  'summary': 'Short-memory dysfunction is a key early feature of Alzheimer Disease. Psychiatric patients may be at higher risk for memory dysfunction and subsequent Alzheimer due to the negative effects of stress and depression on the brain. We carried out longitudinal within-subject studies in male and female psychiatric patients to identify blood gene expression biomarkers that track short term memory as measured by the retention measure in the Hopkins Verbal Learning Test. These biomarkers were prioritized with a convergent functional genomics approach using previous evidence in the field implicating them in Alzheimer Disease. The top candidate biomarkers were tested in an independent cohort for ability to predict state short-term memory, and trait future positive neuropsychological testing for cognitive impairment',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200124391': {'inp_pmids': ['31501549'],\n",
       "  'title': 'Single-cell multi-omic profiling of chromatin conformation and DNA methylome',\n",
       "  'summary': 'Recent advances in the development of single cell epigenomic assays have facilitated the analysis of the gene regulatory landscapes in complex biological systems. Single-cell variations of methods such as DNA methylation-sequencing and ATAC-seq hold tremendous promise for delineating distinct cell types and identifying their critical cis-regulatory sequences. Emerging evidence in recent years has shown that in addition to cis-regulatory sequences, dynamic regulation of 3D chromatin conformation is a critical mechanism for the modulation of gene expressions during development and disease. While assays for the investigation of single-cell 3D chromatin structure have been developed, cell-type specific chromatin conformation in primary human tissues has not been extensively explored. It remains unclear whether single-cell Chromatin Conformation Capture (3C) or Hi-C profiles are suitable for cell type identification and allow the reconstruction of cell-type specific chromatin conformation maps. To address these challenges, we have developed a multi-omic method single-nucleus methyl-3C sequencing (sn-m3C) to profile chromatin conformation and DNA methylation from the same cell. We have shown that bulk m3C and sn-m3C accurately capture chromatin organization information and robustly separate mouse cell types. We have developed a fluorescent-activated nuclei sorting strategy based on DNA content that eliminates nuclei multiplets caused by crosslinking. The sn-m3C-seq method allows high-resolution cell-type classification using two orthogonal types of epigenomic information and the reconstruction of cell-type specific chromatin conformation maps.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 3},\n",
       " '200122497': {'inp_pmids': ['31125107'],\n",
       "  'title': 'Large-scale serum microRNA profiling for developing an esophageal squamous cell carcinoma detection model',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of esophageal squamous cell carcinoma.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200121750': {'inp_pmids': ['32084358'],\n",
       "  'title': 'Functional heterogeneity within the developing zebrafish epicardium',\n",
       "  'summary': 'The epicardium is essential during cardiac development, homeostasis and repair and yet fundamental insights into its underlying cell biology, notably epicardium formation, lineage heterogeneity and functional cross-talk with other cell types in the heart, is currently lacking. In this study, we investigated epicardial heterogeneity and the functional diversity of discrete epicardial subpopulations in the developing zebrafish heart. Single-cell RNA-sequencing uncovered three epicardial subpopulations with specific genetic programmes and distinctive spatial distribution within the developing heart. Perturbation of unique gene signatures uncovered distinct functions associated with each subpopulation and established novel epicardial roles in cell adhesion, migration, and chemotaxis as a mechanism for recruitment of leukocytes into the heart. This work elucidates the mutual spatiotemporal relationships between different epicardial subpopulations and assigns unique function to each during cardiac development. Understanding which mechanisms cells employ to establish a functional epicardium and to communicate with other cardiovascular cell types during development will bring us closer to repairing cellular relationships that are disrupted during cardiovascular disease.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200120584': {'inp_pmids': ['30820472', '31666070', '34686734'],\n",
       "  'title': 'Risk prediction models for dementia constructed by supervised principal component analysis using miRNA expression data',\n",
       "  'summary': 'Alzheimer’s disease (AD) is the most common subtype of dementia, followed by Vascular Dementia (VaD), and Dementia with Lewy Bodies (DLB). Recently, microRNAs (miRNAs) have received a lot of attention as the novel biomarkers for dementia. Here, using serum miRNA expression of 1,601 Japanese individuals, we investigated potential miRNA bio- markers and constructed risk prediction models, based on a supervised principal component analysis (PCA) logistic regression method, according to the subtype of dementia. The final risk prediction model achieved a high accuracy of 0.873 on a validation cohort in AD, when using 78 miRNAs: Accuracy = 0.836 with 86 miRNAs in VaD; Accuracy = 0.825 with 110 miRNAs in DLB. To our knowledge, this is the first report applying miRNA-based risk pre- diction models to a dementia prospective cohort. Our study demonstrates our models to be effective in prospective disease risk prediction; and with further improvement may contribute to practical clinical use in dementia.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 5},\n",
       " '200118723': {'inp_pmids': ['31002671'],\n",
       "  'title': 'Discovery and characterization of variance QTLs in human induced pluripotent stem cells',\n",
       "  'summary': 'Quantification of gene expression levels at the single cell level has revealed that gene expression can vary substantially even across a population of homogeneous cells. However, it is currently unclear what genomic features control variation in gene expression levels, and whether common genetic variants may impact gene expression variation. Here, we take a genome-wide approach to identify expression variance quantitative trait loci (vQTLs). To this end, we generated single cell RNA-seq (scRNA-seq) data from induced pluripotent stem cells (iPSCs) derived from 53 Yoruba individuals. We collected data for a median of 95 cells per individual and a total of 5,447 single cells, and identified 241 mean expression QTLs (eQTLs) at 10% FDR, of which 82% replicate in bulk RNA-seq data from the same individuals. We further identified 14 vQTLs at 10% FDR, but demonstrate that these can also be explained as effects on mean expression. Our study suggests that dispersion QTLs (dQTLs), which could alter the variance of expression independently of the mean, have systematically smaller effect sizes than eQTLs. We estimate that at least 300 cells per individual and 400 individuals would be required to have modest power to detect the strongest dQTLs in iPSCs. These results will guide the design of future studies on understanding the genetic control of gene expression variance.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 3},\n",
       " '200117064': {'inp_pmids': ['31136284'],\n",
       "  'title': 'Serum miRNA-based risk prediction for stroke',\n",
       "  'summary': 'The serum miRNAs were correlated with predicted the incidence of stroke.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200116672': {'inp_pmids': ['30530648', '31820734'],\n",
       "  'title': 'In vivo molecular signatures of severe dengue infection revealed by viscRNA-Seq',\n",
       "  'summary': 'Dengue virus infection can result in severe symptoms including shock and hemorrhage, but an understanding of the molecular correlates of disease severity is lacking. Bulk transcriptomics on blood samples are difficult to interpret because the blood is composed of different cell types that may react differently to virus infection. Dengue virus RNA can be detected in human plasma, however identifying the cells carrying dengue virus through the bloodstream in vivo has proven challenging. Here we used our recently developed viscRNA-Seq approach to profile transcriptomes of thousands of single blood peripheral mononuclear cells from 6 human subjects with dengue fever and severe dengue, as well as to characterize the cell types associated with dengue virus in the human blood. We found that although no bulk transcriptome marker for severe dengue exists, the expression of MX2 in naive B cells, of CD163 in CD14+/CD16+ monocytes and of other genes in specific cell types is highly predictive for severe dengue. We detected virus-associated cells in the blood of two severe dengue patients with high viral load and discovered the majority of these to be B cells expressing germline IgM or IgD immunoglobulin chains and naive markers but also showing signs of activation and expression of CD69, CXCR4, and other surface receptors. In bystander B cells we detected signs of strong immune activation, parallel hypersomatic evolution and, in one severe degue subject, an anomalously large clone of highly mutated, IgG1 plasmablasts that could be reactive to dengue virus. This study presents a high-resolution molecular exploration into dengue virus infection in humans and can be generalized to any RNA virus.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200115513': {'inp_pmids': ['26740022'],\n",
       "  'title': 'miRNA and colorectal cancer: associations with tumor phenotype and survival',\n",
       "  'summary': 'MicroRNAs (miRNA) are a class of small regulatory RNAs that mediate post-transcriptional silencing of specific target mRNAs. Data suggest the importance of miRNAs to cancer development and possibly to survival. Our overall hypothesis is that miRNA expression is unique to tumor molecular phenotype; that miRNA expression levels at time of diagnosis predicts survival; and that miRNA expression is associated with inflammation-related genetic and lifestyle factors key to colorectal cancer (CRC). This study takes a two pronged approach to addressing our hypotheses. While we propose to validate previously identified miRNAs that have been identified as associated with CRC (either by differential expression or from assessment of mutations), we will add to the field through discovery of new and important associations that may be unique to specific molecular phenotypes, to polyp to cancer progression, and to survival. We have analyzed the expression of 2006 human miRNAs using data derived from tumor and paired normal tissue at time of diagnosis from: 1975 people with incident colon cancer or rectal cancer and 290 polyps from colon and rectal cases (included in this study) who reported a prior polyp. MiRNA was obtained from dissected paraffin-embedded tissue and assessed using an Agilent microarray platform. We intend to extend our validation of previously identified mutated miRNAs and differentially expressed miRNAs to determine if these alterations are associated with specific tumor molecular phenotype (TP53, KRAS2, CIMP+, and for colon tumors MSI+), inflammation-related factors, clinical factors and survival. We will identify associations with miRNAs that are related to specific molecular phenotypes, with drivers in the carcinogenic process, and with clinical features and survival. These miRNAs will be validated using targeted Agilent Platform. Associations will be tested based on differential expression for both individual and groups of miRNAs using recent extensions of several statistical methods including ANOVA, logistic regression, and Cox proportional hazards models. Our sample size allows for both a training and validation component, and provides sufficient statistical power to meet the study goals. MiRNAs that are differentially expressed in polyps and in subsequent tumors will provide new insights into targets for screening and treatment as well as provide support that miRNAs function as the “driver” in the carcinogenic process. Testing of mutated miRNAs identified from sequencing in conjunction with tumor phenotype, clinical, and survival data will further validate the importance of these miRNAs, and provide insight as to which CRC molecular pathway the miRNAs function. Our rich dataset of lifestyle, genetic, clinical and prognosis, and tumor molecular phenotype on 1975 CRC and paired normal tissue allows us to examine factors that are associated with miRNA expression and mutation in a large sample of population-based cases in the most cost-efficient manner possible. The miRNAs identified in these analyses will elucidate pathways important in the etiology of CRC and will provide insight into potential targets for screening and treatment. MiRNAs are small, non-protein-coding RNA molecules that regulate gene expression either by post-transcriptionally suppressing mRNA translation or by mRNA degradation. We examine differentially expressed miRNAs in colorectal carcinomas, adenomas, and normal colonic mucosa. Data come from population-based studies of colorectal cancer conducted in Utah and the Kaiser Permanente Medical Care Program. A total of 1893 carcinoma/normal paired samples and 290 adenoma tissue samples were run on the Agilent Human miRNA Microarray V19.0 which contained 2006 miRNAs. We tested for significant differences in miRNA expression between paired carcinoma/adenoma/normal colonic tissue samples. Fewer than 600 miRNAs were expressed in >80% of people for colonic tissue; of these 86.5% were statistically differentially expressed between carcinoma and normal colonic mucosa using a False Discovery Rate of 0.05. Roughly half of these differentially expressed miRNAs showed a progression in levels of expression from normal to adenoma to carcinoma tissue. Other miRNAs appeared to be altered at the normal to adenoma stage, while others were only altered at the adenoma to carcinoma stage or only at the normal to carcinoma stage. Evaluation of the Agilent platform showed a high degree of repeatability (r=0.98) and reasonable agreement with the NanoString platform. Our data suggest that miRNAs are highly dysregulated in colorectal tissue among individuals with colorectal cancer; the pattern of disruption varies by miRNA as tissue progresses from normal to adenoma to carcinoma.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200113740': {'inp_pmids': ['32025611'],\n",
       "  'title': 'High-sensitive circulating microRNA panel for early detection of hepatocellular carcinoma',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detection of hepatocellular carcinoma. Keywords: Non-coding RNA profiling by array',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200109159': {'inp_pmids': ['29576475'],\n",
       "  'title': 'Smart-seq2 analysis of larval zebrafish habenula from the gng8-GFP transgenic line',\n",
       "  'summary': 'The identification of cell types and marker genes is critical for dissecting neural development and function, but the size and complexity of the brain has hindered the comprehensive discovery of cell types. We combined single-cell RNA-seq with anatomical brain registration to create a comprehensive map of the zebrafish habenula, a conserved forebrain hub involved in pain processing and learning. Single-cell transcriptomes of ~13000 habenular cells (>4x coverage) identified 18 neuronal types and dozens of marker genes. Registration of marker genes onto a common reference atlas created a rich resource for anatomical and functional studies and enabled the mapping of active neurons onto neuronal types following aversive stimuli. Strikingly, despite brain growth and functional maturation, cell types were retained between the larval and adult habenula. This study provides a gene expression atlas to dissect habenular development and function and offers a general framework for the comprehensive characterization of other brain regions.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200109158': {'inp_pmids': ['29576475'],\n",
       "  'title': '10X genomics analysis of adult and larval habenula from the gng8-GFP transgenic line',\n",
       "  'summary': 'The identification of cell types and marker genes is critical for dissecting neural development and function, but the size and complexity of the brain has hindered the comprehensive discovery of cell types. We combined single-cell RNA-seq with anatomical brain registration to create a comprehensive map of the zebrafish habenula, a conserved forebrain hub involved in pain processing and learning. Single-cell transcriptomes of ~13000 habenular cells (>4x coverage) identified 18 neuronal types and dozens of marker genes. Registration of marker genes onto a common reference atlas created a rich resource for anatomical and functional studies and enabled the mapping of active neurons onto neuronal types following aversive stimuli. Strikingly, despite brain growth and functional maturation, cell types were retained between the larval and adult habenula. This study provides a gene expression atlas to dissect habenular development and function and offers a general framework for the comprehensive characterization of other brain regions.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200106817': {'inp_pmids': ['30333487', '39587714'],\n",
       "  'title': 'Integrated extracellular microRNA profiling for ovarian cancer screening',\n",
       "  'summary': 'A serum miRNA combination could be a powerful classifier for the detEsophageal Cancertion of patients with ovarian cancer.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200105115': {'inp_pmids': ['29576475'],\n",
       "  'title': 'Comprehensive identification and spatial mapping of habenular neuronal types using single cell RNAseq',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200103708': {'inp_pmids': ['30333487'],\n",
       "  'title': 'The profile of exosomal and cellular miRNA in ovarian cancer cell lines',\n",
       "  'summary': 'To check the profile of exosomal and cellular miRNA in ovarian cancer cell lines, total RNA were extracted from exosomes and cells. Thirteen ovarian cancer cell lines (A2780, ES-2, CAOV3, SKOV3, OV-90, OAW42, MCAS, COV362, RMG-1, RMUG-S, KURAMOCHI, NIH-OVCAR3 and A2780cis) were investigated, and HOSE1, HOSE2 and HOSE3 (human ovarian surface epithelim cell lines) were used as control.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200102248': {'inp_pmids': ['29462153'],\n",
       "  'title': 'RNA-seq of Arabidopsis thaliana seedlings growing in the presence of a bacterial synthetic community under full and low phosphate conditions',\n",
       "  'summary': 'This study evaluates the transcriptome of Arabidopsis thaliana seedlings growing in the presence of a diversity of synthetic bacteria communities under different phosphate availability.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200099962': {'inp_pmids': ['29550329'],\n",
       "  'title': 'The biological age linked with oxidative estress modifies breast cancer aggressiveness',\n",
       "  'summary': 'The incidence of breast cancer increases with age until menopause, and breast cancer is more aggressive in younger women. The existence of epidemiological links between breast cancer and aging indicates that both processes possibly share some common mechanisms of development. Oxidative stress is associated with both cancer susceptibility and aging. Here we observed that ERBB2-positive breast cancer, developed from genetically different ERBB2-positive transgenic mice generated by a backcross, is more aggressive in younger than in older mice, similar to what occurs in humans. In this cohort, we estimated the oxidative biological age using a mathematical model that integrated several subphenotypes directly or indirectly related to oxidative stress. This model included the serum levels of HDL-cholesterol and magnesium that were determined at a disease-free stage, and total AKT1 and glutathione concentrations in the liver at the time of necropsy. Later we defined the grade of aging due to oxidative stress as the increment of aging, which was the difference between the predicted biological age and the chronological age. This comparison permitted the identification of the biologically younger mice with less oxidative stress and older mice with more oxidative stress than would be expected according to their chronological age. Interestingly, biologically older mice developed more aggressive breast cancer than the biologically younger mice. We identified genomic regions on chromosomes 2 and 15 that were linked to the grade of oxidative aging. The levels of expression of Zbp1 located on chromosome 2, a gene related to necroptosis and inflammation, positively correlated with the grade of oxidative aging and tumour aggressiveness. This study shows part of the complex interactions between breast cancer and aging.',\n",
       "  'organism': 'Mus musculus; Mus',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200097693': {'inp_pmids': ['30498128'],\n",
       "  'title': 'Single-cell Multi-omics Sequencing and Analyses of Human Colorectal Cancer',\n",
       "  'summary': 'Although genomic instability, epigenetic abnormality, and gene expression dysregulation are hallmarks of colorectal cancer, these features have not been simultaneously analyzed at single-cell resolution. Using optimized single-cell multi-omics sequencing together with multi-regional sampling of the primary tumor, lymphatic and distant metastases, we provide insights beyond intratumoral heterogeneity. Genome-wide DNA methylation levels were relatively consistent within a single genetic sub-lineage. The genome-wide DNA demethylation patterns of cancer cells were consistent in all 10 sequenced patients. Our work demonstrates the feasibility of reconstructing genetic lineages, and tracing their epigenomic and transcriptomic dynamics with single-cell multi-omics sequencing.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200097179': {'inp_pmids': ['35419551'],\n",
       "  'title': 'Single Cell Methylome Signatures Reveal Distinct Neuronal Populations and Regulatory Elements in the Mammalian Cortex',\n",
       "  'summary': 'The mammalian brain contains diverse neuron cell types with distinct physiological, morphological and molecular characteristics. However, a comprehensive assessment of the epigenetically distinct neuronal classes is currently missing. Cytosine DNA methylation is a stable epigenetic mark that distinguishes neuron types and marks gene regulatory elements. We developed an efficient single-nucleus methylome sequencing approach that allows robust high-throughput neuron-type classification. We generated >6,000 single-nucleus methylomes and identified 16 mouse and 21 human neuronal subpopulations in the frontal cortex. Both CG and non-CG methylation exhibited cell type-specific distributions that recapitulate and extend findings from single neuron transcriptome profiling. Moreover, we found approximately 500,000 neuron-type-specific regulatory elements showing strong differential methylation in mouse and human cortex. Distinct methylation signatures identified an unique human Parvalbumin-expressing inhibitory sub-type and a layer 6 specific excitatory sub-type in mouse. Comparative epigenomic analysis showed stronger conservation of gene regulatory elements in inhibitory compared with excitatory neurons. These findings demonstrate the utility of single nucleus methylome profiling for both expanding the atlas of brain cell types and identifying regulatory elements that potentially drive these differences.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 5},\n",
       " '200096066': {'inp_pmids': ['30320226'],\n",
       "  'title': 'Probing Plasmodium falciparum sexual differentiation at the single cell level',\n",
       "  'summary': 'Malaria parasites go through major transitions during their complex life cycle, yet the underlying differentiation pathways remain obscure. Here we apply single cell transcriptomics to unravel events that initiate sexual development in preparation for transmission of the parasite from human to mosquito. This proof-of-concept study provides a template to capture transcriptional diversity in heterogeneous parasite populations, with major implications for our understanding of parasite biology and the ongoing malaria elimination campaign.',\n",
       "  'organism': 'Plasmodium falciparum NF54',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 3},\n",
       " '200088798': {'inp_pmids': ['29794063'],\n",
       "  'title': 'Time-resolved transcriptome analysis with genetic perturbations reveals a critical time window for effective plant immunity',\n",
       "  'summary': 'Pathogen invasion in plants is associated with transcriptional reprogramming. Enigmatically, plants induce similar transcriptome responses upon infection by virulent or avirulent pathogens. This renders the importance of transcriptional reprogramming for immunity obscure. Here, using RNA-seq, we generate time-series transcriptome data coupled with genetic perturbations to reveal temporal dynamics upon infection by virulent or avirulent strains of a bacterial pathogen, Pseudomonas syringae, in Arabidopsis thaliana. Fast and sustained transcriptional reprogramming occurs upon infection with avirulent strains while virulent strain infection leads to a slower response with comparable gene expression patterns and magnitudes. Importantly, transcriptome analysis of resistant and susceptible mutants responding to avirulent strains links delayed transcriptional reprogramming to compromised immunity. Taken together, our results pinpoint the early critical time window of transcriptional reprogramming for establishing effective immunity against the bacterial pathogen.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200087162': {'inp_pmids': ['30322904'],\n",
       "  'title': 'RNA-seq of Solanum lycopersicum cv. M82 and S. pennellii introgression line 3-5 day old root tip tissue',\n",
       "  'summary': 'The goal of this study was to perform RNA-seq expression analysis on Solanum lycopersicum cv. M82 X S. pennellii introgression lines, deriving expression Quantitative Trait Loci which were analyzed together with pre-existing genomic and phenotypic data to define genes and regulatory pathways controlling tomato root development and observed natural variation. We completed the RNAseq expression profiling analysis and developed a tool to display this information graphically in collaboration with Nicholas Provart at the University of Toronto: http://bar.utoronto.ca/efp_tomato/cgi-bin/efpWeb.cgi?dataSource=ILs_Root_Tip_Brady_Lab To identify candidate genes and pathways we focussed on one root growth trait, root growth angle, and identified two statistically significant genomic regions within tomato root growth angle QTL containing two candidate genes that likely control the gravitropic setpoint angle (CDC73 and PAP27), both of which are conserved between Arabidopsis and tomato, and which we tested using transgenic lines of the Arabidopsis orthologs. A possible regulatory role for suberin in root growth angle control was also identified.',\n",
       "  'organism': 'Solanum lycopersicum; Solanum lycopersicum x Solanum pennellii; Solanum pennellii',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200084422': {'inp_pmids': ['27799057'],\n",
       "  'title': \"Molecular Signatures Underlying Selective Regional Vulnerability to Alzheimer's Disease\",\n",
       "  'summary': \"Alzheimer's disease (AD) is the most common form of dementia, characterized by progressive cognitive impairment and neurodegeneration as a result of abnormal neuronal loss. To elucidate the molecular systems associated with AD, we characterized the gene expression changes associated with multiple clinical and neuropathological traits in 1,053 postmortem brain samples across 19 brain regions from 125 persons dying with varying severities of dementia and variable AD-neuropathology severities.\",\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200080744': {'inp_pmids': ['35440059'],\n",
       "  'title': 'Epigenomic and genome structural diversity in a worldwide collection of Arabidopsis thaliana',\n",
       "  'summary': 'Epigenetic variation can impact gene transcription and may play roles in phenotypic diversity and adaptation. Here we report 1,107 high quality single-base resolution methylomes, and 1,210 transcriptomes from the 1001 Arabidopsis Genomes population. Analyses reveal strong effects of geographic origin on average DNA methylation levels, alterations of gene expression by epialleles and a highly complex genetic basis for DNA methylation. Physical genome maps for nine of the most diverse accessions revealed how transposable elements and other structural variations shaped the epigenome to allow rapid adaptation to environmental changes, with strong emphasis on disease resistance. Analysis of the cistromes and epicistromes in these accessions revealed a significant association between both methylation and nucleotide variation and the conservation of transcription factor binding sites. The Arabidopsis thaliana 1001 Epigenomes Project now provides a comprehensive resource to help further understand how epigenetic variation contributes to both molecular and phenotypes in natural populations of the most widely studied reference plant.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200076381': {'inp_pmids': ['27716510'],\n",
       "  'title': 'Single Cell RNA-seq Study of Midbrain and Dopaminergic Neuron Development in Mouse, Human, and Stem Cells',\n",
       "  'summary': 'In order to get a better molecular understanding of human midbrain development, this study defines cell types of the ventral midbrain in both human and mouse as well as stem cell-derived dopaminergic neuron preparations, and reveals the temporal dynamics of key lineages across development and the adult.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 5},\n",
       " '200073002': {'inp_pmids': ['26749252'],\n",
       "  'title': 'A novel combination of serum microRNAs for detecting breast cancer in the early stage',\n",
       "  'summary': 'MicroRNAs (miRNAs), which are stably present in serum, have been reported to be potentially useful for detecting cancer. In the present study, we examined the expression profiles of serum miRNAs in large cohorts to identify the miRNAs that can be used to detect breast cancer in the early stage. We comprehensively evaluated serum miRNA expression profiles using highly sensitive microarray analysis. A total of 1,280 serum samples of breast cancer patients stored in the National Cancer Center Biobank were used. Additionally, 2,836 serum samples were obtained from non-cancer controls and 514 from patients with other types of cancers or benign diseases. The samples were divided to a training cohort including non-cancer controls, other cancers and breast cancer and a test cohort including non-cancer controls and breast cancer. The training cohort was used to identify a combination of miRNAs that detect breast cancer, and the test cohort was used to validate that combination. miRNA expression was compared between breast cancer and non-breast cancer serum , and a combination of five miRNAs (miR-1246, miR-1307-3p, miR-4634, miR-6861-5p, and miR-6875-5p) was found to detect breast cancer. This combination had a sensitivity of 97.3%, specificity of 82.9%, and accuracy of 89.7% for breast cancer in the test cohort Additionally, the combination could detect breast cancer in the early stage (sensitivity of 98.0% for T0).',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Non-coding RNA profiling by array',\n",
       "  'cluster_id': 1},\n",
       " '200072698': {'inp_pmids': ['26566685'],\n",
       "  'title': 'Site-specific methylation and acetylation of lysine residues in the C-terminal domain (CTD) of RNA polymerase II',\n",
       "  'summary': 'Dynamic modification of the carboxy-terminal domain (CTD) of RNA polymerase II (RNAPII) regulates transcription-coupled processes in eukaryotes. The CTD in mammals is composed of 52 heptad-repeats with the consensus sequence Y1-S2-P3-T4-S5-P6-S7. Repeats in the distal part of CTD deviate from the consensus sequence and have frequently replaced serine at position 7 by lysine residues (K7). Mass spectrometry analysis revealed modification of K7 residues in heptad-repeats 39, 42, and 47/49 by acetylation and in heptad-repeats 38, 39, 40, 42, and 47/49 by mono-, di-, or trimethylation. Notably, acetylated as well as di- and tri-methylated K7 residues were found exclusively in phosphorylated CTD peptides, while mono-methylated K7 residues occurred also in non-phosphorylated CTD peptides. Methylation of K7 residues was further studied with the monoclonal antibody (mAb) 1F5, which recognizes mono- and di-methylated K7 in CTD. ChIP experiments revealed high levels of K7 methylation at the transcriptional start site of genes. The low levels of K7 methylation in the body of genes results from impairment of epitope recognition in hyper-phosphorylated CTD by mAb 1F5. In agreement with this notion, phosphatase treatment of hyper-phosphorylated CTD or treatment of cells with kinase inhibitor flavopiridol restored the reactivity of mAb 1F5 towards methylated K7 residues in CTD. We conclude that methylation and acetylation of K7 residues further expand the mammalian CTD code and potentially contribute to regulation of gene expression.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200066017': {'inp_pmids': ['25939354'],\n",
       "  'title': 'Effect of Chromomethylase-2 and Dicer-like-3 knockouts on DNA methylation',\n",
       "  'summary': 'Whole genome bisulfite sequencing was performed on Chromomethylase-2 and Dicer-like-3 knockout mutants in order to confirm the results from genome wide association mapping and to identify the respective genomic regions that they target.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200065685': {'inp_pmids': ['25939354'],\n",
       "  'title': 'Population epigenomics of Swedish Arabidopsis',\n",
       "  'summary': 'Plants display remarkable developmental and phenotypic plasticity in order to adapt to their environment. It has long been postulated that epigenetics plays a key role in these processes, but with one or two exceptions, solid evidence for the role of epigenetic variation in these processes is lacking. A key impediment to understanding these processes is the lack of information on the extent of epigenetic variation and how it relates to genetic and phenotypic variation in natural population, both over the lifecycle of an individual, and over evolutionary time. Here we show that genetic variants under selection in the north of Sweden appear to drive variation in DNA methylation, which in turn is highly correlated with local climate. Selective sweeps and genetic variants associated with adaptation to the local environment have previously been identified within the Swedish Arabidopsis population. Our finding that they harbour variants responsible for climate associated epigenetic variation strongly supports the role of epigenetic processes in local adaptation. These findings provide a basis for further dissecting the role of epigenetics in local adaptation at the molecular level',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200063945': {'inp_pmids': ['26280576'],\n",
       "  'title': 'Reinstating transcriptome plasticity and memory function in mouse models for cognitive decline',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing; Expression profiling by high throughput sequencing; Non-coding RNA profiling by high throughput sequencing',\n",
       "  'cluster_id': 0},\n",
       " '200063944': {'inp_pmids': ['26280576'],\n",
       "  'title': 'small RNA-Seq of the CA1 hippocampal subregion from 3 month and 20 month old animals treated orally with vehicle or SAHA',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Non-coding RNA profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200063943': {'inp_pmids': ['26280576'],\n",
       "  'title': 'mRNA-Seq of the CA1 hippocampal subregion and liver from 3 month and 20 month old animals treated orally with vehicle or SAHA and mRNA-Seq of CA1 of 10 month old WT or APP/PS1-21 transgenic animals treated orally with vehicle or SAHA',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200063942': {'inp_pmids': ['26280576'],\n",
       "  'title': 'Genome-wide analysis of H4K12 acetylation in neuronal and non-neuronal nuclei in young animals treated with vehicle, WT littermates of APP/PS1-21 animals treated with vehicle and aged and APP/PS1-21 animals treated with oral vehicle or SAHA for 4 weeks',\n",
       "  'summary': \"Aging and increased amyloid burden are major risk factors for cognitive diseases such as Alzheimer's Disease (AD). An effective therapy does not yet exist. Here we use mouse models for age-associated memory impairment and amyloid deposition to study transcriptome and cell type-specific epigenome plasticity at the systems level in the brain and in peripheral organs. We show that at the level of epigenetic gene-expression aging and amyloid pathology are associated with inflammation and impaired synaptic function in the hippocampal CA1 region. While inflammation is associated with increased gene-expression that is linked to a subset of transcription factors, de-regulation of plasticity genes is mediated via different mechanisms in the amyloid and the aging model. Amyloid pathology impairs histone-acetylation and decreases expression of plasticity genes while aging affects differential splicing that is linked to altered H4K12 acetylation at the intron-exon junction in neurons but not in non-neuronal cells. We furthermore show that oral administration of the clinically approved histone deacetylase inhibitor Vorinostat not only restores spatial memory, but also exhibits an anti-inflammatory action and reinstates epigenetic balance and transcriptional homeostasis at the level of gene expression and exon usage. This is the first systems-level investigation of transcriptome plasticity in the hippocampal CA1 region in aging and AD models and of the effects of an orally dosed histone deacetylase inhibitor. Our data has important implications for the development of minimally invasive and cost-effective therapeutic strategies against age-associated cognitive decline. In fact, our data strongly suggest to test Vorinostat in patients suffering from AD.\",\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "  'cluster_id': 7},\n",
       " '200063203': {'inp_pmids': ['35172154'],\n",
       "  'title': 'A restricted period for formation of Outer Subventricular Zone defined by Cdh1 and Trnp1 levels',\n",
       "  'summary': 'The Outer Subventricular Zone (OSVZ) is a germinal layer playing key roles in the development of the neocortex, with particular relevance in gyrencephalic species like human and ferret where it contains abundant basal Radial Glia Cells (bRGCs) that promote cortical expansion. Here we identify a brief period in ferret embryonic development when apical RGCs generate a burst of bRGCs that become founders of the OSVZ. After this period, bRGCs in the OSVZ proliferate and self-renew exclusively locally, thereby forming a self-sustained lineage independent from the other germinal layers. The time window for the brief period of OSVZ bRGC production is delineated by the coincident down-regulation of Cdh1 and Trnp1, and their up-regulation reduces bRGC production and prevents OSVZ seeding. This mechanism in cortical development may have key relevance in brain evolution and disease',\n",
       "  'organism': 'Mustela putorius furo',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 5},\n",
       " '200060687': {'inp_pmids': ['35172154'],\n",
       "  'title': 'Sharp changes in gene expression levels along germinal layers distinguish the development of gyrencephaly',\n",
       "  'summary': 'Gyrencephalic species develop folds in the cerebral cortex in a stereotypic manner, but the genetic mechanisms underlying this unique patterning process are unknown. We present a large-scale transcriptomic analysis of individual germinal layers in the developing cortex of the gyrencephalic ferret, comparing between regions prospective of fold and fissure. We find unique transcriptional signatures in each germinal compartment, indicating that thousands of genes are differentially expressed between regions, including ~80% of genes mutated in human cortical malformations. Regional differences result from sharp changes in gene expression levels, which occur at multiple locations across the developing cortex of ferret and human, but not in the lissencephalic mouse. These complex expression patterns emerge sequentially during development and map the eventual location of specific folds and fissures. Our findings demonstrate that step-wise gene expression maps within germinal layers distinguish the development of gyrencephaly from lissencephaly, and may contribute to identify novel genes responsible for human malformations.',\n",
       "  'organism': 'Mustela putorius furo',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 5},\n",
       " '200060202': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Fathead Minnow (Pimephales promelas) and Zebrafish (Danio rerio)',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Pimephales promelas',\n",
       "  'exp_type': 'Expression profiling by array; Third-party reanalysis',\n",
       "  'cluster_id': 0},\n",
       " '200060167': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Zebrafish (Danio rerio)',\n",
       "  'summary': 'Fathead minnow and zebrafish are among the most intensively studied fish species in environmental toxicogenomics. To aid the assessment and interpretation of subtle transcriptomic effects from treatment conditions of interest, there needs to be a better characterization and understanding of the natural variation in gene expression among fish individuals within populations. Little effort, however, has been made in this area. Leveraging the transcriptomics data from a number of our toxicogenomics studies conducted over the years, we conducted a meta-analysis of nearly 600 microarrays generated from the ovary tissue of untreated, reproductively mature fathead minnow and zebrafish samples. As expected, there was considerable batch-to-batch transcriptomic variation; this “batch-effect” appeared to impact the fish transcriptomes randomly. The overall level of variation within-batch was quite low in fish ovary tissue, making it a suitable system for studying chemical stressors with subtle biological effects. The within-batch variation, however, differed considerably among individual genes and molecular pathways. This difference in variability is probably both technical and biological, thus suggesting a need to take into account both the expression levels and variance in evaluating and interpreting the transcriptional impact on genes and pathways by experimental conditions. There was significant conservation of both the genomes and transcriptomes between fathead minnow and zebrafish. The conservation to such a degree would enable not only a comparative biology approach in studying the mechanisms of action underlying environmental stressors, but also effective sharing of a large amount of existing public transcriptomics data for future development of toxicogenomics applications.',\n",
       "  'organism': 'Danio rerio',\n",
       "  'exp_type': 'Third-party reanalysis; Expression profiling by array',\n",
       "  'cluster_id': 2},\n",
       " '200060126': {'inp_pmids': ['25493933'],\n",
       "  'title': 'Natural Variation in Fish Transcriptomes: Comparative Analysis of the Fathead Minnow (Pimephales promelas)',\n",
       "  'summary': 'Fathead minnow and zebrafish are among the most intensively studied fish species in environmental toxicogenomics. To aid the assessment and interpretation of subtle transcriptomic effects from treatment conditions of interest, there needs to be a better characterization and understanding of the natural variation in gene expression among fish individuals within populations. Little effort, however, has been made in this area. Leveraging the transcriptomics data from a number of our toxicogenomics studies conducted over the years, we conducted a meta-analysis of nearly 600 microarrays generated from the ovary tissue of untreated, reproductively mature fathead minnow and zebrafish samples. As expected, there was considerable batch-to-batch transcriptomic variation; this “batch-effect” appeared to impact the fish transcriptomes randomly. The overall level of variation within-batch was quite low in fish ovary tissue, making it a suitable system for studying chemical stressors with subtle biological effects. The within-batch variation, however, differed considerably among individual genes and molecular pathways. This difference in variability is probably both technical and biological, thus suggesting a need to take into account both the expression levels and variance in evaluating and interpreting the transcriptional impact on genes and pathways by experimental conditions. There was significant conservation of both the genomes and transcriptomes between fathead minnow and zebrafish. The conservation to such a degree would enable not only a comparative biology approach in studying the mechanisms of action underlying environmental stressors, but also effective sharing of a large amount of existing public transcriptomics data for future development of toxicogenomics applications.',\n",
       "  'organism': 'Pimephales promelas',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 2},\n",
       " '200054680': {'inp_pmids': ['25939354'],\n",
       "  'title': 'DNA methylation variation in Arabidopsis has a genetic basis and appears to be involved in local adaptation',\n",
       "  'summary': 'We investigated DNA methylation variation in Swedish Arabidopsis thaliana accessions. We found that methylation of transposable elements is temperature sensitive and associated with genetic polymorphism in both cis and trans, whereas gene body methylation is associated with genetic polymorphism in trans. Additionally, complementary RNA-Seq data for the Arabidopsis accessions were used to correlate methylation changes with gene expression across environments.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200054362': {'inp_pmids': ['25340342'],\n",
       "  'title': 'Effects of Changes in Food Supply at the Time of Sex Differentiation on the Gonadal Transcriptome of Juvenile Fish. Implications for Natural and Farmed Populations',\n",
       "  'summary': 'Background: Food supply is a major factor influencing growth rates in animals. This has important implications for both natural and farmed fish populations, since food restriction may difficult reproduction. However, a study on the effects of food supply on the development of juvenile gonads has never been transcriptionally described in fish. Methods and Findings: This study investigated the consequences of growth on gonadal transcriptome of European sea bass in: 1) 4-month-old sexually undifferentiated fish, comparing the gonads of fish with the highest vs. the lowest growth, to explore a possible link between transcriptome and future sex, and 2) testis from 11-month-old juveniles where growth had been manipulated through changes in food supply. The four groups used were: i) sustained fast growth, ii) sustained slow growth, iii) accelerated growth, iv) decelerated growth. The transcriptome of undifferentiated gonads was not drastically affected by initial natural differences in growth. Further, changes in the expression of genes associated with protein turnover were seen, favoring catabolism in slow-growing fish and anabolism in fast-growing fish. Moreover, while fast-growing fish took energy from glucose, as deduced from the pathways affected and the analysis of protein-protein interactions examined, in slow-growing fish lipid metabolism and gluconeogenesis was favored. Interestingly, the highest transcriptomic differences were found when forcing initially fast-growing fish to decelerate their growth, while accelerating growth of initially slow-growing fish resulted in full transcriptomic convergence with sustained fast-growing fish. Conclusions: Food availability during sex differentiation shapes the juvenile testis transcriptome, as evidenced by adaptations to different energy balances. Remarkably, this occurs in absence of major histological changes in the testis. Thus, fish are able to recover transcriptionally their testes if they are provided with enough food supply during sex differentiation; however, an initial fast growth does not represent any advantage in terms of transcriptional fitness if later food becomes scarce.',\n",
       "  'organism': 'Dicentrarchus labrax',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200054292': {'inp_pmids': ['25939354'],\n",
       "  'title': 'DNA methylation variation in Arabidopsis has a genetic basis and appears to be involved in local adaptation',\n",
       "  'summary': 'Here we investigate DNA methylation variation in Swedish Arabidopsis thaliana accessions, demonstrating that methylation of transposable elements is temperature sensitive and associated with genetic polymorphism in both cis and trans, whereas gene body methylation is highly correlated with climate of origin and associated with genetic polymorphism in trans that shows evidence of local adaptation. While genome-wide surveys of naturally occurring DNA methylation have been published previously, the degree of genetic control revealed here is unprecedented. Furthermore, the observation that DNA methylation is associated with climate, and is apparently adaptively important, is completely novel.',\n",
       "  'organism': 'Arabidopsis thaliana',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200047798': {'inp_pmids': ['24223949'],\n",
       "  'title': 'Perinatal-Estrogen-Induced Changes in Gene Expression Related to Brain Sexual Differentiation in Mice',\n",
       "  'summary': 'Sexual dimorphism of the behaviors or physiological functions in mammals is mainly due to the sex difference of the brain. The goal of this study is to identify genes mediating sexaul dimorphism of the brain. The large-scale analysis with microarray in the present study is an attempt to obtain the candidate gene(s) mediating the perinatal estrogen effect causing the brain sexual differentiation.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 5},\n",
       " '200040539': {'inp_pmids': ['23042784'],\n",
       "  'title': 'Genome-wide DNA hydroxymethylation changes are associated with neurodevelopmental genes in the developing human brain',\n",
       "  'summary': '5-hydroxymethylcytosine (5-hmC) is a newly discovered modified form of cytosine that has been suspected to be an important epigenetic modification in neurodevelopment. While DNA methylation dynamics have already been implicated during neurodevelopment, little is known about hydroxymethylation in this process. Here we report DNA hydroxymethylation dynamics during cerebellum development in the human brain. Overall, we find a positive correlation between 5-hmC levels and cerebellum development. Genome-wide profiling reveals that 5-hmC is highly enriched on specific gene regions, including exons and especially the untranslated regions (UTRs), but it is depleted on introns and intergenic regions. Furthermore, we have identified fetus-specific and adult-specific differentially hydroxymethylated regions (DhMRs), most of which overlap with genes and CpG island shores. Surprisingly, during development DhMRs are highly enriched in genes encoding mRNAs that can be regulated by fragile X mental retardation protein (FMRP), some of which are disrupted in autism, as well as in many known autism genes. Our results suggest that 5-hmC-mediated epigenetic regulation may broadly impact the development of the human brain, and its dysregulation could contribute to the molecular pathogenesis of neurodevelopmental disorders.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200034429': {'inp_pmids': ['22219169'],\n",
       "  'title': 'Expression and DNA methylation of colon cancer cell lines',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array; Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 0},\n",
       " '200034427': {'inp_pmids': ['22219169'],\n",
       "  'title': 'DNA methylation analysis of Colon Cancer cell line SW48',\n",
       "  'summary': 'We combined gene expression experiments in response to HDAC inhibitor Depsipeptide with DNA methylation level to assess gene reactivation arising from hypermethylated promoters.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200034077': {'inp_pmids': ['22219169'],\n",
       "  'title': 'Colon Cancer cell line YB5 untreated cells vs YB5 treated 24h with Depsipeptide at 20 nM',\n",
       "  'summary': 'Study the effect of HDACi Depsipeptide on gene expression. We combined these data with whole genome DNA methylation microarray to test the reactivation of genes that have an hypermethylated promoter region.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200033528': {'inp_pmids': ['22384383'],\n",
       "  'title': 'Genome-wide survey of large rare copy number variations in Alzheimer’s disease among Caribbean Hispanics',\n",
       "  'summary': 'Recently genome-wide association studies have identified significant association between Alzheimer’s disease and variations in CLU, PICALM, BIN1, CR1, MS4A4/MS4A6E, CD2AP, CD33, EPHA1 and ABCA7. However, the pathogenic variants in these loci have not yet been found. We conducted a genome-wide scan for large copy number variations (CNVs) in a dataset of Caribbean Hispanic origin (554 controls and 559 cases with late-onset Alzheimer’s disease) that was previously investigated in a SNP-based genome-wide association study using Illumina HumanHap 650Y platform. We ran four CNV calling algorithms and analyzed rare large CNVs (>100 Kb) to obtain high-confidence calls that were detected by at least two algorithms. In total, 734 such CNVs were observed in our dataset. Global burden analyses did not reveal significant differences between cases and controls in CNV rate, distribution of deletions or duplications, total or average CNV size; and number of genes affected by CNVs. However, we observed a nominal association between Alzheimer’s disease and a ~470 Kb duplication on chromosome15q11.2 (P=0.037). This duplication, encompassing up to five genes (TUBGCP5, CYFIP1, NIPA2, NIPA1 and WHAMML1) was present in 10 cases (2.6%) and 3 controls (0.8%). The dosage increase of CYFIP1 and NIPA1 genes was further confirmed by quantitative PCR. The current study did not detect CNVs (including common CNVs) that affect novel Alzheimer’s disease loci reported by large genome-wide association studies. However, since the array technology used in our study has limitations in detecting small CNVs, future studies must carefully assess novel AD associated genes for the presence of disease related CNVs.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome variation profiling by SNP array',\n",
       "  'cluster_id': 5},\n",
       " '200028094': {'inp_pmids': ['21613409'],\n",
       "  'title': 'A DNA Methylation Fingerprint of 1,628 Human Samples',\n",
       "  'summary': 'DNA methylation is the best characterized of the different layers that make up the epigenetic setting. Most of the studies characterizing DNA methylation patterns have been restricted to particular genomic loci in a limited number of human samples and pathological conditions. The recently arrived single-base-resolution technologies for DNA methylation are extremely helpful tools, but are not yet applicable and affordable for studying large groups of subjects. Herein, we present a compromise between an extremely comprehensive study of a human sample population with an intermediate level of resolution of CpGs at the genomic level. We obtained a DNA methylation fingerprint of 1,628 human samples where we interrogated 1,505 CpG sites. The DNA methylation patterns revealed show this epigenetic mark to be critical in tissue-type definition and stemness, particularly around transcription start sites that are not within a CpG island. For disease, the generated DNA methylation fingerprints show that, during tumorigenesis, human cancer cells underwent a progressive gain of promoter CpG island hypermethylation and a loss of CpG methylation in non-CpG island promoters. Although transformed cells are those where DNA methylation disruption is more obvious, we observed that other common human diseases, such as neurological and autoimmune disorders, had their own distinct DNA methylation profiles. Most importantly, we provide proof of principle that the obtained DNA methylation fingerprints might be useful for translational purposes by showing that are able to identify the tumor type origin of Cancers of Unknown Primary (CUPs). Thus, the DNA methylation patterns identified across the largest spectrum of samples, tissues and diseases reported to date constitute a baseline for developing higher-resolution DNA methylation maps, and provide important clues concerning the contribution of CpG methylation to tissue identity and its changes in the most prevalent human diseases.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by array',\n",
       "  'cluster_id': 2},\n",
       " '200023694': {'inp_pmids': ['21057496'],\n",
       "  'title': 'Analysis and design of RNA sequencing experiments for identifying mRNA isoform regulation',\n",
       "  'summary': 'Through alternative splicing, most human genes express multiple isoforms that may have distinct or even antagonistic functions. To infer isoform regulation based on data from high-throughput sequencing of cDNA fragments (RNA-Seq), we have developed MISO, a computational model that estimates the expression level of alternatively spliced exons and mRNA isoforms and provides intuitive measures of conﬁdence in these estimates. Incorporation of the length distribution of inserted cDNA fragments in paired-end RNA-Seq analysis in MISO enables dramatic improvements in estimation of alternative splicing levels relative to previous methods. We show that one lane of paired-end RNA-Seq data can provide far more information about splicing than two lanes of single-end data, depending critically on properties of the distribution of cDNA fragment lengths in the sequenced library. MISO also leads to an intuitive method to detect diﬀerentially regulated exons or isoforms. Application of this method implicates the RNA splicing factor hnRNP H in regulation of alternative cleavage and polyadenylation, a role that is supported by UV crosslinking/immunoprecipitation/high-throughput sequencing (CLIP-Seq) analysis. Together, our results provide a probabilistic framework for RNA-Seq analysis, derive functional insights into pre-mRNA processing, and yield guidelines for the optimal design of RNA-Seq experiments for studies of gene and isoform expression.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Expression profiling by high throughput sequencing; Other',\n",
       "  'cluster_id': 7},\n",
       " '200022872': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating H3K4me2 methylation',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by genome tiling array; Expression profiling by array',\n",
       "  'cluster_id': 0},\n",
       " '200022870': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating intragenic H3K4me2 methylation (ChIP-chip)',\n",
       "  'summary': 'Dynamic histone H3K4 methylation is an important epigenetic component of transcriptional regulation. However, most of our current understanding of this histone mark is confined to regulation of transcriptional initiation. We now show that human LSD2/KDM1b/AOF1, the human homolog of LSD1, is a novel H3K4me1/2 demethylase that specifically regulates histone H3K4 methylation within intragenic regions of its target genes. Genome-wide mapping reveals that LSD2 associates predominantly with the gene bodies of actively transcribed genes, but is markedly absent from promoters. Depletion of endogenous LSD2 results in an increase of H3K4me2 as well as a decrease of H3K9me2 at LSD2 binding sites, and a consequent dysregulation of target gene transcription. Furthermore, characterization of LSD2 complex revealed that LSD2 forms active complexes with euchromatic histone methyltransferases EHMT1/2 and NSD3 as well as cellular factors involved in active transcription elongation. These data provide a possible molecular mechanism linking LSD2 to transcriptional regulation post initiation.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Genome binding/occupancy profiling by genome tiling array',\n",
       "  'cluster_id': 7},\n",
       " '200022859': {'inp_pmids': ['20670891'],\n",
       "  'title': 'Human LSD2/KDM1b/AOF1 regulates gene transcription by modulating H3K4me2 methylation (Affymetrix)',\n",
       "  'summary': 'Dynamic histone H3K4 methylation is an important epigenetic component of transcriptional regulation. However, most of our current understanding of this histone mark is confined to regulation of transcriptional initiation. We now show that human LSD2/KDM1b/AOF1, the human homolog of LSD1, is a novel H3K4me1/2 demethylase that specifically regulates histone H3K4 methylation within intragenic regions of its target genes. Genome-wide mapping reveals that LSD2 associates predominantly with the gene bodies of actively transcribed genes, but is markedly absent from promoters. Depletion of endogenous LSD2 results in an increase of H3K4me2 as well as a decrease of H3K9me2 at LSD2 binding sites, and a consequent dysregulation of target gene transcription. Furthermore, characterization of LSD2 complex revealed that LSD2 forms active complexes with euchromatic histone methyltransferases EHMT1/2 and NSD3 as well as cellular factors involved in active transcription elongation. These data provide a possible molecular mechanism linking LSD2 to transcriptional regulation post initiation. We used microarray analysis to identify the subset of genes that are differentially expression after depletion of endogenous LSD2/KDM1b/AOF1.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '200021662': {'inp_pmids': ['20385583'],\n",
       "  'title': 'A tripartite paternally methylated region within the Gpr1-Zdbf2 imprinted domain on mouse chromosome 1 identified by meDIP-on-chip',\n",
       "  'summary': 'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'organism': 'Homo sapiens; Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 0},\n",
       " '200018178': {'inp_pmids': ['20385583'],\n",
       "  'title': 'Methylation status in human sperm',\n",
       "  'summary': 'Genomic imprinting describes the expression of a subset of mammalian genes from one parental chromosome. The parent-of-origin specific expression of imprinted genes relies on DNA methylation of CpG-dinucleotides at differentially methylated regions (DMRs) during gametogenesis. We identified the paternally methylated DMR at human chromosome 2 near the imprinted ZDBF2 gene using a methylated-DNA immunoprecipitation-on-chip (meDIP-on-chip) method applied to DNA from sperm.',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200018126': {'inp_pmids': ['20385583'],\n",
       "  'title': 'Methylation status in AG- and PG-derived cells and sperm',\n",
       "  'summary': 'Genomic imprinting describes the expression of a subset of mammalian genes from one parental chromosome. The parent-of-origin specific expression of imprinted genes relies on DNA methylation of CpG-dinucleotides at differentially methylated regions (DMRs) during gametogenesis. We identified the paternally methylated DMRs at mouse chromosome 1 near the imprinted Zdbf2 gene using a methylated-DNA immunoprecipitation-on-chip (meDIP-on-chip) method applied to DNA from parthenogenetic (PG)- and androgenetic (AG)-derived cells and sperm.',\n",
       "  'organism': 'Mus musculus',\n",
       "  'exp_type': 'Methylation profiling by genome tiling array',\n",
       "  'cluster_id': 2},\n",
       " '200017554': {'inp_pmids': ['19723310', '20602769'],\n",
       "  'title': 'Nucleosome deposition and DNA methylation at coding region boundaries',\n",
       "  'summary': 'We profiled CpG methylation for human T cells and compared this pattern with public T cell nucleosome (H2A.Z) and polymerase II profiles (SRA000234). Focusing on DNA regions surrounding the start codon and stop codon, instead of the transcription start and end sites, we discovered a very intriguing pattern, namely methylation and nucleosomal peaks at those regions, more prominent than peaks near transcript boundaries. Elongating polymerases tend to pause near both coding ends immediately upstream of the epigenetic peaks, causing a significant reduction in elongation efficiency, suggesting their role in control of RNA splicing. Conserved features in underlying protein coding sequences seem to dictate their evolutionary conservation across multiple species. The nucleosomal and methylation marks are commonly associated with high sequence-encoded DNA-bending propensity but differentially with CpG density. Keywords: Epigenetics',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Methylation profiling by high throughput sequencing',\n",
       "  'cluster_id': 2},\n",
       " '200015745': {'inp_pmids': ['20485568'],\n",
       "  'title': 'Abundant Quantitative Trait Loci for CpG Methylation and Expression Across Human Brain Tissues',\n",
       "  'summary': 'A fundamental challenge in the post-genome era is to understand and annotate the consequences of genetic variation, particularly within the context of human tissues. We describe a set of integrated experiments designed to investigate the effects of common genetic variability on DNA methylation, mRNA expression and microRNA (miRNA) expression in four distinct human brain regions. We show that brain tissues may be readily distinguished based on methylation status or expression profile. We find an abundance of genetic cis regulation mRNA expression and show for the first time abundant quantitative trait loci for DNA CpG methylation. We observe that the largest magnitude effects occur across distinct brain regions. We believe these data, which we have made publicly available, will be useful in understanding the biological effects of genetic variation. Authorized Access data: Mapping of GEO sample accessions to dbGaP subject/sample IDs is available through dbGaP Authorized Access, see http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000249.v1.p1',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'exp_type': 'Expression profiling by array; Non-coding RNA profiling by array; Methylation profiling by array',\n",
       "  'cluster_id': 2},\n",
       " '200009990': {'inp_pmids': ['19211887'],\n",
       "  'title': 'Transcriptional profiles of rodent hippocampal CA1 tissue during aging and cognitive decline',\n",
       "  'summary': 'Recent microarray studies in the hippocampus of rodents or Alzheimer’s disease (AD) subjects have identified a substantial number of cellular pathways/processes correlated with aging and cognitive decline. However, the temporal relationships among these expression changes or with cognitive impairment have not been studied in depth. Here, using Affymetrix microarrays, immunohistochemistry and Morris water maze cognitive testing across 5 age groups of male F344 rats (n=9-15/group, one microarray per animal), we systematically analyzed the temporal sequence and cellular localization of aging changes in expression. These were correlated with performance scores on the hippocampus-dependent Morris Water Maze task. Significant microarray results were sorted in to Early, Intermediate, Midlife, and Late patterns of expression, and functionally categorized (Early- downregulated neural development, lipid synthesis and energy-utilization; upregulated ribosomal synthesis, growth, stress/inflammatory, lysosome and protein/lipid degradation. Intermediate- increased defense/inflammatory activation and decreased transporter activity; Midlife- downregulated energy-dependent signaling and neurite growth, upregulated astroglial activation, Ca2+-binding, cholesterol/lipid trafficking, myelinogenic processes and additional lysosome/inflammation; Late- further recruitment of genes in already-altered pathways). Immunohistochemistry revealed a primarily astrocytic localization of the processes upregulated in midlife, as well as increased density of myelin proteins. Evidence of cognitive impairment first appeared in the 12-month-old group (midlife) and was increased further in the 23-month-old group, exhibiting the highest correlations with some upregulated genes related to cholesterol transport (e.g., Apoe, Abca2), protein management and ion binding. Some upregulated genes for inflammation (Il6st) and myelinogenesis (Pmp22) also correlated with impairment. Together, the data are consistent with a novel sequential cascade model of brain aging in which metabolic alterations early in maturity are followed by inflammation and midlife activation of an astrocyte-centered cholesterol trafficking pathway that stimulates oligodendrocyte remyelination programs. Importantly, this cholesterol trafficking pathway also may compete for astroglial bioenergetic support of neurons, in turn, leading to downregulation of energy-dependent pathways needed to sustain cognitive functions. Keywords: aging time course',\n",
       "  'organism': 'Rattus norvegicus',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7},\n",
       " '3915': {'inp_pmids': ['19211887'],\n",
       "  'title': 'Aging and cognitive decline: hippocampus',\n",
       "  'summary': 'Analysis of hippocampi from F344 males, at 5 age points across the adult lifespan, tested on the Morris water maze (MWM) task. MWM is a dorsal hippocampal-dependent, spatial memory task. MWM performance showed an age effect. Results provide insight into the molecular basis of cognitive aging.',\n",
       "  'organism': 'Rattus norvegicus',\n",
       "  'exp_type': 'Expression profiling by array',\n",
       "  'cluster_id': 7}}"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gds_dict"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
